
# List of molecular design using Generative AI and Deep Learning 
![contributing-image](https://img.shields.io/badge/contributions-welcome-brightgreen.svg?style=flat)

related to  **Generative AI** and **Deep Learning** for  **molecular/drug design**. 

![Molecular GenerativeAI](https://github.com/AspirinCode/papers-for-molecular-design-using-DL/blob/main/figures/DMGsinchemsci.png)[Ref: Generative Models as an Emerging Paradigm in the Chemical Sciences]


**Updating ...**  

## Molecular Optimization
[Molecular Optimization](https://github.com/AspirinCode/papers-for-molecular-design-using-DL/blob/main/Molecular_Optimization.md) will welcome ！！！



## Recommendations and References


**Large Language Model for Biomedical Science, Molecule, Protein, Material Discovery**

https://github.com/HHW-zhou/LLM4Mol


**List of papers about Proteins Design using Deep Learning**

https://github.com/Peldom/papers_for_protein_design_using_DL

**Awesome Generative AI**

https://github.com/steven2358/awesome-generative-ai

**awesome-molecular-generation**

https://github.com/amorehead/awesome-molecular-generation

**A Survey of Artificial Intelligence in Drug Discovery**  

https://github.com/dengjianyuan/Survey_AI_Drug_Discovery

**Geometry Deep Learning for Drug Discovery and Life Science**  

https://github.com/3146830058/Geometry-Deep-Learning-for-Drug-Discovery-and-Life-Science


## Menu

  - [ Generative AI for Scientific Discovery](#Generative-AI-for-Scientific-Discovery)

  - [ Reviews](#reviews)

  - [ Datasets and Benchmarks](#Datasets-and-Benchmarks)
    - [Datasets](#Datasets)
    - [Benchmarks](#Benchmarks)

  - [ Drug-likeness and Evaluation metrics](#Drug-likeness-and-Evaluation-metrics)
    - [QED](#QED)
    - [QEPPI](#QEPPI)
    - [SAscore](#SAscore)
    - [RAscore](#RAscore)
    - [Evaluation metrics](#Evaluation-metrics)
  - [Deep Learning-based design](#Deep-Learning-based-design)
    - [RNN-based](#RNN-based)
    - [LSTM-based](#LSTM-based)
    - [Autoregressive-models](#Autoregressive-models)
    - [Transformer-based](#Transformer-based)
    - [VAE-based](#VAE-based)
    - [GAN-based](#GAN-based)
    - [Flow-based](#Flow-based)
    - [Score-Based](#Score-Based)
    - [Energy-based](#Energy-based)
    - [Diffusion-based](#Diffusion-based)
    - [RL-based](#RL-based)
    - [Multi-task DMGs](#Multi-task-DMGs)
    - [Monte Carlo Tree Search](#Monte-Carlo-Tree-Search)


  - [Text-driven molecular generation models](#Text-driven-molecular-generation-models)
  - [Multi-Target based deep molecular generative models](#Multi-Target-based-deep-molecular-generative-models)
  - [Ligand-based deep molecular generative models](#Ligand-based-deep-molecular-generative-models)
  - [Pharmacophore-based deep molecular generative models](#Pharmacophore-based-deep-molecular-generative-models)
  - [Structure-based deep molecular generative models](#Structure-based-deep-molecular-generative-models)

  - [Fragment-based deep molecular generative models](#Fragment-based-deep-molecular-generative-models)
    - [Scaffold-based DMGs](#Scaffold-based-DMGs)
    - [Fragment-based DMGs](#Fragment-based-DMGs)
    - [Motifs-based DMGs](#Motifs-based-DMGs)
    - [Linkers-based DMGs](#Linkers-based-DMGs)

  - [Chemical Reaction-based deep molecular generative models](#Chemical-Reaction-based-deep-molecular-generative-models)

  - [Omics-based deep molecular generative models](#Omics-based-deep-molecular-generative-models)

  - [Multi-Objective deep molecular generative models](#Multi-Objective-deep-molecular-generative-models)

  - [Quantum deep molecular generative models](#Quantum-deep-molecular-generative-models)


## Generative AI for Scientific Discovery


* **Accelerating Material Design with the Generative Toolkit for Scientific Discovery**  
Manica, Matteo and Cadow, Joris and Christofidellis, Dimitrios and Dave, Ashish and Born, Jannis and Clarke, Dean and Teukam, Yves Gaetan Nana and Hoffman, Samuel C and Buchan, Matthew and Chenthamarakshan, Vijil and others  
[npj Comput Mater 9, 69 (2023)](https://www.nature.com/articles/s41524-023-01028-1) |  [code](https://github.com/GT4SD/gt4sd-core) 




## Reviews

* **How will generative AI disrupt data science in drug discovery?**[2023]   
 Vert, JP.  
[Nat Biotechnol (2023)](https://doi.org/10.1038/s41587-023-01789-6)

* **Generative Models as an Emerging Paradigm in the Chemical Sciences**[2023]   
 Anstine, Dylan M., and Olexandr Isayev.  
[JACS (2023)](https://pubs.acs.org/doi/10.1021/jacs.2c13467)

* **Chemical language models for de novo drug design: Challenges and opportunities**[2023]  
Grisoni, Francesca.  
[Current Opinion in Structural Biology 79 (2023)](https://doi.org/10.1016/j.sbi.2023.102527)

* **Artificial intelligence in multi-objective drug design**[2023]   
 Luukkonen, Sohvi, Helle W. van den Maagdenberg, Michael TM Emmerich, and Gerard JP van Westen.  
[Current Opinion in Structural Biology 79 (2023)](https://doi.org/10.1016/j.sbi.2023.102537)

* **Integrating structure-based approaches in generative molecular design**[2023]   
 Thomas, Morgan, Andreas Bender, and Chris de Graaf.  
[Current Opinion in Structural Biology 79 (2023)](https://doi.org/10.1016/j.sbi.2023.102559)

* **Open data and algorithms for open science in AI-driven molecular informatics**[2023]  
 Brinkhaus, Henning Otto, Kohulan Rajan, Jonas Schaub, Achim Zielesny, and Christoph Steinbeck.  
[Current Opinion in Structural Biology 79 (2023)](https://doi.org/10.1016/j.sbi.2023.102542)

* **Structure-based drug design with geometric deep learning**[2023]  
  Isert, Clemens, Kenneth Atz, and Gisbert Schneider.  
[Current Opinion in Structural Biology 79 (2023)](https://doi.org/10.1016/j.sbi.2023.102548)

* **MolGenSurvey: A Systematic Survey in Machine Learning Models for Molecule Design**[2022]  
  Du, Yuanqi, Tianfan Fu, Jimeng Sun, and Shengchao Liu.  
[arXiv:2203.14500 (2022)](https://arxiv.org/abs/2203.14500)

* **Deep generative molecular design reshapes drug discovery**[2022]  
  Zeng, Xiangxiang, Fei Wang, Yuan Luo, Seung-gu Kang, Jian Tang, Felice C. Lightstone, Evandro F. Fang, Wendy Cornell, Ruth Nussinov, and Feixiong Cheng.  
[Cell Reports Medicine (2022)](https://doi.org/10.1016/j.xcrm.2022.100794)

* **Structure-based drug discovery with deep learning**[2022]  
  Özçelik, Rıza, Derek van Tilborg, José Jiménez-Luna, and Francesca Grisoni.  
[ChemBioChem (2022)](https://doi.org/10.1002/cbic.202200776)

* **Generative models for molecular discovery: Recent advances and challenges**[2022]  
 Bilodeau, Camille, Wengong Jin, Tommi Jaakkola, Regina Barzilay, and Klavs F. Jensen.  
[Computational Molecular Science 12.5 (2022)](https://doi.org/10.1002/wcms.1608)

* **Generative machine learning for de novo drug discovery: A systematic review**[2022]  
 Martinelli, Dominic.   
[Computers in Biology and Medicine 145 (2022)](https://doi.org/10.1016/j.compbiomed.2022.105403)

* **Docking-based generative approaches in the search for new drug candidates**[2022]  
 Danel, Tomasz, Jan Łęski, Sabina Podlewska, and Igor T. Podolak.   
[Drug Discovery Today (2022)](https://doi.org/10.1016/j.drudis.2022.103439)

* **Advances and Challenges in De Novo Drug Design Using Three-Dimensional Deep Generative Models**[2022]  
  Xie, Weixin, Fanhao Wang, Yibo Li, Luhua Lai, and Jianfeng Pei.   
[J. Chem. Inf. Model. 2022, 62, 10, 2269–2279](https://pubs.acs.org/doi/10.1021/acs.jcim.2c00042)

* **Deep learning to catalyze inverse molecular design**[2022]  
  Alshehri, Abdulelah S., and Fengqi You.   
[Chemical Engineering Journal 444 (2022)](https://doi.org/10.1016/j.cej.2022.136669)

* **AI in 3D compound design**[2022]  
  Hadfield, Thomas E., and Charlotte M. Deane.   
[Current Opinion in Structural Biology 73 (2022)](https://doi.org/10.1016/j.sbi.2021.102326)

* **Deep learning approaches for de novo drug design: An overview**[2021]  
 Wang, Mingyang, Zhe Wang, Huiyong Sun, Jike Wang, Chao Shen, Gaoqi Weng, Xin Chai, Honglin Li, Dongsheng Cao, and Tingjun Hou.  
[Current Opinion in Structural Biology 72 (2022)](https://doi.org/10.1016/j.sbi.2021.10.001)

* **Generative chemistry: drug discovery with deep learning generative models**[2021]  
 Bian, Yuemin, and Xiang-Qun Xie.  
[Journal of Molecular Modeling 27 (2021)](https://link.springer.com/article/10.1007/s00894-021-04674-8)

* **Generative Deep Learning for Targeted Compound Design**[2021]  
 Sousa, Tiago, João Correia, Vítor Pereira, and Miguel Rocha.  
[J. Chem. Inf. Model. 2021, 61, 11, 5343–5361](https://pubs.acs.org/doi/10.1021/acs.jcim.0c01496)

* **Generative Models for De Novo Drug Design**[2021]  
 Tong, Xiaochu, Xiaohong Liu, Xiaoqin Tan, Xutong Li, Jiaxin Jiang, Zhaoping Xiong, Tingyang Xu, Hualiang Jiang, Nan Qiao, and Mingyue Zheng.  
[Journal of Medicinal Chemistry 64.19 (2021)](https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00927)

* **Molecular design in drug discovery: a comprehensive review of deep generative models**[2021]  
 Cheng, Yu, Yongshun Gong, Yuansheng Liu, Bosheng Song, and Quan Zou.   
[Briefings in bioinformatics 22.6 (2021)](https://doi.org/10.1093/bib/bbab344)

* **De novo molecular design and generative models**[2021]  
 Meyers, Joshua, Benedek Fabian, and Nathan Brown.  
[Drug Discovery Today 26.11 (2021)](https://doi.org/10.1016/j.drudis.2021.05.019)

* **Deep learning for molecular design—a review of the state of the art**[2019]  
 Elton, Daniel C., Zois Boukouvalas, Mark D. Fuge, and Peter W. Chung.  
[Molecular Systems Design & Engineering 4.4 (2019)](https://pubs.rsc.org/en/content/articlelanding/2019/me/c9me00039a)

* **Inverse molecular design using machine learning: Generative models for matter engineering**[2018]  
 Sanchez-Lengeling, Benjamin, and Alán Aspuru-Guzik.  
[Science 361.6400 (2018)](https://www.science.org/doi/10.1126/science.aat2663)

## Datasets and Benchmarks

### Datasets

**DrugBank**

https://go.drugbank.com/

**ZINC**

https://zinc15.docking.org/

https://zinc20.docking.org/

**PubChem**

https://pubchem.ncbi.nlm.nih.gov/

**ChEMBL**

https://www.ebi.ac.uk/chembl/

**GDB Databases**

https://gdb.unibe.ch/downloads/

**QM Dataset**

http://quantum-machine.org/datasets/

**ChemSpider**

http://www.chemspider.com/

**COCONUT**  
Collection of Open Natural Products database

https://coconut.naturalproducts.net/

### Benchmarks


* **Generative Models Should at Least Be Able to Design Molecules That Dock Well: A New Benchmark** [2023]  
Ciepliński, Tobiasz, Tomasz Danel, Sabina Podlewska, and Stanisław Jastrzȩbski.  
[J. Chem. Inf. Model. 2023, 63, 11, 3238–3247](https://pubs.acs.org/doi/10.1021/acs.jcim.2c01355) |  [code](https://github.com/cieplinski-tobiasz/smina-docking-benchmark) 

* **Tartarus: A Benchmarking Platform for Realistic And Practical Inverse Molecular Design** [2022]  
Nigam, AkshatKumar, Robert Pollice, Gary Tom, Kjell Jorner, Luca A.  
[arXiv:2209.12487v1](https://arxiv.org/abs/2209.12487) |  [code](https://github.com/aspuru-guzik-group/Tartarus) 


* **Molecular Sets (MOSES): A benchmarking platform for molecular generation models**  

  [Frontiers in pharmacology 11 (2020)](https://doi.org/10.3389/fphar.2020.565644) |  [code](https://github.com/molecularsets/moses) 


* **GuacaMol: Benchmarking Models for de Novo Molecular Design**

https://github.com/BenevolentAI/guacamol


## Drug-likeness and Evaluation metrics

**Drug-likeness** may be defined as a complex balance of various molecular properties and structure features which determine whether particular molecule is similar to the known drugs. These properties, mainly hydrophobicity, electronic distribution, hydrogen bonding characteristics, molecule size and flexibility and of course presence of various pharmacophoric features influence the behavior of molecule in a living organism, including bioavailability, transport properties, affinity to proteins, reactivity, toxicity, metabolic stability and many others.

https://github.com/AspirinCode/DrugAI_Drug-Likeness

### QED
quantitative estimation of drug-likeness

Bickerton, G., Paolini, G., Besnard, J. et al. Quantifying the chemical beauty of drugs. Nature Chem 4, 90–98 (2012). https://doi.org/10.1038/nchem.1243

### QEPPI
quantitative estimate of protein-protein interaction targeting drug-likeness

https://github.com/ohuelab/QEPPI

### SAscore
**Estimation of synthetic accessibility score of drug-like molecules based on molecular complexity and fragment contributions**  
[J Cheminform 1, 8 (2009)](http://www.jcheminf.com/content/1/1/8) |  [code](https://github.com/rdkit/rdkit/tree/master/Contrib/SA_Score) 

### RAscore
**Retrosynthetic accessibility score (RAscore) – rapid machine learned synthesizability classification from AI driven retrosynthetic planning**  
[Chemical Science 12.9 (2021)](https://pubs.rsc.org/en/content/articlelanding/2021/sc/d0sc05401a) |  [code](https://github.com/reymond-group/RAscore) 

### Evaluation metrics

* **FCD : Fréchet ChemNet Distance**
  
  Fréchet ChemNet Distance: A Metric for Generative Models for Molecules in Drug Discovery

  https://github.com/bioinf-jku/FCD

* **Perplexity-Based Molecule Ranking and Bias Estimation of Chemical Language Models** [2022]  
  Moret, M., Grisoni, F., Katzberger, P. and Schneider, G.  
[J. Chem. Inf. Model. 2022, 62, 5, 1199–1206](https://pubs.acs.org/doi/10.1021/acs.jcim.2c00079) |  [code](https://github.com/ETHmodlab/CLM_perplexity) 

* **MolScore**
  
  An automated scoring function to facilitate and standardize evaluation of goal-directed generative models for de novo molecular design

  https://github.com/MorganCThomas/MolScore



## Deep Learning-based design


### RNN-based

* **Magicmol: a light-weighted pipeline for drug-like molecule evolution and quick chemical space exploration** [2023]  
Chen, Lin, Qing Shen, and Jungang Lou.  
[BMC Bioinformatics (2023)](https://doi.org/10.1186/s12859-023-05286-0) |  [code](https://github.com/Josefjosda/Magicmol) 

* **Augmented Hill-Climb increases reinforcement learning efficiency for language-based de novo molecule generation** [2022]  
Thomas, M., O’Boyle, N.M., Bender, A. et al.  
[ J Cheminform (2022)](https://doi.org/10.1186/s13321-022-00646-z) |  [code](https://github.com/MorganCThomas/SMILES-RNN) 

* **De novo molecule design with chemical language models** [2022]  
Grisoni, F., Schneider, G.  
[Artificial Intelligence in Drug Design. Methods in Molecular Biology, vol 2390.(2022)](https://doi.org/10.1007/978-1-0716-1787-8_9) |  [code](https://github.com/grisoniFr/de_novo_design_RNN) 

* **Optimizing Recurrent Neural Network Architectures for De Novo Drug Design** [2021]  
Santos, B. P., Abbasi, M., Pereira, T., Ribeiro, B., & Arrais, J. P.  
[Paper](https://ieeexplore.ieee.org/document/9474742) |  [code](https://github.com/larngroup/RNN-Drug-Generation) 

* **A recurrent neural network (RNN) that generates drug-like molecules for drug discovery** [2021]  
[code](https://github.com/shiwentao00/Molecule-RNN) 

* **A molecule generative model used interaction fingerprint (docking pose) as constraints** [2021]  
[code](https://github.com/jeah-z/IFP-RNN) 

* **Bidirectional Molecule Generation with Recurrent Neural Networks** [2020]  
Grisoni, F., Moret, M., Lingwood, R., & Schneider, G.  
[J. Chem. Inf. Model. (2020)](https://pubs.acs.org/doi/10.1021/acs.jcim.9b00943) |  [code](https://github.com/robinlingwood/BIMODAL) 

* **Direct steering of de novo molecular generation with descriptor conditional recurrent neural networks** [2019]  
Kotsias, PC., Arús-Pous, J., Chen, H. *et al.*  
[Nat Mach Intell 2, 254–265 (2020)](https://www.nature.com/articles/s42256-020-0174-5) |  [code](https://github.com/pcko1/Deep-Drug-Coder) 

* **ChemTS: An Efficient Python Library for de novo Molecular Generation** [2017]  
Yang, X., Zhang, J., Yoshizoe, K., Terayama, K., & Tsuda, K.  
[Science and Technology of Advanced Materials (2017)](https://www.tandfonline.com/doi/full/10.1080/14686996.2017.1401424) |  [code](https://github.com/tsudalab/ChemTS) 




### LSTM-based

* **Leveraging molecular structure and bioactivity with chemical language models for de novo drug design** [2023]  
  Kotsias, PC., Arús-Pous, J., Chen, H. *et al.*  
  [Nat Commun 14, 114 (2023)](https://www.nature.com/articles/s41467-022-35692-6) |  [code](https://github.com/ETHmodlab/hybridCLMs/tree/v1.0) 

* **SMILES-based CharLSTM with finetuning and goal-directed generation via policy gradient** [2022]  

  [code](https://github.com/gmattedi/Smiles-LSTM) 

* **DeLA-Drug: A Deep Learning Algorithm for Automated Design of Druglike Analogues** [2022]  
  Creanza, T. M., Lamanna, G., Delre, P., Contino, M., Corriero, N., Saviano, M., ... & Ancona, N.   
  [J. Chem. Inf. Model. (2022)](https://pubs.acs.org/doi/10.1021/acs.jcim.2c00205) |  [Web](http://www.ba.ic.cnr.it/softwareic/deladrugportal/) 

* **De novo design and bioactivity prediction of SARS-CoV-2 main protease inhibitors using recurrent neural network-based transfer learning** [2021]  
Santana, M.V.S., Silva-Jr, F.P.  
  [BMC Chemistry 15, 8 (2021)](https://bmcchem.biomedcentral.com/articles/10.1186/s13065-021-00737-2) |  [code](https://github.com/marcossantanaioc/De_novo_design_SARSCOV2) 

* **Generative Recurrent Networks for De Novo Drug Design** [2018]  
  Gupta, A., Müller, A. T., Huisman, B. J., Fuchs, J. A., Schneider, P., & Schneider, G.   
  [Mol Inform. 2018](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5836943/) |  [code](https://github.com/topazape/LSTM_Chem) 

* **Generative Recurrent Neural Networks for De Novo Drug Design** [2017]  
  Gupta, Anvita, et al.   
  [Mol Inform. 2018](https://onlinelibrary.wiley.com/doi/10.1002/minf.201700111) |  [code](https://github.com/SilviaAmAm/MolBot) 


### Autoregressive-models

* **Domain-Agnostic Molecular Generation with Self-feedback** [2023]  
 Yin Fang, Ningyu Zhang, Zhuo Chen, Xiaohui Fan, Huajun Chen   
  [arXiv:2301.11259v3](https://arxiv.org/abs/2301.11259) |  [code](https://github.com/zjunlp/MolGen) 

* **GraphAF: a Flow-based Autoregressive Model for Molecular Graph Generation** [2020]  
  Shi, C., Xu, M., Zhu, Z., Zhang, W., Zhang, M., & Tang, J.   
    [ICLR (2020)](https://openreview.net/forum?id=S1esMkHYPr) |[arXiv:2001.09382](https://arxiv.org/abs/2001.09382) |  [code](https://github.com/DeepGraphLearning/GraphAF) 



###  Transformer-based

* **Adaptive language model training for molecular designs** [2023]  
 Andrew E. Blanchard, Debsindhu Bhowmik, Zachary Fox, John Gounley, Jens Glaser, Belinda S. Akpa & Stephan Irle.  
  [ J Cheminform 15, 59 (2023)](https://doi.org/10.1186/s13321-023-00719-7) |  [code](https://code.ornl.gov/candle/mlmol%20in%20the%20adaptive-lm%20directory) 

* **CMGN: a conditional molecular generation net to design target-specific molecules with desired properties** [2023]   
 Yang, Minjian, Hanyu Sun, Xue Liu, Xi Xue, Yafeng Deng, and Xiaojian Wang.    
  [Briefings in Bioinformatics, 2023;, bbad185](https://doi.org/10.1093/bib/bbad185) |  [code](https://github.com/WJmodels/CMGN) 

* **cMolGPT: A Conditional Generative Pre-Trained Transformer for Target-Specific De Novo Molecular Generation** [2023]   
 Wang, Ye, Honggang Zhao, Simone Sciabola, and Wenlu Wang.    
  [Molecules 2023, 28(11), 4430](https://doi.org/10.3390/molecules28114430) |  [code](https://github.com/VV123/cMolGPT) 

* **Molecule generation using transformers and policy gradient reinforcement learning** [2023]   
 Mazuz, E., Shtar, G., Shapira, B. et al.    
  [Sci Rep 13, 8799 (2023)](https://doi.org/10.1038/s41598-023-35648-w) |  [code](https://github.com/eyalmazuz/MolGen) 

* **iupacGPT: IUPAC-based large-scale molecular pre-trained model for property prediction and molecule generation** [2023]   
 Jiashun Mao,, Jianmin  Wang, Kwang-Hwi Cho, Kyoung Tai No    
  [chemrxiv-2023-5kjvh](https://doi.org/10.26434/chemrxiv-2023-5kjvh) |  [code](https://github.com/AspirinCode/iupacGPT) 

* **Molecular Generation with Reduced Labeling through Constraint Architecture** [2023]   
 Wang, Jike, Yundian Zeng, Huiyong Sun, Junmei Wang, Xiaorui Wang, Ruofan Jin, Mingyang Wang et al.    
  [J. Chem. Inf. Model. (2023)](https://pubs.acs.org/doi/full/10.1021/acs.jcim.3c00579) |  [code](https://github.com/jkwang93/Frag-G_M) 

* **Generative Discovery of Novel Chemical Designs using Diffusion Modeling and Transformer Deep Neural Networks with Application to Deep Eutectic Solvents** [2023]   
 Luu, Rachel K., Marcin Wysokowski, and Markus J. Buehler.    
  [arXiv:2304.12400v1](https://arxiv.org/abs/2304.12400) |  [code](https://github.com/lamm-mit/MoleculeDiffusionTransformer) 

* **Regression Transformer enables concurrent sequence regression and generation for molecular language modelling** [2023]   
 Born, J., Manica, M.    
  [Nat Mach Intell 5, 432–444 (2023)](https://doi.org/10.1038/s42256-023-00639-z) |  [code](https://github.com/GT4SD/gt4sd-core/tree/main/examples/regression_transformer) 

* **Transformer-based molecular generative model for antiviral drug design** [2023]  
 mao, jiashun; wang, jianming; zeb, amir; Cho, Kwang-Hwi; jin, haiyan; Kim, Jongwan; Lee, Onju; Wang, Yunyun; No, Kyoung Tai.    
  [Paper](https://dx.doi.org/10.2139/ssrn.4345811) |  [code](https://github.com/AspirinCode/TransAntivirus) 

* **Target Specific De Novo Design of Drug Candidate Molecules with Graph Transformer-based Generative Adversarial Networks** [2023]   
  Ünlü, Atabey, Elif Çevrim, Ahmet Sarıgün, Hayriye Çelikbilek, Heval Ataş Güvenilir, Altay Koyaş, Deniz Cansen Kahraman, Ahmet Rifaioğlu, and Abdurrahman Olğaç.   
  [arXiv:2302.07868v5](https://arxiv.org/abs/2302.07868)

* **DrugEx v3: scaffold-constrained drug design with graph transformer-based reinforcement learning** [2023]  
  Liu, X., Ye, K., van Vlijmen, H.W.T. et al.   
  [J Cheminform 15, 24 (2023)](https://doi.org/10.1186/s13321-023-00694-z) |  [code](https://github.com/CDDLeiden/DrugEx) 

* **Explore drug-like space with deep generative models** [2023]  
  Wang, Jianmin, et al.  
  [Methods (2023)](https://doi.org/10.1016/j.ymeth.2023.01.004) |  [code](https://github.com/AspirinCode/drug-likeness_space) 

* **Large-scale chemical language representations capture molecular structure and properties** [2022]  
  Ross, J., Belgodere, B., Chenthamarakshan, V., Padhi, I., Mroueh, Y., & Das, P.   
  [Nat Mach Intell 4, 1256–1264 (2022)](https://www.nature.com/articles/s42256-022-00580-7) |  [code](https://github.com/IBM/molformer) 

* **AlphaDrug: protein target specific de novo molecular generation** [2022]  
  Qian, Hao, Cheng Lin, Dengwei Zhao, Shikui Tu, and Lei Xu.  
  [ PNAS Nexus (2022)](https://academic.oup.com/pnasnexus/article/1/4/pgac227/6751929) |  [code](https://github.com/CMACH508/AlphaDrug) 

* **Can We Quickly Learn to “Translate” Bioactive Molecules with Transformer Models?** [2022]  
  Bagal, V., Aggarwal, R., Vinod, P. K., & Priyakumar, U. D.  
  [chemrxiv-2022-gln27](https://doi.org/10.26434/chemrxiv-2022-gln27)

* **MolGPT: Molecular Generation Using a Transformer-Decoder Model** [2022]  
  Bagal, V., Aggarwal, R., Vinod, P. K., & Priyakumar, U. D.  
  [J. Chem. Inf. Model. 2022, 62, 9, 2064–2076](https://pubs.acs.org/doi/10.1021/acs.jcim.1c00600) |  [code](https://github.com/devalab/molgpt) 
  
* **Tailoring Molecules for Protein Pockets: a Transformer-based Generative Solution for Structured-based Drug Design** [2022]  
  Wu, K., Xia, Y., Fan, Y., Deng, P., Liu, H., Wu, L., ... & Liu, T. Y.  
  [arXiv.2209.06158](https://arxiv.org/abs/2209.06158) |  [code](https://github.com/HankerWu/TamGent) 
 
 * **Exploiting pretrained biochemical language models for targeted drug design** [2022]  
  Uludoğan, Gökçe, Elif Ozkirimli, Kutlu O. Ulgen, Nilgün Karalı, and Arzucan Özgür.  
  [Bioinformatics (2022)](https://doi.org/10.1093/bioinformatics/btac482) |  [code](https://github.com/boun-tabi/biochemical-lms-for-drug-design) 
 
* **A Transformer-based Generative Model for De Novo Molecular Design** [2022]  
  Wang, Wenlu, et al.  
  [arXiv:2210.08749v2](https://arxiv.org/abs/2210.08749)

* **Translation between Molecules and Natural Language** [2022]  
  Edwards, C., Lai, T., Ros, K., Honke, G., & Ji, H.  
  [arXiv:2204.11817v3](https://arxiv.org/abs/2204.11817) |  [code](https://github.com/blender-nlp/MolT5) 

* **Regression Transformer enables concurrent sequence regression and generation for molecular language modeling** [2022]  
  Born, Jannis and Manica, Matteo  
  [arXiv:2202.01338v3](https://arxiv.org/abs/2202.01338) |  [code](https://github.com/IBM/regression-transformer) 

* **Generative Pre-Training from Molecules** [2021]  
  Adilov, Sanjar.  
  [J. Chem. Inf. Model. 2022, 62, 9, 2064–2076](https://pubs.acs.org/doi/10.1021/acs.jcim.1c00600) |  [code](https://github.com/sanjaradylov/smiles-gpt) 

* **Transformers for Molecular Graph Generation** [2021]  
  Cofala, Tim, and Oliver Kramer.  
  [ESANN 2021 ](https://www.esann.org/sites/default/files/proceedings/2021/ES2021-112.pdf) |  [code](https://gitlab.uni-oldenburg.de/gies6280/molegent) 

* **Spatial Generation of Molecules with Transformers** [2021]  
  Cofala, Tim, and Oliver Kramer.  
  [IJCNN52387.2021.9533439 (2021)](https://ieeexplore.ieee.org/abstract/document/9533439) |  [code](https://gitlab.uni-oldenburg.de/gies6280/molegent) 

* **Generative Chemical Transformer: Neural Machine Learning of Molecular Geometric Structures from Chemical Language via Attentio** [2021]  
  Hyunseung Kim, Jonggeol Na*, and Won Bo Lee*.  
  [J. Chem. Inf. Model. 2021, 61, 12, 5804–5814](https://pubs.acs.org/doi/abs/10.1021/acs.jcim.1c01289) |  [code](https://github.com/Hyunseung-Kim/molGCT) 

* **C5T5: Controllable Generation of Organic Molecules with Transformer** [2021]  
  Rothchild, D., Tamkin, A., Yu, J., Misra, U., & Gonzalez, J.  
  [arXiv:2108.10307v1](https://arxiv.org/abs/2108.10307) |  [code](https://github.com/dhroth/c5t5) 

* **Molecular optimization by capturing chemist’s intuition using deep neural networks** [2021]  
  He, J., You, H., Sandström, E. et al.   
  [J Cheminform 13, 26 (2021)](https://doi.org/10.1186/s13321-021-00497-0) |  [code](https://github.com/MolecularAI/deep-molecular-optimization) 

* **Transformer neural network for protein-specific de novo drug generation as a machine translation problem** [2021]  
  Grechishnikova, Daria.   
  [Sci Rep 11, 321 (2021)](https://www.nature.com/articles/s41598-020-79682-4) |  [code](https://github.com/dariagrechishnikova/molecule_structure_generation) 

* **Transmol: repurposing a language model for molecular generation** [2021]  
  Grechishnikova, Daria.   
  [RSC advances. 2021;11(42):25921-32.](https://pubs.rsc.org/en/content/articlelanding/2021/ra/d1ra03086h) |  [code](https://gitlab.com/cheml.io/public/transmol) 

* **Attention-based generative models for de novo molecular design** [2021]  
  Dollar, O., Joshi, N., Beck, D.A. and Pfaendtner, J.,  
  [Chemical Science 12.24 (2021)](https://pubs.rsc.org/en/content/articlelanding/2021/sc/d1sc01050f) |  [code](https://github.com/oriondollar/TransVAE) 



### VAE-based

* **A Protein-Ligand Interaction-focused 3D Molecular Generative Framework for Generalizable Structure-based Drug Design** [2023]  
 Zhung W, Kim H, Kim WY.   
  [chemrxiv-2023-jsjwx](https://doi.org/10.26434/chemrxiv-2023-jsjwx) |  [code](https://github.com/ACE-KAIST/DeepICL) 

* **VGAE-MCTS: a New Molecular Generative Model combining Variational Graph Auto-Encoder and Monte Carlo Tree Search** [2023]  
 Iwata, Hiroaki, Taichi Nakai, Takuto Koyama, Shigeyuki Matsumoto, Ryosuke Kojima, and Yasushi Okuno.   
  [chemrxiv-2023-q8419-v2](https://doi.org/10.26434/chemrxiv-2023-q8419-v2) 

* **Deep Generation Model Guided by the Docking Score for Active Molecular Design** [2023]  
 Yang, Yuwei, Chang-Yu Hsieh, Yu Kang, Tingjun Hou, Huanxiang Liu, and Xiaojun Yao.   
  [J. Chem. Inf. Model. 2023, 63, 10, 2983–2991](https://pubs.acs.org/doi/full/10.1021/acs.jcim.3c00572) |  [code](https://github.com/jaechanglim/CVAE) 

* **Direct De Novo Molecule Generation Using Probabilistic Diverse Variational Autoencoder** [2023]  
 Singh Bhadwal, Arun, and Kamal Kumar.     
  [Computer Vision and Machine Intelligence. Lecture Notes in Networks and Systems, vol 586. Springer (2023)](https://link.springer.com/chapter/10.1007/978-981-19-7867-8_2) 

* **MoVAE: A Variational AutoEncoder for Molecular Graph Generation** [2023]   
 Lin, Zerun, Yuhan Zhang, Lixin Duan, Le Ou-Yang, and Peilin Zhao.   
  [Society for Industrial and Applied Mathematics, 2023.](https://doi.org/10.1137/1.9781611977653.ch58) 

* **Gex2SGen: Designing Drug-like Molecules from Desired Gene Expression Signatures** [2023]   
 Das, Dibyajyoti, Broto Chakrabarty, Rajgopal Srinivasan, and Arijit Roy.   
  [J. Chem. Inf. Model. (2023)](https://doi.org/10.1021/acs.jcim.2c01301)

* **COMA: efficient structure-constrained molecular generation using contractive and margin losses** [2023]  
 Choi, J., Seo, S. & Park, S.     
  [J Cheminform 15, 8 (2023)](https://doi.org/10.1186/s13321-023-00679-y) |  [code](https://github.com/mathcom/COMA) 

* **Design of potent antimalarials with generative chemistry** [2022]   
  Godinez, W.J., Ma, E.J., Chao, A.T. et al.  
  [Nat Mach Intell 4, 180–186 (2022)](https://doi.org/10.1038/s42256-022-00448-w)  |  [code](https://github.com/Novartis/JAEGER) 

* **Accelerating Bayesian Optimization for Biological Sequence Design with Denoising Autoencoders** [2022]  
  Stanton, S., Maddox, W., Gruver, N., Maffettone, P., Delaney, E., Greenside, P., & Wilson, A. G.   
  [PMLR 162:20459-20478, 2022](https://arxiv.org/abs/2203.12742) 

* **Conditional β-VAE for De Novo Molecular Generation** [2022]  
  Richards, Ryan J., and Austen M. Groener.   
  [arXiv:2205.01592v1 ](https://arxiv.org/abs/2205.01592) 

* **MGCVAE: Multi-Objective Inverse Design via Molecular Graph Conditional Variational Autoencoder** [2022]  
  Lee, Myeonghun, and Kyoungmin Min.  
  [J. Chem. Inf. Model. 2022, 62, 12, 2943–2950](https://pubs.acs.org/doi/10.1021/acs.jcim.2c00487) |  [code](https://github.com/mhlee216/MGCVAE) 

* **RELATION: A Deep Generative Model for Structure-Based De Novo Drug Design** [2022]  
  Wang, M., Hsieh, C.Y., Wang, J., Wang, D., Weng, G., Shen, C., Yao, X., Bing, Z., Li, H., Cao, D. and Hou, T.,  
  [J. Med. Chem. 2022, 65, 13, 9478–9492](https://pubs.acs.org/doi/10.1021/acs.jmedchem.2c00732) |  [code](https://github.com/micahwang/RELATION) 

* **3DLinker: An E(3) Equivariant Variational Autoencoder for Molecular Linker Design** [2022]  
  Huang, Yinan, Xingang Peng, Jianzhu Ma, and Muhan Zhang.   
  [Paper]https://arxiv.org/abs/2205.07309) |  [code](https://github.com/GraphPKU/3DLinker) 

* **Molecule Generation by Principal Subgraph Mining and Assembling** [2022]  
  Kong, X., Huang, W., Tan, Z., & Liu, Y.  
  [NeurIPS 2022](https://openreview.net/forum?id=ATfARCRmM-a) |[arXiv:2106.15098v4](https://arxiv.org/abs/2106.15098) |  [code](https://github.com/THUNLP-MT/PS-VAE) 
  
* **LIMO: Latent Inceptionism for Targeted Molecule Generation** [2022]  
  Eckmann, Peter, Kunyang Sun, Bo Zhao, Mudong Feng, Michael K. Gilson, and Rose Yu.  
  [ICML (2022)](https://proceedings.mlr.press/v162/eckmann22a/eckmann22a.pdf) | [arXiv:2206.09010v1](https://arxiv.org/abs/2206.09010) |  [code](https://github.com/rose-stl-lab/limo) 

* **Predicting chemical structure using reinforcement learning with a stack-augmented conditional variational autoencoder** [2022]  
  Kim, H., Ko, S., Kim, B.J. *et al.*  
  [J Cheminform 14, 83 (2022)](https://jcheminf.biomedcentral.com/articles/10.1186/s13321-022-00666-9) |  [code](https://github.com/HwanheeKim813/stack_CVAE) 

* **Geometry-Based Molecular Generation With Deep Constrained Variational Autoencoder** [2022]   
  Li, Chunyan, Junfeng Yao, Wei Wei, Zhangming Niu, Xiangxiang Zeng, Jin Li, and Jianmin Wang.   
  [IEEE Transactions on Neural Networks and Learning Systems (2022)](https://ieeexplore.ieee.org/abstract/document/9714718) |  [code](https://github.com/anny0316/GEOM-CVAE) 

* **Inverse design of nanoporous crystalline reticular materials with deep generative models.** [2021]  
  Yao, Z., Sánchez-Lengeling, B., Bobbitt, N.S. et al.  
  [Nat Mach Intell 3, 76–86 (2021)](https://doi.org/10.1038/s42256-020-00271-1) |  [code](https://github.com/zhenpengyao/Supramolecular_VAE) 

* **Attention-based generative models for de novo molecular design** [2021]  
  Dollar, O., Joshi, N., Beck, D.A. and Pfaendtner, J.,  
  [Chemical Science 12.24 (2021)](https://pubs.rsc.org/en/content/articlelanding/2021/sc/d1sc01050f) |  [code](https://github.com/oriondollar/TransVAE) 

* **Toward efficient generation, correction, and properties control of unique drug-like structures** [2021]  
  Druchok, Maksym, Dzvenymyra Yarish, Oleksandr Gurbych, and Mykola Maksymenko.  
  [Journal of Computational Chemistry 42.11 (2021)](https://onlinelibrary.wiley.com/doi/10.1002/jcc.26494) |  [code](https://github.com/SoftServeInc/novel-molecule-generation) 

* **Compressed graph representation for scalable molecular graph generation** [2020]  
  Kwon, Youngchun, Dongseon Lee, Youn-Suk Choi, Kyoham Shin, and Seokho Kang.  
  [J Cheminform 12, 58 (2020)](https://jcheminf.biomedcentral.com/articles/10.1186/s13321-020-00463-2) |  [code](https://github.com/seokhokang/graphvae_compress) 

* **Fragment Graphical Variational AutoEncoding for Screening Molecules with Small Data** [2019]  
  Armitage, John, Leszek J. Spalek, Malgorzata Nguyen, Mark Nikolka, Ian E. Jacobs, Lorena Marañón, Iyad Nasrallah et al.  
  [arXiv:1910.13325v2](https://arxiv.org/abs/1910.13325) |  [code](https://github.com/OE-FET/FraGVAE) 

* **Molecular generative model based on conditional variational autoencoder for de novo molecular design** [2018]  
  Lim, J., Ryu, S., Kim, J. W., & Kim, W. Y.   
  [J Cheminform 10, 31 (2018)](https://jcheminf.biomedcentral.com/articles/10.1186/s13321-018-0286-7) |  [code](https://github.com/jaechanglim/CVAE) 

* **Automatic chemical design using a data-driven continuous representation of molecules** [2017]  
  Gómez-Bombarelli, R., Wei, J. N., Duvenaud, D., Hernández-Lobato, J. M., Sánchez-Lengeling, B., Sheberla, D., ... & Aspuru-Guzik, A.   
  [ACS Cent. Sci. 2018](https://pubs.acs.org/doi/10.1021/acscentsci.7b00572) | [arXiv:1610.02415v3](https://arxiv.org/abs/1610.02415) |  [code](https://github.com/tuantla80/VAE-Molecular-Generation) 




### GAN-based

* **Mol-Zero-GAN: Zero-Shot Adaptation of Molecular Generative Adversarial Network for Specific Protein Targets** [2023]   
Ravipas Aphikulvanich*， Natapol Pornputtapong， Duangdao Wichadakul   
[Paper](https://doi.org/10.26434/chemrxiv-2023-lv2m1) |  [code](https://github.com/cucpbioinfo/Mol-Zero-GAN) 

* **De Novo Design of Molecules Towards Biased Properties via a Deep Generative Framework and Iterative Transfer Learning** [2023]   
Sattari, Kianoosh, Dawei Li, Yunchao Xie, Olexandr Isayev, and Jian Lin.   
[Paper](https://doi.org/10.26434/chemrxiv-2023-0zv2f-v2) |  [code](https://github.com/linresearchgroup/RRCGAN_Molecules) 

* **MolFilterGAN: a progressively augmented generative adversarial network for triaging AI-designed molecules** [2023]   
Liu, X., Zhang, W., Tong, X. et al.   
[J Cheminform 15, 42 (2023)](https://doi.org/10.1186/s13321-023-00711-1) |  [code](https://github.com/MolFilterGAN/MolFilterGAN) 

* **Deep generative model for drug design from protein target sequence** [2023]   
Yangyang Chen, Zixu Wang, Lei Wang, Jianmin Wang, Pengyong Li, Dongsheng Cao, Xiangxiang Zeng, Xiucai Ye & Tetsuya Sakurai.   
[J Cheminform 15, 38 (2023)](https://doi.org/10.1186/s13321-023-00702-2) |  [code](https://github.com/viko-3/TargetGAN) 

* **Cell morphology-guided de novo hit design by conditioning GANs on phenotypic image features** [2022]  
Zapata, Paula A. Marin, Oscar Méndez-Lucio, Tuan Le, Carsten Jörn Beese, Jörg Wichard, David Rouquié, and Djork-Arné Clevert.  
[Digital Discovery (2023)](https://doi.org/10.1039/D2DD00081D) |  [code](https://github.com/Bayer-Group/CPMolGAN) 

* **Generating 3D molecules conditional on receptor binding sites with deep generative models** [2022]  
Ragoza, Matthew, Tomohide Masuda, and David Ryan Koes.  
[Chemical science. 2022;13(9):2701-13.](https://pubs.rsc.org/en/content/articlelanding/2022/sc/d1sc05976a) |  [code](https://github.com/mattragoza/liGAN) 

* **Designing optimized drug candidates with Generative Adversarial Network** [2022]  
Abbasi, M., Santos, B.P., Pereira, T.C. et al.  
[J Cheminform 14, 40 (2022)](https://jcheminf.biomedcentral.com/articles/10.1186/s13321-022-00623-6) |  [code](https://github.com/larngroup/GAN-Drug-Generator) 

* **De novo molecular design with deep molecular generative models for PPI inhibitors** [2022]  
Jianmin Wang, Yanyi Chu, Jiashun Mao, Hyeon-Nae Jeon, Haiyan Jin, Amir Zeb, Yuil Jang, Kwang-Hwi Cho, Tao Song, Kyoung Tai No.  
[Briefings in Bioinformatics,July 2022, bbac285,](https://doi.org/10.1093/bib/bbac285) |  [code](https://github.com/AspirinCode/iPPIGAN) 

* **Improvement on Generative Adversarial Network for Targeted Drug Design** [2021]  
Santos, B. P., Abbasi, M., Pereira, T., Ribeiro, B., & Arrais, J.  
[ESANN.(2021)](https://www.cisuc.uc.pt/download-file/16987/IR5glkL7JLe3EZrI59BI) 

* **Generative Adversarial Networks for De Novo Molecular Design** [2021]  
Lee, Y.J., Kahng, H. and Kim, S.B.,  
[Molecular Informatics 40.10 (2021)](https://doi.org/10.1002/minf.202100045) |  [code](https://github.com/dudwojae/SMILES-MaskGAN) 

* **Mol-CycleGAN: a generative model for molecular optimization** [2020]  
Maziarka, Łukasz, Agnieszka Pocha, Jan Kaczmarczyk, Krzysztof Rataj, Tomasz Danel, and Michał Warchoł  
[J Cheminform 12, 2 (2020)](https://jcheminf.biomedcentral.com/articles/10.1186/s13321-019-0404-1) |  [code](https://github.com/ardigen/mol-cycle-gan) 

* **MolGAN: An implicit generative model for small molecular graph** [2018]  
De Cao, N. and Kipf, T.,  
[arXiv:1805.11973 (2018)](https://arxiv.org/abs/1805.11973) |  [code](https://github.com/yongqyu/MolGAN-pytorch) 

* **Objective-Reinforced Generative Adversarial Networks (ORGAN) for Sequence Generation Models** [2017]  
Guimaraes, G.L., Sanchez-Lengeling, B., Outeiral, C., Farias, P.L.C. and Aspuru-Guzik, A.,  
[ arXiv:1705.10843 (2017)](https://arxiv.org/abs/1705.10843) |  [code](https://github.com/gablg1/ORGAN) 



### Flow-based


* **Multi-view deep learning based molecule design and structural optimization accelerates the SARS-CoV-2 inhibitor discovery** [2022]  
Chao Pang , Yu Wang , Yi Jiang , Ruheng Wang , Ran Su  , and Leyi Wei.  
[arXiv:2212.01575 (2022)](https://arxiv.org/abs/2212.01575) |  [code](https://github.com/Pang-chao/MEDICO) 

* **Biological Sequence Design with GFlowNets** [2022]  
Jain, M., Bengio, E., Hernandez-Garcia, A., Rector-Brooks, J., Dossou, B.F., Ekbote, C.A., Fu, J., Zhang, T., Kilgour, M., Zhang, D. and Simine, L.  
[International Conference on Machine Learning. PMLR, (2022)](https://proceedings.mlr.press/v162/jain22a.html) |  [code](https://github.com/MJ10/BioSeq-GFN-AL) 

* **FastFlows: Flow-Based Models for Molecular Graph Generation** [2022]  
Frey, N.C., Gadepally, V. and Ramsundar, B.  
[arXiv:2201.12419 (2022)](https://arxiv.org/abs/2201.12419)

* **Flow Network based Generative Models for Non-Iterative Diverse Candidate Generation** [2021]  
Bengio, E., Jain, M., Korablyov, M., Precup, D. and Bengio, Y.  
[Neural Information Processing Systems 34 (2021)](https://proceedings.neurips.cc/paper/2021/hash/e614f646836aaed9f89ce58e837e2310-Abstract.html) |  [code](https://github.com/GFNOrg/gflownet) 

* **MoFlow: An Invertible Flow Model for Generating Molecular Graphs** [2020]  
Zang, Chengxi, and Fei Wang.  
[KDD '20 (2020)](https://dl.acm.org/doi/abs/10.1145/3394486.3403104) |  [code](https://github.com/calvin-zcx/moflow) 

* **GraphNVP: an Invertible Flow-based Model for Generating Molecular Graphs** [2020]   
Madhawa, K., Ishiguro, K., Nakago, K. and Abe, M.  
[arXiv:1905.11600 (2019)](https://openreview.net/forum?id=ryxQ6T4YwB)

### Score-Based


* **Exploring Chemical Space with Score-based Out-of-distribution Generation** [2023]   
 Lee, Seul, Jaehyeong Jo, and Sung Ju Hwang.    
  [arXiv:2206.07632v3](https://arxiv.org/abs/2206.07632) | [code](https://github.com/SeulLee05/MOOD) 

* **Score-Based Generative Models for Molecule Generation** [2022]  
Gnaneshwar, Dwaraknath, et al.  
[arXiv:2203.04698 (2022)](https://arxiv.org/abs/2203.04698) 




### Energy-based

* **Energy-based Generative Models for Target-specific Drug Discovery** [2022]  
Li, Junde, Collin Beaudoin, and Swaroop Ghosh.  
[arXiv:2212.02404 (2022)](https://arxiv.org/abs/2212.02404) |  [code](https://github.com/jundeli/TagMol) 

* **MOG: Molecular Out-of-distribution Generation with Energy-based Models** [2021]  
Lee, Seul, Dong Bok Lee, and Sung Ju Hwang.  
[Paper](https://openreview.net/forum?id=qkTEaJ9orc1) 


### Diffusion-based

* **SILVR: Guided Diffusion for Molecule Generation** [2023]   
 Runcie, Nicholas T., and Antonia SJS Mey.    
  [arXiv:2304.10905v1](https://arxiv.org/abs/2304.10905) |  [code](https://github.com/meyresearch/SILVR) 

* **Guided Diffusion for Inverse Molecular Design** [2023]   
 Weiss, Tomer, Luca Cosmo, Eduardo Mayo Yanes, Sabyasachi Chakraborty, Alex M. Bronstein, and Renana Gershoni-Poranne.    
  [chemrxiv-2023-z8ltp](https://doi.org/10.26434/chemrxiv-2023-z8ltp) |  [code](https://github.com/tomer196/GaUDI) 

* **Generative Discovery of Novel Chemical Designs using Diffusion Modeling and Transformer Deep Neural Networks with Application to Deep Eutectic Solvents** [2023]   
 Luu, Rachel K., Marcin Wysokowski, and Markus J. Buehler.    
  [arXiv:2304.12400v1](https://arxiv.org/abs/2304.12400) |  [code](https://github.com/lamm-mit/MoleculeDiffusionTransformer) 

* **Geometric Latent Diffusion Models for 3D Molecule Generation** [2023]   
  Minkai Xu, Alexander Powers, Ron Dror, Stefano Ermon, Jure Leskovec   
  [arXiv:2305.01140v1](https://arxiv.org/abs/2305.01140) |  [code](https://github.com/MinkaiXu/GeoLDMf) 

* **3D Equivariant Diffusion for Target-Aware Molecule Generation and Affinity Prediction** [2023]   
  Guan, Jiaqi, Wesley Wei Qian, Xingang Peng, Yufeng Su, Jian Peng, and Jianzhu Ma.   
  [ICLR (2023)](https://openreview.net/forum?id=kJqXEPXMsE0) |  [code](https://github.com/guanjq/targetdiff) 

* **Structure-based Drug Design with Equivariant Diffusion Models** [2023]  
Schneuing, A., Du, Y., Harris, C., Jamasb, A., Igashov, I., Du, W., ... & Correia, B.   
[arXiv:2210.13695 (2022)](https://openreview.net/forum?id=uKmuzIuVl8z) |  [code](https://github.com/arneschneuing/DiffSBDD) 

* **Equivariant 3D-Conditional Diffusion Models for Molecular Linker Desig** [2023]  
Igashov, I., Stärk, H., Vignac, C., Satorras, V.G., Frossard, P., Welling, M., Bronstein, M. and Correia, B.,   
[arXiv:2210.05274 (2022)](https://openreview.net/forum?id=cnsHSSLnHVV) |  [code](https://github.com/igashov/DiffLinker) 

* **MiDi: Mixed Graph and 3D Denoising Diffusion for Molecule Generation** [2023]  
Vignac, Clement, Nagham Osman, Laura Toni, and Pascal Frossard.  
[arXiv:2302.09048 (2023)](https://arxiv.org/abs/2302.09048) |  [code](https://github.com/cvignac/MiDi) 

* **Geometry-Complete Diffusion for 3D Molecule Generation** [2023]  
Morehead, Alex, and Jianlin Cheng.  
[arXiv:2302.04313 (2023)](https://arxiv.org/abs/2302.04313) |  [code](https://github.com/BioinfoMachineLearning/bio-diffusion) 

* **MDM: Molecular Diffusion Model for 3D Molecule Generation** [2022]  
Huang, Lei, Hengtong Zhang, Tingyang Xu, and Ka-Chun Wong.  
[arXiv:2209.05710 (2022)](https://arxiv.org/abs/2209.05710)

* **Diffusion-based Molecule Generation with Informative Prior Bridges** [2022]  
Lemeng Wu, Chengyue Gong, Xingchao Liu, Mao Ye, Qiang Liu  
[NeurIPS (2022)](https://openreview.net/forum?id=QagNEt9k8Vi)

* **Equivariant Diffusion for Molecule Generation in 3D** [2022]  
Hoogeboom, Emiel, Vıctor Garcia Satorras, Clément Vignac, and Max Welling.  
[International Conference on Machine Learning. PMLR, (2022)](https://proceedings.mlr.press/v162/hoogeboom22a.html) |  [code](https://github.com/ehoogeboom/e3_diffusion_for_molecules) 




### RL-based

* **Molecule generation using transformers and policy gradient reinforcement learning** [2023]   
 Mazuz, E., Shtar, G., Shapira, B. et al.    
  [Sci Rep 13, 8799 (2023)](https://doi.org/10.1038/s41598-023-35648-w) |  [code](https://github.com/eyalmazuz/MolGen) 

* **Augmented Memory: Capitalizing on Experience Replay to Accelerate De Novo Molecular Design** [2023]   
 Guo, Jeff, and Philippe Schwaller.    
  [chemrxiv-2023-qmqmq-v3](https://doi.org/10.26434/chemrxiv-2023-qmqmq-v2) |  [code](https://github.com/schwallergroup/augmented_memory) 

* **Generating Potential Protein-Protein Interaction Inhibitor Molecules Based on Physicochemical Properties** [2023]   
 Ohue, Masahito, Yuki Kojima, and Takatsugu Kosugi.   
  [Paper](https://www.preprints.org/manuscript/202305.0704/v1) |  [code](https://github.com/ohuelab/iPPI-REINVENT) 

* **Tree-Invent: A novel molecular generative model constrained with topological tree** [2023]   
 Mingyuan Xu, HongMing Chen.   
  [chemrxiv-2023-m77vk](https://doi.org/10.26434/chemrxiv-2023-m77vk) |  [code](https://github.com/MingyuanXu/Tree-Invent) 

* **De Novo Drug Design by Iterative Multi-Objective Deep Reinforcement Learning with Graph-based Molecular Quality Assessment** [2023]   
 Yi Fang, Xiaoyong Pan, Hong-Bin Shen.    
  [Bioinformatics 39.4 (2023)](https://doi.org/10.1093/bioinformatics/btad157) |  [code](https://github.com/yifang000/QADD) 

* **DrugEx v3: scaffold-constrained drug design with graph transformer-based reinforcement learning** [2023]  
  Liu, X., Ye, K., van Vlijmen, H.W.T. et al.   
  [ J Cheminform 15, 24 (2023)](https://doi.org/10.1186/s13321-023-00694-z) |  [code](https://github.com/CDDLeiden/DrugEx) 

* **COMA: efficient structure-constrained molecular generation using contractive and margin losses** [2023]  
 Choi, J., Seo, S. & Park, S.     
  [J Cheminform 15, 8 (2023)](https://doi.org/10.1186/s13321-023-00679-y) |  [code](https://github.com/mathcom/COMA) 

* **Generative and reinforcement learning approaches for the automated de novo design of bioactive compounds** [2022]  
Korshunova, M., Huang, N., Capuzzi, S. et al.  
[Commun Chem 5, 129 (2022)](https://doi.org/10.1038/s42004-022-00733-0) |  [code](https://github.com/isayevlab/rl_experiments) 

* **Augmented Hill-Climb increases reinforcement learning efficiency for language-based de novo molecule generation** [2022]  
Thomas, M., O’Boyle, N.M., Bender, A. et al.  
[ J Cheminform 14, 68 (2022)](https://doi.org/10.1186/s13321-022-00646-z) |  [code](https://github.com/MorganCThomas/SMILES-RNN) 


* **Predicting chemical structure using reinforcement learning with a stack-augmented conditional variational autoencoder** [2022]  
  Kim, H., Ko, S., Kim, B.J. *et al.*  
  [J Cheminform 14, 83 (2022)](https://jcheminf.biomedcentral.com/articles/10.1186/s13321-022-00666-9) |  [code](https://github.com/HwanheeKim813/stack_CVAE) 

* **De Novo Drug Design Using Reinforcement Learning with Graph-Based Deep Generative Models** [2022]  
Atance, S.R., Diez, J.V., Engkvist, O., Olsson, S. and Mercado, R.  
[J. Chem. Inf. Model. 2022, 62, 20, 4863–4872](https://pubs.acs.org/doi/10.1021/acs.jcim.2c00838) |  [code](https://github.com/olsson-group/RL-GraphINVENT) 

* **DRlinker: Deep Reinforcement Learning for Optimization in Fragment Linking Design** [2022]
  Tan, Y., Dai, L., Huang, W., Guo, Y., Zheng, S., Lei, J., ... & Yang, Y. 
  [J. Chem. Inf. Model. 2022, 62, 23, 5907–5917](https://pubs.acs.org/doi/10.1021/acs.jcim.2c00982) |  [code](https://github.com/biomed-AI/DRlinker) 

* **Widely Used and Fast De Novo Drug Design by a Protein Sequence-Based Reinforcement Learning Model** [2022]  
Li, Yaqin, Lingli Li, Yongjin Xu, and Yi Yu.
[bioRxiv (2022)](https://arxiv.org/abs/2209.07405) 

* **Molecular Design Method Using a Reversible Tree Representation of Chemical Compounds and Deep Reinforcement Learning** [2022]  
  Ishitani, R., Kataoka, T. and Rikimaru, K.  
  [J. Chem. Inf. Model. 2022, 62, 17, 4032–4048](https://pubs.acs.org/doi/10.1021/acs.jcim.2c00366) |  [code](https://github.com/pfnet-research/RJT-RL) 

* **Accelerated rational PROTAC design via deep learning and molecular simulations** [2022]  
  Zheng, S., Tan, Y., Wang, Z. et al.  
  [Nat Mach Intell 4, 739–748 (2022)](https://www.nature.com/articles/s42256-022-00527-y) |  [code](https://github.com/biomed-AI/PROTAC-RL) 
  
* **Improving de novo molecular design with curriculum learning** [2022]  
  Guo, J., Fialková, V., Arango, J.D. et al.  
  [Nat Mach Intell 4, 555–563 (2022)](https://doi.org/10.1038/s42256-022-00494-4) |  [code](https://github.com/MolecularAI/ReinventCommunity/blob/master/notebooks/Automated_Curriculum_Learning_Demo.ipynb)  
 
* **De novo design of protein target specific scaffold-based Inhibitors via Reinforcement Learning** [2022]  
  Bontha M, McNaughton A, Knutson C, Pope J, Kumar N.  
  [arXiv:2205.10473 (2022)](https://arxiv.org/abs/2205.10473)

* **Autonomous molecule generation using reinforcement learning and docking to develop potential novel inhibitors** [2022]  
  Jeon, W., Kim, D.  
  [Sci Rep 10, 22104 (2020)](https://doi.org/10.1038/s41598-020-78537-2) |  [code](https://github.com/wsjeon92/morld) 

* **Scalable Fragment-Based 3D Molecular Design with Reinforcement Learning** [2022]  
  Flam-Shepherd, Daniel, Alexander Zhigalin, and Alán Aspuru-Guzik.  
  [arXiv:2202.00658 (2022)](https://arxiv.org/abs/2202.00658) 

* **Hit and Lead Discovery with Explorative RL and Fragment-based Molecule Generation** [2021]  
  Yang, S., Hwang, D., Lee, S., Ryu, S., & Hwang, S. J.  
  [Neural Information Processing Systems 34 (2021)](https://proceedings.neurips.cc/paper/2021/hash/41da609c519d77b29be442f8c1105647-Abstract.html) |  [code](https://github.com/AITRICS/FREED) 

* **Unlocking reinforcement learning for drug design** [2021]  
 
  [code](https://github.com/DarkMatterAI/mrl) 

* **MoleGuLAR: Molecule Generation Using Reinforcement Learning with Alternating Rewards** [2021]  
Goel, Manan, Shampa Raghunathan, Siddhartha Laghuvarapu, and U. Deva Priyakumar.  
  [J. Chem. Inf. Model. 2021, 61, 12, 5815–5826](https://pubs.acs.org/doi/10.1021/acs.jcim.1c01341) |  [code](https://github.com/devalab/MoleGuLAR) 

* **Memory-Assisted Reinforcement Learning for Diverse Molecular De Novo Design** [2020]  
Blaschke T, Engkvist O, Bajorath J, Chen H.  
  [Journal of cheminformatics 12.1 (2020)](https://jcheminf.biomedcentral.com/articles/10.1186/s13321-020-00473-0) |  [code](https://github.com/tblaschke/reinvent-memory) 

* **DeepGraphMolGen, a multi-objective, computational strategy for generating molecules with desirable properties: a graph convolution and reinforcement learning approach** [2020]  
  Khemchandani, Yash, Stephen O’Hagan, Soumitra Samanta, Neil Swainston, Timothy J. Roberts, Danushka Bollegala, and Douglas B. Kell.   
  [J Cheminform 12, 53 (2020)](https://doi.org/10.1186/s13321-020-00454-3) |  [code](https://github.com/dbkgroup/prop_gen) 

* **Reinforcement Learning for Molecular Design Guided by Quantum Mechanics** [2020]  
Simm, G., Pinsler, R. and Hernández-Lobato, J.M.,  
  [nternational Conference on Machine Learning. PMLR (2020)](http://proceedings.mlr.press/v119/simm20b/simm20b.pdf) |  [code](https://github.com/gncs/molgym) 

* **Molecular de-novo design through deep reinforcement learning** [2017]  
Olivecrona, M., Blaschke, T., Engkvist, O. et al.  
  [ J Cheminform 9, 48 (2017)](https://doi.org/10.1186/s13321-017-0235-x) |  [code](https://github.com/tblaschke/reinvent) 


### Multi-task DMGs

* **Molecular Language Model as Multi-task Generator** [2023]  
 Fang, Y., Zhang, N., Chen, Z., Fan, X. and Chen, H.    
  [arXiv:2301.11259 (2023)](https://arxiv.org/abs/2301.11259) |  [code](https://github.com/zjunlp/MolGen) 




### Monte Carlo Tree Search


* **VGAE-MCTS: a New Molecular Generative Model combining Variational Graph Auto-Encoder and Monte Carlo Tree Search** [2023]  
 Iwata, Hiroaki, Taichi Nakai, Takuto Koyama, Shigeyuki Matsumoto, Ryosuke Kojima, and Yasushi Okuno.   
  [chemrxiv-2023-q8419-v2](https://doi.org/10.26434/chemrxiv-2023-q8419-v2) 



## Text-driven molecular generation models


* **Mol-Instructions: A Large-Scale Biomolecular Instruction Dataset for Large Language Models** [2023]  
 Yin Fang, Xiaozhuan Liang, Ningyu Zhang, Kangwei Liu, Rui Huang, Zhuo Chen, Xiaohui Fan, Huajun Chen   
  [arXiv:2306.08018v1](https://arxiv.org/abs/2306.08018) |  [code](https://github.com/zjunlp/Mol-Instructions) 

* **Domain-Agnostic Molecular Generation with Self-feedback** [2023]  
 Yin Fang, Ningyu Zhang, Zhuo Chen, Xiaohui Fan, Huajun Chen   
  [arXiv:2301.11259v3](https://arxiv.org/abs/2301.11259) |  [code](https://github.com/zjunlp/MolGen) 




## Multi-Target based deep molecular generative models


* **De novo generation of dual-target ligands using adversarial training and reinforcement learning** [2021]  
 Lu, Fengqing, Mufei Li, Xiaoping Min, Chunyan Li, and Xiangxiang Zeng.    
  [Briefings in Bioinformatics 22.6 (2021)](https://doi.org/10.1093/bib/bbab333) | [code](https://github.com/lllfq/DLGN) 

* **Compound dataset and custom code for deep generative multi-target compound design** [2021]  
 Blaschke, Thomas, and Jürgen Bajorath.    
  [Future Science OA 7.6 (2021)](https://doi.org/10.2144/fsoa-2021-0033) |  [code](https://github.com/tblaschke/reinvent-multi-target) 


## Ligand-based deep molecular generative models

* **Regression Transformer enables concurrent sequence regression and generation for molecular language modeling** [2023]  
  Born, Jannis and Manica, Matteo  
  [Nat Mach Intell 5, 432–444 (2023)](https://doi.org/10.1038/s42256-023-00639-z) |  [arXiv:2202.01338v3](https://arxiv.org/abs/2202.01338) |  [code](https://github.com/IBM/regression-transformer) 

* **Domain-Agnostic Molecular Generation with Self-feedback** [2023]  
 Yin Fang, Ningyu Zhang, Zhuo Chen, Xiaohui Fan, Huajun Chen   
  [arXiv:2301.11259v3](https://arxiv.org/abs/2301.11259) |  [code](https://github.com/zjunlp/MolGen) 

* **Transformer-based molecular generative model for antiviral drug design** [2023]  
 mao, jiashun; wang, jianming; zeb, amir; Cho, Kwang-Hwi; jin, haiyan; Kim, Jongwan; Lee, Onju; Wang, Yunyun; No, Kyoung Tai.    
  [Available at SSRN 4345811 (2023)](https://dx.doi.org/10.2139/ssrn.4345811) |  [code](https://github.com/AspirinCode/TransAntivirus) 

* **Leveraging molecular structure and bioactivity with chemical language models for de novo drug design** [2023]  
  Kotsias, PC., Arús-Pous, J., Chen, H. *et al.*  
  [Nat Commun 14, 114 (2023)](https://www.nature.com/articles/s41467-022-35692-6) |  [code](https://github.com/ETHmodlab/hybridCLMs/tree/v1.0) 

* **Explore drug-like space with deep generative models** [2023]  
Wang, Jianmin, et al.   
[Methods (2023)](https://doi.org/10.1016/j.ymeth.2023.01.004) |  [code](https://github.com/AspirinCode/drug-likeness_space) 

* **De novo molecular design with deep molecular generative models for PPI inhibitors** [2022]  
  Jianmin Wang, Yanyi Chu, Jiashun Mao, Hyeon-Nae Jeon, Haiyan Jin, Amir Zeb, Yuil Jang, Kwang-Hwi Cho, Tao Song, Kyoung Tai No.  
  [Briefings in Bioinformatics 23.4 (2022)](https://doi.org/10.1093/bib/bbac285) |  [code](https://github.com/AspirinCode/iPPIGAN) 

* **DeLA-Drug: A Deep Learning Algorithm for Automated Design of Druglike Analogues** [2022]  
  Creanza, T. M., Lamanna, G., Delre, P., Contino, M., Corriero, N., Saviano, M., ... & Ancona, N.  
  [J. Chem. Inf. Model. 2022, 62, 6, 1411–1424](https://pubs.acs.org/doi/10.1021/acs.jcim.2c00205) |  [Web](http://www.ba.ic.cnr.it/softwareic/deladrugportal/) 

* **SMILES-based CharLSTM with finetuning and goal-directed generation via policy gradient** [2022]  

  [code](https://github.com/gmattedi/Smiles-LSTM) 

* **Large-scale chemical language representations capture molecular structure and properties** [2022]  
Ross, J., Belgodere, B., Chenthamarakshan, V., Padhi, I., Mroueh, Y., & Das, P.   
[Nat Mach Intell 4, 1256–1264 (2022)](https://www.nature.com/articles/s42256-022-00580-7) |  [code](https://github.com/IBM/molformer) 

* **Augmented Hill-Climb increases reinforcement learning efficiency for language-based de novo molecule generation** [2022]  
  Thomas, M., O’Boyle, N.M., Bender, A. et al.  
  [J Cheminform 14, 68 (2022)](https://doi.org/10.1186/s13321-022-00646-z) |  [code](https://github.com/MorganCThomas/SMILES-RNN) 

* **De novo molecule design with chemical language models** [2022]  
  Grisoni, F., Schneider, G.  
  [Artificial Intelligence in Drug Design (2022)](https://doi.org/10.1007/978-1-0716-1787-8_9) |  [code](https://github.com/grisoniFr/de_novo_design_RNN) 

* **Can We Quickly Learn to “Translate” Bioactive Molecules with Transformer Models?** [2022]  
  Bagal, V., Aggarwal, R., Vinod, P. K., & Priyakumar, U. D.  
  [J. Chem. Inf. Model. 2023, 63, 6, 1734–1744](https://pubs.acs.org/doi/full/10.1021/acs.jcim.2c01618) | [chemrxiv-2022-gln27](https://doi.org/10.26434/chemrxiv-2022-gln27)

* **MolGPT: Molecular Generation Using a Transformer-Decoder Model** [2022]   
  Bagal, V., Aggarwal, R., Vinod, P. K., & Priyakumar, U. D.  
  [J. Chem. Inf. Model. 2022, 62, 9, 2064–2076](https://pubs.acs.org/doi/10.1021/acs.jcim.1c00600) |  [code](https://github.com/devalab/molgpt) 

* **A Transformer-based Generative Model for De Novo Molecular Design** [2022]  
  Wang, Wenlu, et al.  
  [arXiv:2210.08749 (2022)](https://arxiv.org/abs/2210.08749) 

* **Translation between Molecules and Natural Language** [2022]  
  Edwards, C., Lai, T., Ros, K., Honke, G., & Ji, H.  
  [arXiv:2204.11817 (2022)](https://arxiv.org/abs/2204.11817) |  [code](https://github.com/blender-nlp/MolT5) 

* **Optimizing Recurrent Neural Network Architectures for De Novo Drug Design** [2021]  
  Santos, B. P., Abbasi, M., Pereira, T., Ribeiro, B., & Arrais, J. P.  
  [CBMS. IEEE, (2021)](https://ieeexplore.ieee.org/document/9474742) |  [code](https://github.com/larngroup/RNN-Drug-Generation) 

* **A recurrent neural network (RNN) that generates drug-like molecules for drug discovery** [2021]  

  [code](https://github.com/shiwentao00/Molecule-RNN) 

* **A molecule generative model used interaction fingerprint (docking pose) as constraints** [2021]  
  [code](https://github.com/jeah-z/IFP-RNN) 

* **De novo design and bioactivity prediction of SARS-CoV-2 main protease inhibitors using recurrent neural network-based transfer learning** [2021]  
  Santana, M.V.S., Silva-Jr, F.P.   
  [BMC chemistry 15.1 (2021)](https://bmcchem.biomedcentral.com/articles/10.1186/s13065-021-00737-2) |  [code](https://github.com/marcossantanaioc/De_novo_design_SARSCOV2) 

* **Generative Pre-Training from Molecules** [2021]  
  Adilov, Sanjar.  
  [J. Chem. Inf. Model. 2022, 62, 9, 2064–2076](https://pubs.acs.org/doi/10.1021/acs.jcim.1c00600) |  [code](https://github.com/sanjaradylov/smiles-gpt) 

* **Transformers for Molecular Graph Generation** [2021]  
  Cofala, Tim, and Oliver Kramer.  
  [ESANN. 2021](https://www.esann.org/sites/default/files/proceedings/2021/ES2021-112.pdf) |  [code](https://gitlab.uni-oldenburg.de/gies6280/molegent) 

* **Spatial Generation of Molecules with Transformers** [2021]  
  Cofala, Tim, and Oliver Kramer.  
  [IJCNN. IEEE, (2021)](https://ieeexplore.ieee.org/abstract/document/9533439) |  [code](https://gitlab.uni-oldenburg.de/gies6280/molegent) 

* **Generative Chemical Transformer: Neural Machine Learning of Molecular Geometric Structures from Chemical Language via Attention** [2021]  
  Hyunseung Kim, Jonggeol Na*, and Won Bo Lee*.  
  [J. Chem. Inf. Model. 2021, 61, 12, 5804–5814](https://pubs.acs.org/doi/abs/10.1021/acs.jcim.1c01289) |  [code](https://github.com/Hyunseung-Kim/molGCT) 

* **C5T5: Controllable Generation of Organic Molecules with Transformers** [2021]  
  Rothchild, D., Tamkin, A., Yu, J., Misra, U., & Gonzalez, J.  
  [arXiv:2108.10307 (2021)](https://arxiv.org/abs/2108.10307) |  [code](https://github.com/dhroth/c5t5) 

* **Molecular optimization by capturing chemist’s intuition using deep neural networks** [2021]  
  He, J., You, H., Sandström, E. et al.   
  [ J Cheminform 13, 26 (2021)](https://doi.org/10.1186/s13321-021-00497-0) |  [code](https://github.com/MolecularAI/deep-molecular-optimization) 

* **Transmol: repurposing a language model for molecular generation** [2021]  
  Grechishnikova, Daria.   
  [RSC advances 11.42 (2021)](https://pubs.rsc.org/en/content/articlelanding/2021/ra/d1ra03086h) |  [code](https://gitlab.com/cheml.io/public/transmol) 

* **Attention-based generative models for de novo molecular design** [2021]  
  Dollar, O., Joshi, N., Beck, D.A. and Pfaendtner, J.  
  [Chemical Science 12.24 (2021)](https://pubs.rsc.org/en/content/articlelanding/2021/sc/d1sc01050f) |  [code](https://github.com/oriondollar/TransVAE) 

* **Bidirectional Molecule Generation with Recurrent Neural Networks** [2020]  
  Grisoni, F., Moret, M., Lingwood, R., & Schneider, G.  
  [J. Chem. Inf. Model. 2020, 60, 3, 1175–1183](https://pubs.acs.org/doi/10.1021/acs.jcim.9b00943) |  [code](https://github.com/robinlingwood/BIMODAL) 

* **GraphAF: a Flow-based Autoregressive Model for Molecular Graph Generation** [2020]  
Shi, C., Xu, M., Zhu, Z., Zhang, W., Zhang, M., & Tang, J.   
[arXiv:2001.09382 (2020)](https://arxiv.org/abs/2001.09382) |  [code](https://github.com/DeepGraphLearning/GraphAF) 

* **Direct steering of de novo molecular generation with descriptor conditional recurrent neural networks** [2019]
  Kotsias, PC., Arús-Pous, J., Chen, H. *et al.*
  [Nat Mach Intell 2, 254–265 (2020)](https://www.nature.com/articles/s42256-020-0174-5) |  [code](https://github.com/pcko1/Deep-Drug-Coder) 

* **Generative Recurrent Networks for De Novo Drug Design** [2018]  
  Gupta, A., Müller, A. T., Huisman, B. J., Fuchs, J. A., Schneider, P., & Schneider, G.   
  [Molecular informatics 37.1-2 (2018)](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5836943/) |  [code](https://github.com/topazape/LSTM_Chem) 

* **Generative Recurrent Neural Networks for De Novo Drug Design** [2017]  
  Gupta, Anvita, et al.   
  [Molecular informatics 37.1-2 (2018)r](https://onlinelibrary.wiley.com/doi/10.1002/minf.201700111) |  [code](https://github.com/SilviaAmAm/MolBot) 

* **ChemTS: An Efficient Python Library for de novo Molecular Generation** [2017]  
  Yang, X., Zhang, J., Yoshizoe, K., Terayama, K., & Tsuda, K.  
  [Science and technology of advanced materials 18.1 (2017)](https://arxiv.org/abs/1710.00616) |  [arXiv:1710.00616v1](https://arxiv.org/abs/1710.00616) |  [code](https://github.com/tsudalab/ChemTS) 


## Pharmacophore-based deep molecular generative models

* **PGMG: A Pharmacophore-Guided Deep Learning Approach for Bioactive Molecular Generation** [2022]  
  Zhu, Huimin, Renyi Zhou, Jing Tang, and Min Li.  
  [arXiv:2207.00821 (2022)](https://arxiv.org/abs/2207.00821) |  [code](https://github.com/CSUBioGroup/PGMG) 

* **Deep generative design with 3D pharmacophoric constraints** [2021]  
  mrie, Fergus and Hadfield, Thomas E and Bradley, Anthony R and Deane, Charlotte M.  
  [Chemical science 12.43 (2021)](https://pubs.rsc.org/en/content/articlelanding/2021/SC/D1SC02436A) |  [code](https://github.com/XiaotanYu/DEVELOP-ZYH) 



## Structure-based deep molecular generative models

* **A Simple Way to Incorporate Target Structural Information in Molecular Generative Models** [2023]  
 Zhang, Wenyi, Kaiyue Zhang, and Jing Huang.   
  [Journal of Chemical Information and Modeling (2023)](https://pubs.acs.org/doi/10.1021/acs.jcim.3c00293) |  [code](https://github.com/JingHuangLab/SWIT) 

* **A Protein-Ligand Interaction-focused 3D Molecular Generative Framework for Generalizable Structure-based Drug Design** [2023]  
 Zhung W, Kim H, Kim WY.   
  [chemrxiv-2023-jsjwx](https://doi.org/10.26434/chemrxiv-2023-jsjwx) |  [code](https://github.com/ACE-KAIST/DeepICL) 

* **Mol-Zero-GAN: Zero-Shot Adaptation of Molecular Generative Adversarial Network for Specific Protein Targets** [2023]   
Ravipas Aphikulvanich*， Natapol Pornputtapong， Duangdao Wichadakul   
[chemrxiv-2023-lv2m1](https://doi.org/10.26434/chemrxiv-2023-lv2m1) |  [code](https://github.com/cucpbioinfo/Mol-Zero-GAN) 

* **Molecule Generation For Target Protein Binding with Structural Motifs** [2023]   
  Zhang, Zaixi, Yaosen Min, Shuxin Zheng, and Qi Liu.   
  [The Eleventh International Conference on Learning Representations. (2023)](https://openreview.net/forum?id=Rq13idF0F73) |  [code](https://github.com/zaixizhang/FLAG) 

* **Deep generative model for drug design from protein target sequence** [2023]   
Yangyang Chen, Zixu Wang, Lei Wang, Jianmin Wang, Pengyong Li, Dongsheng Cao, Xiangxiang Zeng, Xiucai Ye & Tetsuya Sakurai.   
[J Cheminform 15, 38 (2023)](https://doi.org/10.1186/s13321-023-00702-2) |  [code](https://github.com/viko-3/TargetGAN) 

* **3D Equivariant Diffusion for Target-Aware Molecule Generation and Affinity Prediction** [2023]   
  Guan, Jiaqi, Wesley Wei Qian, Xingang Peng, Yufeng Su, Jian Peng, and Jianzhu Ma.   
  [The Eleventh International Conference on Learning Representations. (2023)](https://openreview.net/forum?id=kJqXEPXMsE0) |  [code](https://github.com/guanjq/targetdiff) 

* **Target Specific De Novo Design of Drug Candidate Molecules with Graph Transformer-based Generative Adversarial Networks** [2023]     
  Ünlü, Atabey, Elif Çevrim, Ahmet Sarıgün, Hayriye Çelikbilek, Heval Ataş Güvenilir, Altay Koyaş, Deniz Cansen Kahraman, Ahmet Rifaioğlu, and Abdurrahman Olğaç.   
  [arXiv:2302.07868 (2023)](https://arxiv.org/abs/2302.07868)

* **Structure-based Drug Design with Equivariant Diffusion Models** [2023]  
Schneuing, A., Du, Y., Harris, C., Jamasb, A., Igashov, I., Du, W., ... & Correia, B.   
[arXiv:2210.13695 (2022)](https://openreview.net/forum?id=uKmuzIuVl8z) |  [code](https://github.com/arneschneuing/DiffSBDD) 

* **A multilevel generative framework with hierarchical self-contrasting for bias control and transparency in structure-based ligand design** [2022]   
  Chan, Lucian, Rajendra Kumar, Marcel Verdonk, and Carl Poelking.   
  [Nat Mach Intell 4, 1130–1142 (2022)](https://doi.org/10.1038/s42256-022-00564-7) |  [code](https://github.com/capoe/libpqr) 

* **Reinforced Genetic Algorithm for Structure-based Drug Design** [2022]   
  Fu, Tianfan, Wenhao Gao, Connor Coley, and Jimeng Sun.  
  [Advances in Neural Information Processing Systems 35 (2022)](https://openreview.net/forum?id=Qx6UPW0r9Lf) |  [code](https://github.com/futianfan/reinforced-genetic-algorithm) 

* **Exploiting pretrained biochemical language models for targeted drug design** [2022]  
  Uludoğan, Gökçe, Elif Ozkirimli, Kutlu O. Ulgen, Nilgün Karalı, and Arzucan Özgür.  
  [Bioinformatics 38.Supplement_2 (2022)](https://doi.org/10.1093/bioinformatics/btac482) |  [code](https://github.com/boun-tabi/biochemical-lms-for-drug-design) 

* **RELATION: A Deep Generative Model for Structure-Based De Novo Drug Design** [2022]  
  Wang, M., Hsieh, C.Y., Wang, J., Wang, D., Weng, G., Shen, C., Yao, X., Bing, Z., Li, H., Cao, D. and Hou, T.,  
  [Journal of Medicinal Chemistry 65.13 (2022)](https://pubs.acs.org/doi/10.1021/acs.jmedchem.2c00732) |  [code](https://github.com/micahwang/RELATION) 

* **Tailoring Molecules for Protein Pockets: a Transformer-based Generative Solution for Structured-based Drug Design** [2022]  
  Wu, K., Xia, Y., Fan, Y., Deng, P., Liu, H., Wu, L., ... & Liu, T. Y.  
  [arXiv:2209.06158 (2022)](https://arxiv.org/abs/2209.06158) |  [code](https://github.com/HankerWu/TamGent) 

* **De novo design of protein target specific scaffold-based Inhibitors via Reinforcement Learning** [2022]  
  Bontha M, McNaughton A, Knutson C, Pope J, Kumar N.  
  [arXiv:2205.10473 (2022)](https://openreview.net/forum?id=k-ES3OH7eqp)

* **AlphaDrug: protein target specific de novo molecular generation** [2022]  
  Qian, Hao, Cheng Lin, Dengwei Zhao, Shikui Tu, and Lei Xu.  
  [PNAS Nexus 1.4 (2022)](https://academic.oup.com/pnasnexus/article/1/4/pgac227/6751929) |  [code](https://github.com/CMACH508/AlphaDrug) 

* **LIMO: Latent Inceptionism for Targeted Molecule Generation** [2022]  
  Eckmann, Peter, Kunyang Sun, Bo Zhao, Mudong Feng, Michael K. Gilson, and Rose Yu.  
  [arXiv:2206.09010 (2022)](https://arxiv.org/abs/2206.09010) |  [code](https://github.com/rose-stl-lab/limo) 

* **Pocket2Mol: Efficient Molecular Sampling Based on 3D Protein Pockets** [2022]  
 Peng, Xingang, Shitong Luo, Jiaqi Guan, Qi Xie, Jian Peng, and Jianzhu Ma.  
  [International Conference on Machine Learning. PMLR, (2022)](https://proceedings.mlr.press/v162/peng22b.html) |  [code](https://github.com/pengxingang/Pocket2Mol) 

* **Autonomous molecule generation using reinforcement learning and docking to develop potential novel inhibitors** [2022]  
  Jeon, W., Kim, D.  
  [Sci Rep 10, 22104 (2020)](https://doi.org/10.1038/s41598-020-78537-2) |  [code](https://github.com/wsjeon92/morld) 

* **Target-Focused Library Design by Pocket-Applied Computer Vision and Fragment Deep Generative Linking** [2022]  
  Eguida, Merveille, Christel Schmitt-Valencia, Marcel Hibert, Pascal Villa, and Didier Rognan.  
  [Journal of Medicinal Chemistry 65.20 (2022)](https://pubs.acs.org/doi/10.1021/acs.jmedchem.2c00931) |  [code](https://github.com/kimeguida/POEM) 

* **Incorporating Target-Specific Pharmacophoric Information into Deep Generative Models for Fragment Elaboration** [2022]  
  Hadfield, Thomas E., Fergus Imrie, Andy Merritt, Kristian Birchall, and Charlotte M. Deane.  
  [J. Chem. Inf. Model. 2022, 62, 10, 2280–2292](https://pubs.acs.org/doi/10.1021/acs.jcim.1c01311) |  [code](https://github.com/tomhadfield95/STRIFE) 

* **Fragment-Based Ligand Generation Guided By Geometric Deep Learning On Protein-Ligand Structure** [2022]  
  Powers, Alexander S., Helen H. Yu, Patricia Suriana, and Ron O. Dror.  
  [bioRxiv (2022)](https://www.biorxiv.org/content/10.1101/2022.03.17.484653v2) 

* **Zero-Shot 3D Drug Design by Sketching and Generating** [2022]  
  Long, Siyu, Yi Zhou, Xinyu Dai, and Hao Zhou.   
  [arXiv:2209.13865 (2022)](https://nips.cc/media/neurips-2022/Slides/54457.pdf) |  [code](https://github.com/longlongman/DESERT) 

* **Structure-based de novo drug design using 3D deep generative models** [2021]  
  Li, Yibo, Jianfeng Pei, and Luhua Lai.    
  [Chemical science 12.41 (2021)](https://pubs.rsc.org/en/content/articlelanding/2021/sc/d1sc04444c)

* **Transformer neural network for protein-specific de novo drug generation as a machine translation proble** [2021]  
  Grechishnikova, Daria.   
  [Sci Rep 11, 321 (2021)](https://www.nature.com/articles/s41598-020-79682-4) |  [code](https://github.com/dariagrechishnikova/molecule_structure_generation) 

* **Structure-aware generation of drug-like molecules** [2021]  
  Drotár, P., Jamasb, A.R., Day, B., Cangea, C. and Liò, P.,   
  [arXiv:2111.04107 (2021)](https://arxiv.org/abs/2111.04107) 

* **A 3D Generative Model for Structure-Based Drug Design** [2021]  
  Luo, S., Guan, J., Ma, J., & Peng, J.   
  [Advances in Neural Information Processing Systems 34 (2021)](https://openreview.net/forum?id=yDwfVD_odRo) |  [code](https://github.com/luost26/3D-Generative-SBDD) 

* **Structure-Based de Novo Molecular Generator Combined with Artificial Intelligence and Docking Simulations** [2021]  
  Ma, B., Terayama, K., Matsumoto, S., Isaka, Y., Sasakura, Y., Iwata, H., Araki, M. and Okuno, Y.   
  [J. Chem. Inf. Model. 2021, 61, 7, 3304–3313](https://pubs.acs.org/doi/10.1021/acs.jcim.1c00679) |  [code](https://github.com/clinfo/SBMolGen) 




## Fragment-based deep molecular generative models


### Scaffold-based DMGs


* **DrugEx v3: scaffold-constrained drug design with graph transformer-based reinforcement learning** [2023]  
  Liu, X., Ye, K., van Vlijmen, H.W.T. et al.    
  [J Cheminform 15, 24 (2023)](https://doi.org/10.1186/s13321-023-00694-z) |  [code](https://github.com/CDDLeiden/DrugEx) 

* **Sc2Mol: a scaffold-based two-step molecule generator with variational autoencoder and transformer** [2023]  
  Zhirui Liao, Lei Xie, Hiroshi Mamitsuka, Shanfeng Zhu.  
  [Bioinformatics 39.1 (2023)](https://doi.org/10.1093/bioinformatics/btac814) |  [code](https://github.com/zhiruiliao/Sc2Mol) 

* **De novo design of protein target specific scaffold-based Inhibitors via Reinforcement Learning** [2022]  
  Bontha M, McNaughton A, Knutson C, Pope J, Kumar N.  
  [arXiv:2205.10473 (2022)](https://openreview.net/forum?id=k-ES3OH7eqp)

* **LibINVENT: Reaction-based Generative Scaffold Decoration for in Silico Library Design** [2022]  
  Fialková, V., Zhao, J., Papadopoulos, K., Engkvist, O., Bjerrum, E.J., Kogej, T. and Patronov, A   
  [J. Chem. Inf. Model. 2022, 62, 9, 2046–2063](https://pubs.acs.org/doi/10.1021/acs.jcim.1c00469) |  [code](https://github.com/MolecularAI/Lib-INVENT) 

* **Learning to Extend Molecular Scaffolds with Structural Motifs** [2022]  
  Maziarz, Krzysztof, Henry Jackson-Flux, Pashmina Cameron, Finton Sirockin, Nadine Schneider, Nikolaus Stiefl, Marwin Segler, and Marc Brockschmidt.   
  [arXiv:2103.03864 (2021)](https://openreview.net/forum?id=ZTsoE8G3GG)

* **Deep scaffold hopping with multimodal transformer neural networks** [2021]  
Zheng, Shuangjia, Zengrong Lei, Haitao Ai, Hongming Chen, Daiguo Deng, and Yuedong Yang.  
  [J Cheminform 13, 87 (2021)](https://doi.org/10.1186/s13321-021-00565-5) |  [code](https://github.com/prokia/deepHops) 

* **Kinase Inhibitor Scaffold Hopping with Deep Learning Approaches** [2021]  
Hu, Lizhao, Yuyao Yang, Shuangjia Zheng, Jun Xu, Ting Ran, and Hongming Chen.  
  [J. Chem. Inf. Model. 2021, 61, 10, 4900–4912](https://pubs.acs.org/doi/10.1021/acs.jcim.1c00608) |  [code](https://github.com/YuYaoYang2333/SyntaLinker) 

* **3D-Scaffold: A Deep Learning Framework to Generate 3D Coordinates of Drug-like Molecules with Desired Scaffolds** [2021]  
Joshi, Rajendra P., Niklas WA Gebauer, Mridula Bontha, Mercedeh Khazaieli, Rhema M. James, James B. Brown, and Neeraj Kumar.  
  [J. Phys. Chem. B 2021, 125, 44, 12166–12176](https://pubs.acs.org/doi/10.1021/acs.jpcb.1c06437) |  [code](https://github.com/PNNL-CompBio/3D_Scaffold) 

* **SMILES-Based Deep Generative Scaffold Decorator for De-Novo Drug Design** [2020]  
Arús-Pous, Josep, Atanas Patronov, Esben Jannik Bjerrum, Christian Tyrchan, Jean-Louis Reymond, Hongming Chen, and Ola Engkvist.  
  [J Cheminform 12, 38 (2020)](https://doi.org/10.1186/s13321-020-00441-8) | [chemrxiv.11638383.v1](https://doi.org/10.26434/chemrxiv.11638383.v1) |  [code](https://github.com/undeadpixel/reinvent-scaffold-decorator) 

* **Scaffold-based molecular design with a graph generative model** [2020]  
Lim, Jaechang, Sang-Yeon Hwang, Seokhyun Moon, Seungsu Kim, and Woo Youn Kim.  
  [Chemical science 11.4 (2020)](https://pubs.rsc.org/en/content/articlelanding/2020/SC/C9SC04503A) |  [code](https://github.com/jaechanglim/GGM) 


### Motifs-based DMGs

* **MAGNet: Motif-Agnostic Generation of Molecules from Shapes** [2023]   
  Hetzel, Leon, Johanna Sommer, Bastian Rieck, Fabian Theis, and Stephan Günnemann.   
  [arXiv:2305.19303 (2023)](https://arxiv.org/abs/2305.19303) 

* **Molecule Generation For Target Protein Binding with Structural Motifs** [2023]   
  Zhang, Zaixi, Yaosen Min, Shuxin Zheng, and Qi Liu.   
  [The Eleventh International Conference on Learning Representations. (2023)](https://openreview.net/forum?id=Rq13idF0F73) |  [code](https://github.com/zaixizhang/FLAG) 

* **De Novo Molecular Generation via Connection-aware Motif Mining** [2023]   
  Zijie Geng, Shufang Xie, Yingce Xia, Lijun Wu, Tao Qin, Jie Wang, Yongdong Zhang, Feng Wu, Tie-Yan Liu   
  [arXiv:2302.01129 (2023)](https://openreview.net/pdf?id=Q_Jexl8-qDihttps://openreview.net/pdf?id=Q_Jexl8-qDi) |  [code](https://github.com/MIRALab-USTC/AI4Sci-MiCaM) 

* **Learning to Extend Molecular Scaffolds with Structural Motifs** [2022]  
  Maziarz, Krzysztof, Henry Jackson-Flux, Pashmina Cameron, Finton Sirockin, Nadine Schneider, Nikolaus Stiefl, Marwin Segler, and Marc Brockschmidt.   
  [arXiv:2103.03864 (2021)](https://openreview.net/forum?id=ZTsoE8G3GG)

* **Hierarchical generation of molecular graphs using structural motifs** [2020]  
  Jin, Wengong, Regina Barzilay, and Tommi Jaakkola.   
  [International conference on machine learning. PMLR, (2020)](https://dl.acm.org/doi/abs/10.5555/3524938.3525387) |  [code](https://github.com/wengong-jin/hgraph2graph) 



### Fragment-based DMGs

* **Molecular Generation with Reduced Labeling through Constraint Architecture** [2023]   
 Wang, Jike, Yundian Zeng, Huiyong Sun, Junmei Wang, Xiaorui Wang, Ruofan Jin, Mingyang Wang et al.    
  [J. Chem. Inf. Model. 2023, 63, 11, 3319–3327](https://pubs.acs.org/doi/full/10.1021/acs.jcim.3c00579) |  [code](https://github.com/jkwang93/Frag-G_M) 

* **Tree-Invent: A novel molecular generative model constrained with topological tree** [2023]   
 Mingyuan Xu, HongMing Chen.   
  [chemrxiv-2023-m77vk](https://doi.org/10.26434/chemrxiv-2023-m77vk) |  [code](https://github.com/MingyuanXu/Tree-Invent) 

* **MacFrag: segmenting large-scale molecules to obtain diverse fragments with high qualities** [2023]  
  Yanyan Diao, Feng Hu, Zihao Shen, Honglin Li*.  
  [Bioinformatics (2023)](https://doi.org/10.1093/bioinformatics/btad012) |  [code](https://github.com/yydiao1025/MacFrag) 

* **Fragment-based Deep Molecular Generation using Hierarchical Chemical Graph Representation and Multi-Resolution Graph Variational Autoencoder** [2023]  
  Gao, Zhenxiang, Xinyu Wang, Blake Blumenfeld Gaines, Xuetao Shi, Jinbo Bi, and Minghu Song.  
  [Molecular Informatics (2023)](https://doi.org/10.1002/minf.202200215)

* **Fragment-based t-SMILES for de novo molecular generation** [2023]  
  Wu, Juan-Ni, Tong Wang, Yue Chen, Li-Juan Tang, Hai-Long Wu, and Ru-Qin Yu.  
  [arXiv:2301.01829 (2023)](https://arxiv.org/abs/2301.01829) |  [code](https://github.com/juanniwu/t-SMILES) 

* **Target-Focused Library Design by Pocket-Applied Computer Vision and Fragment Deep Generative Linking** [2022]  
  Eguida, Merveille, Christel Schmitt-Valencia, Marcel Hibert, Pascal Villa, and Didier Rognan.  
  [Journal of Medicinal Chemistry 65.20 (2022): 13771-13783](https://pubs.acs.org/doi/10.1021/acs.jmedchem.2c00931) |  [code](https://github.com/kimeguida/POEM) 

* **Incorporating Target-Specific Pharmacophoric Information into Deep Generative Models for Fragment Elaboration** [2022]  
  Hadfield, Thomas E., Fergus Imrie, Andy Merritt, Kristian Birchall, and Charlotte M. Deane.  
  [J. Chem. Inf. Model. 2022, 62, 10, 2280–2292](https://pubs.acs.org/doi/10.1021/acs.jcim.1c01311) |  [code](https://github.com/tomhadfield95/STRIFE) 

* **Fragment-Based Ligand Generation Guided By Geometric Deep Learning On Protein-Ligand Structure** [2022]  
  Powers, Alexander S., Helen H. Yu, Patricia Suriana, and Ron O. Dror.  
  [bioRxiv (2022)](https://www.biorxiv.org/content/10.1101/2022.03.17.484653v2) 

* **FAME: Fragment-based Conditional Molecular Generation for Phenotypic Drug Discovery** [2022]  
  Pham, Thai-Hoang, Lei Xie, and Ping Zhang.  
  [SDM. Society for Industrial and Applied Mathematics, (2022)](https://www.biorxiv.org/content/10.1101/2022.01.21.477292v1) 

* **Scalable Fragment-Based 3D Molecular Design with Reinforcement Learning** [2022]  
  Flam-Shepherd, Daniel, Alexander Zhigalin, and Alán Aspuru-Guzik.  
  [arXiv:2202.00658 (2022)](https://arxiv.org/abs/2202.00658) 

* **Hit and Lead Discovery with Explorative RL and Fragment-based Molecule Generation** [2021]  
  Yang, S., Hwang, D., Lee, S., Ryu, S., & Hwang, S. J.  
  [Advances in Neural Information Processing Systems 34 (2021)](https://arxiv.org/abs/2110.01219) |  [code](https://github.com/AITRICS/FREED) 

* **Automated Generation of Novel Fragments Using Screening Data, a Dual SMILES Autoencoder, Transfer Learning and Syntax Correction** [2021]  
  Bilsland, Alan E., Kirsten McAulay, Ryan West, Angelo Pugliese, and Justin Bower.  
  [J. Chem. Inf. Model. 2021, 61, 6, 2547–2559](https://pubs.acs.org/doi/10.1021/acs.jcim.0c01226) |  [code](https://github.com/abilsland/fragmentEncoder) 

* **A Deep Generative Model for Fragment-Based Molecule Generation** [2020]  
  Podda, Marco, Davide Bacciu, and Alessio Micheli.  
  [International Conference on Artificial Intelligence and Statistics. PMLR, (2020)](http://proceedings.mlr.press/v108/) |  [code](https://github.com/hengwei-chan/fragment-based-de-novo) 

* **Multi-Objective Molecule Generation using Interpretable Substructures** [2020]  
  Jin, Wengong, Regina Barzilay, and Tommi Jaakkola.   
  [International conference on machine learning. PMLR, (2020)](http://proceedings.mlr.press/v119/jin20b.html) |  [code](https://github.com/wengong-jin/multiobj-rationale) 

* **Fragment Graphical Variational AutoEncoding for Screening Molecules with Small Data** [2019]  
  Armitage, John, Leszek J. Spalek, Malgorzata Nguyen, Mark Nikolka, Ian E. Jacobs, Lorena Marañón, Iyad Nasrallah et al.  
  [arXiv:1910.13325 (2019)](https://arxiv.org/abs/1910.13325) |  [code](https://github.com/OE-FET/FraGVAE) 




### Linkers-based DMGs

* **Fragment Linker Prediction Using the Deep Encoder-Decoder Network for PROTACs Drug Design** [2023]  
Kao, Chien-Ting, Chieh-Te Lin, Cheng-Li Chou, and Chu-Chung Lin.   
[J. Chem. Inf. Model. 2023, 63, 10, 2918–2927](https://pubs.acs.org/doi/full/10.1021/acs.jcim.2c01287) |  [code](https://github.com/AnHorn/AIMLinker) 

* **Equivariant 3D-Conditional Diffusion Models for Molecular Linker Desig** [2023]  
Igashov, I., Stärk, H., Vignac, C., Satorras, V.G., Frossard, P., Welling, M., Bronstein, M. and Correia, B.,   
[arXiv:2210.05274 (2022)](https://arxiv.org/abs/2210.05274) |  [code](https://github.com/igashov/DiffLinker) 

* **DRlinker: Deep Reinforcement Learning for Optimization in Fragment Linking Design** [2022]  
  Tan, Y., Dai, L., Huang, W., Guo, Y., Zheng, S., Lei, J., ... & Yang, Y.  
  [J. Chem. Inf. Model. 2022, 62, 23, 5907–5917](https://pubs.acs.org/doi/10.1021/acs.jcim.2c00982) |  [code](https://github.com/biomed-AI/DRlinker) 

* **3DLinker: An E(3) Equivariant Variational Autoencoder for Molecular Linker Design** [2022]  
  Huang, Yinan, Xingang Peng, Jianzhu Ma, and Muhan Zhang.   
  [arXiv:2205.07309 (2022)](https://arxiv.org/abs/2205.07309) |  [code](https://github.com/GraphPKU/3DLinker) 

* **SyntaLinker-Hybrid: A deep learning approach for target specific drug design** [2022]  
  Feng, Yu, Yuyao Yang, Wenbin Deng, Hongming Chen, and Ting Ran.    
  [Artificial Intelligence in the Life Sciences 2 (2022)](https://doi.org/10.1016/j.ailsci.2022.100035) 

* **Deep Generative Models for 3D Linker Design** [2020]  
  Imrie, Fergus, Anthony R. Bradley, Mihaela van der Schaar, and Charlotte M. Deane.  
  [J. Chem. Inf. Model. 2020, 60, 4, 1983–1995](https://pubs.acs.org/doi/10.1021/acs.jcim.9b01120) |  [code](https://github.com/fimrie/DeLinker) 

* **SyntaLinker: automatic fragment linking with deep conditional transformer neural networks** [2020]  
  Yang, Yuyao, Shuangjia Zheng, Shimin Su, Chao Zhao, Jun Xu, and Hongming Chen.  
  [Chemical science 11.31 (2020)](https://pubs.rsc.org/en/content/articlelanding/2020/sc/d0sc03126g) |  [code](https://github.com/YuYaoYang2333/SyntaLinker) 


## Chemical Reaction-based deep molecular generative models

* **Uni-RXN: A Unified Framework Bridging the Gap between Chemical Reaction Pretraining and Conditional Molecule Generation** [2023]   
Bo Qiang, Yiran Zhou, Yuheng Ding, Ningfeng Liu, Song Song, Liangren Zhang, Bo Huang, Zhenming Liu   
[arXiv:2303.06965 (2023)](https://arxiv.org/abs/2303.06965) |  [code](https://github.com/qiangbo1222/Uni-RXN-official) 

* **Molecular Generative Model via Retrosynthetically Prepared Chemical Building Block Assembly** [2023]   
Seo, Seonghwan, Jaechang Lim, and Woo Youn Kim.   
[Advanced Science (2023)](https://doi.org/10.1002/advs.202206674) |  [code](https://github.com/SeonghwanSeo/BBAR) 

* **Synthesis-Aware Generation of Structural Analogues** [2022]   
  Dolfus, Uschi, Hans Briem, and Matthias Rarey.   
  [J. Chem. Inf. Model. 2022, 62, 15, 3565–3576](https://pubs.acs.org/doi/10.1021/acs.jcim.2c00246) |  [code](https://github.com/whitead/synspace) 

* **ChemistGA: A Chemical Synthesizable Accessible Molecular Generation Algorithm for Real-World Drug Discovery** [2022]   
  Wang, Jike, Xiaorui Wang, Huiyong Sun, Mingyang Wang, Yundian Zeng, Dejun Jiang, Zhenxing Wu et al.   
[Journal of Medicinal Chemistry 65.18 (2022)](https://pubs.acs.org/doi/10.1021/acs.jmedchem.2c01179) |  [code](https://github.com/jkwang93/ChemistGA) 

* **Generating reaction trees with cascaded variational autoencoders** [2022]   
  Nguyen, Dai Hai, and Koji Tsuda.   
  [The Journal of Chemical Physics 156.4 (2022)](https://doi.org/10.1063/5.0076749) |  [code](https://github.com/haidnguyen0909/rxngenerator) 

* **Synthesis-Aware Generation of Structural Analogues** [2022]   
  Dolfus, Uschi, Hans Briem, and Matthias Rarey.   
  [J. Chem. Inf. Model. 2022, 62, 15, 3565–3576](https://pubs.acs.org/doi/10.1021/acs.jcim.2c00246)

* **SynthI: A New Open-Source Tool for Synthon-Based Library Design** [2022]   
  Zabolotna, Yuliana, Dmitriy M. Volochnyuk, Sergey V. Ryabukhin, Kostiantyn Gavrylenko, Dragos Horvath, Olga Klimchuk, Oleksandr Oksiuta, Gilles Marcou, and Alexandre Varnek.   
  [J. Chem. Inf. Model. 2022, 62, 9, 2151–2163](https://pubs.acs.org/doi/10.1021/acs.jcim.1c00754) |  [code](https://github.com/Laboratoire-de-Chemoinformatique/Synt-On) 

* **Integrating Synthetic Accessibility with AI-based Generative Drug Design** [2021]   
  Parrot, Maud, Hamza Tajmouati, Vinicius Barros Ribeiro da Silva, Brian Atwood, Robin Fourcade, Yann Gaston-Mathé, Nicolas Do Huu, and Quentin Perron.   
  [chemrxiv-2021-jkhzw-v2](https://doi.org/10.26434/chemrxiv-2021-jkhzw-v2) |  [code](https://github.com/iktos/generation-under-synthetic-constraint) 

## Omics-based deep molecular generative models

* **Gex2SGen: Designing Drug-like Molecules from Desired Gene Expression Signatures** [2023]   
 Das, Dibyajyoti, Broto Chakrabarty, Rajgopal Srinivasan, and Arijit Roy.   
  [J. Chem. Inf. Model. 2023, 63, 7, 1882–1893](https://doi.org/10.1021/acs.jcim.2c01301)

* **PaccMannRL: De novo generation of hit-like anticancer molecules from transcriptomic data via reinforcement learning** [2021]  
  Born, Jannis, Matteo Manica, Ali Oskooei, Joris Cadow, Greta Markert, and María Rodríguez Martínez.   
  [Iscience 24.4 (2021)](https://doi.org/10.1016/j.isci.2021.102269) |  [code](https://github.com/PaccMann/paccmann_omics) 

* **Molecular Generation for Desired Transcriptome Changes With Adversarial Autoencoders** [2020]  
  Shayakhmetov, Rim, Maksim Kuznetsov, Alexander Zhebrak, Artur Kadurin, Sergey Nikolenko, Alexander Aliper, and Daniil Polykovskiy.   
  [Frontiers in Pharmacology (2020)](https://doi.org/10.3389/fphar.2020.00269) |  [code](https://github.com/insilicomedicine/BiAAE) 

* **De novo generation of hit-like molecules from gene expression signatures using artificial intelligence** [2020]  
  Méndez-Lucio, Oscar, Benoit Baillif, Djork-Arné Clevert, David Rouquié, and Joerg Wichard.   
  [Nat Commun 11, 10 (2020)](https://doi.org/10.1038/s41467-019-13807-w) 


## Multi-Objective deep molecular generative models

* **MolSearch: Search-based Multi-objective Molecular Generation and Property Optimization** [2022]   
  Sun, Mengying, Jing Xing, Han Meng, Huijun Wang, Bin Chen, and Jiayu Zhou.   
  [KDD '2022](https://dl.acm.org/doi/abs/10.1145/3534678.3542676) |  [code](https://github.com/illidanlab/MolSearch) 

* **MGCVAE: Multi-Objective Inverse Design via Molecular Graph Conditional Variational Autoencoder** [2022]   
  Lee, Myeonghun, and Kyoungmin Min.   
  [J. Chem. Inf. Model. 2022, 62, 12, 2943–2950](https://pubs.acs.org/doi/10.1021/acs.jcim.2c00487) |  [code](https://github.com/mhlee216/MGCVAE) 

* **Multi-Objective Molecule Generation using Interpretable Substructures** [2020]  
  Jin, Wengong, Regina Barzilay, and Tommi Jaakkola.   
  [ICML (2020)](http://proceedings.mlr.press/v119/jin20b.html) |  [code](https://github.com/wengong-jin/multiobj-rationale) 

* **DeepGraphMolGen, a multi-objective, computational strategy for generating molecules with desirable properties: a graph convolution and reinforcement learning approach** [2020]  
  Khemchandani, Yash, Stephen O’Hagan, Soumitra Samanta, Neil Swainston, Timothy J. Roberts, Danushka Bollegala, and Douglas B. Kell.   
  [ J Cheminform 12, 53 (2020)](https://doi.org/10.1186/s13321-020-00454-3) |  [code](https://github.com/dbkgroup/prop_gen) 

* **Multi-objective de novo drug design with conditional graph generative model** [2018]  
  Li, Yibo, Liangren Zhang, and Zhenming Liu.   
  [J Cheminform 10, 33 (2018)](https://doi.org/10.1186/s13321-018-0287-6) |  [code](https://github.com/kevinid/molecule_generator) 


## Quantum deep molecular generative models


* **Quantum Generative Models for Small Molecule Drug Discovery** [2021]  
  Li, Junde, Rasit O. Topaloglu, and Swaroop Ghosh.   
  [IEEE Transactions on Quantum Engineering (2021)](https://ieeexplore.ieee.org/document/9520764) |  [code](https://github.com/jundeli/quantum-gan) 




