[![License: GPL](https://img.shields.io/badge/License-GPL-yellow)](https://github.com/AspirinCode/papers-for-molecular-design-using-DL)
[![contributing](https://img.shields.io/badge/contributions-welcome-brightgreen)](https://github.com/AspirinCode/papers-for-molecular-design-using-DL)

# List of molecular design (molecular conformation generation) using Generative AI and Deep Learning 


related to  **Generative AI** and **Deep Learning** for  **molecular/drug design** and **molecular conformation generation**. 

![Molecular GenerativeAI](https://github.com/AspirinCode/papers-for-molecular-design-using-DL/blob/main/figures/df4md.png)


**Updating ...**  

## Molecular Optimization
[Molecular Optimization](https://github.com/AspirinCode/papers-for-molecular-design-using-DL/blob/main/Molecular_Optimization.md) will welcome ！！！


## Menu

| Menu | Menu | Menu | Menu |
| ------ | :---------- | ------ | ------ |
| [ Generative AI for Scientific Discovery](#generative-ai-for-scientific-discovery) | [ Reviews](#reviews) | [ Datasets and Benchmarks](#datasets-and-benchmarks) | [ Drug-likeness and Evaluation metrics](#drug-likeness-and-evaluation-metrics) |
| [Deep Learning-based design](#deep-learning-based-design) | [Text-driven molecular generation models](#text-driven-molecular-generation-models) | [Multi-Target based deep molecular generative models](#multi-Target-based-deep-molecular-generative-models) | [Ligand-based deep molecular generative models](#ligand-based-deep-molecular-generative-models) |
| [Pharmacophore-based deep molecular generative models](#pharmacophore-based-deep-molecular-generative-models) | [Structure-based deep molecular generative models](#structure-based-deep-molecular-generative-models) | [Fragment-based deep molecular generative models](#fragment-based-deep-molecular-generative-models) | [Scaffold-based DMGs](#scaffold-based-dmgs) |
| [Fragment-based DMGs](#fragment-based-dmgs) | [Motifs-based DMGs](#motifs-based-dmgs) | [Linkers-based DMGs](#linkers-based-dmgs) | [Chemical Reaction-based deep molecular generative models](#chemical-reaction-based-deep-molecular-generative-models) |
| [Omics-based deep molecular generative models](#omics-based-deep-molecular-generative-models) | [Multi-Objective deep molecular generative models](#multi-objective-deep-molecular-generative-models) | [Quantum deep molecular generative models](#quantum-deep-molecular-generative-models) | [Recommendations and References](#recommendations-and-references) |
| [Spectra(Mass/NMR)-based](#spectra-based) | [Mass Spectra-based](#mass-spectra-based) | [NMR Spectra-based](#nmr-spectra-based) | [Cryo-EM Maps-based](#cryo-em-maps-based) |

  - [ Drug-likeness and Evaluation metrics](#drug-likeness-and-evaluation-metrics)

| Datasets |  Benchmarks | Drug-likeness| Evaluation metrics  |
| ------ | :---------- | ------ | ------ |
| [Datasets](#datasets) | [Benchmarks](#benchmarks) | [QED](#qed) | [SAscore](#sascore) |
|  |  | [QEPPI](#qeppi) | [RAscore](#rascore) |
|  |  |  | [Evaluation metrics](#evaluation-metrics) |
|  |  |  | [Molecular generative validation](#molecular-generative-validation) |


  - [Generative AI for Molecular Conformation](#generative-ai-for-molecular-conformation)


| Menu | Menu | 
| ------ | ------ | 
| [Benchmark for Molecular Conformer Ensembles](#benchmark-for-molecular-conformer-ensembles) | [Reviews for Molecular Conformation Generation](#reviews-for-molecular-conformation-generation) | 
| [VAE-based Molecular Conformation Generation](#vae-based-molecular-conformation-generation) | [GAN-based Molecular Conformation Generation](#gan-based-molecular-conformation-generation) | 
|  [Energy-based Molecular Conformation Generation](#energy-based-molecular-conformation-generation) |   | 
|  [Diffusion-based Molecular Conformation Generation](#diffusion-based-molecular-conformation-generation) |   | 
|  [RL-based Molecular Conformation Generation](#rl-based-molecular-conformation-generation) |   | 
|  [GNN-based Molecular Conformation Generation](#gnn-based-molecular-conformation-generation) |   | 


  - [Deep Learning-based design](#deep-learning-based-design)

| Menu | Menu | Menu | Menu |
| ------ | :---------- | ------ | ------ |
| [RNN-based](#rnn-based) | [LSTM-based](#lstm-based) | [Autoregressive-models](#autoregressive-models) | [Transformer-based](#transformer-based) |
| [VAE-based](#vae-based) | [GAN-based](#gan-based) | [Flow-based](#flow-based) | [ Prompt-based](#prompt-based)|
| [Score-Based](#score-Based) | [Energy-based](#energy-based) | [Diffusion-based](#diffusion-based) | [Active Learning DMGs](#active-learning-dmgs) |
| [RL-based](#rl-based) | [Multi-task DMGs](#multi-task-dmgs) | [Monte Carlo Tree Search](#monte-carlo-tree-search) | [Genetic Algorithm-based](#genetic-algorithm-based) |
| [Evolutionary Algorithm-based](#evolutionary-algorithm-based) | [Large Language Model-based](#large-language-model-based)  |  |  |


## Recommendations and References

 **awesome-AI4ProteinConformation-MD**

 https://github.com/AspirinCode/awesome-AI4ProteinConformation-MD


**Large Language Model for Biomedical Science, Molecule, Protein, Material Discovery**

https://github.com/HHW-zhou/LLM4Mol


**List of papers about Proteins Design using Deep Learning**

https://github.com/Peldom/papers_for_protein_design_using_DL

**Awesome Generative AI**

https://github.com/steven2358/awesome-generative-ai

**awesome-molecular-generation**

https://github.com/amorehead/awesome-molecular-generation

**A Survey of Artificial Intelligence in Drug Discovery**  

https://github.com/dengjianyuan/Survey_AI_Drug_Discovery

**Geometry Deep Learning for Drug Discovery and Life Science**  

https://github.com/3146830058/Geometry-Deep-Learning-for-Drug-Discovery-and-Life-Science



## Generative AI for Scientific Discovery


* **Accelerating Material Design with the Generative Toolkit for Scientific Discovery**  
Manica, Matteo and Cadow, Joris and Christofidellis, Dimitrios and Dave, Ashish and Born, Jannis and Clarke, Dean and Teukam, Yves Gaetan Nana and Hoffman, Samuel C and Buchan, Matthew and Chenthamarakshan, Vijil and others  
[npj Comput Mater 9, 69 (2023)](https://www.nature.com/articles/s41524-023-01028-1) |  [code](https://github.com/GT4SD/gt4sd-core) 




## Reviews

* **Deep Lead Optimization: Leveraging Generative AI for Structural Modification** [2024]  
 Zhang, Odin, Haitao Lin, Hui Zhang, Huifeng Zhao, Yufei Huang, Yuansheng Huang, Dejun Jiang, Chang-yu Hsieh, Peichen Pan, and Tingjun Hou.   
[arXiv:2404.19230 (2024)](https://arxiv.org/abs/2404.19230)  

* **Unlocking the Potential of Generative Artificial Intelligence in Drug Discovery** [2024]   
 Romanelli, Virgilio, Carmen Cerchia, and Antonio Lavecchia.   
  [Applications of Generative AI (2024)](https://link.springer.com/chapter/10.1007/978-3-031-46238-2_3)  

* **Recent Advances in Automated Structure-Based De Novo Drug Design** [2024]   
 Bai, Qifeng, Jian Ma, and Tingyang Xu.   
  [J. Chem. Inf. Model. (2024)](https://doi.org/10.1021/acs.jcim.4c00247)  

* **AI Deep Learning Generative Models for Drug Discovery** [2024]   
 Bai, Qifeng, Jian Ma, and Tingyang Xu.   
  [Applications of Generative AI. Cham: Springer International Publishing (2024)](https://link.springer.com/chapter/10.1007/978-3-031-46238-2_23)  

* **Deep Generative Models in De Novo Drug Molecule Generation** [2024]   
 Xiangru Tang, Howard Dai, Elizabeth Knight, Fang Wu, Yunyang Li, Tianxiao Li, Mark Gerstein   
  [arXiv:2402.08703 (2024)](https://arxiv.org/abs/2402.08703) |  [code](https://github.com/gersteinlab/GenAI4Drug)  

* **Deep Generative Models in De Novo Drug Molecule Generation** [2023]   
 Chao Pang, Jianbo Qiao, Xiangxiang Zeng, Quan Zou, and Leyi Wei*   
  [J. Chem. Inf. Model. (2023)](https://doi.org/10.1021/acs.jcim.3c01496) 

* **The Hitchhiker’s Guide to Deep Learning Driven Generative Chemistry** [2023]   
 Yan Ivanenkov, Bogdan Zagribelnyy, Alex Malyshev, Sergei Evteev, Victor Terentiev, Petrina Kamya, Dmitry Bezrukov, Alex Aliper, Feng Ren, and Alex Zhavoronkov   
  [ACS Med. Chem. Lett. (2023)](https://doi.org/10.1021/acsmedchemlett.3c00041) 

* **Quantum computing for near-term applications in generative chemistry and drug discovery** [2023]   
 Pyrkov, Alexey, Alex Aliper, Dmitry Bezrukov, Yen-Chu Lin, Daniil Polykovskiy, Petrina Kamya, Feng Ren, and Alex Zhavoronkov.   
  [Drug Discovery Today (2023)](https://doi.org/10.1016/j.drudis.2023.103675) 

* **A Systematic Survey in Geometric Deep Learning for Structure-based Drug Design**[2023]   
 Zaixi Zhang, Jiaxian Yan, Qi Liu, Enhong Chen  
[arXiv:2306.11768v2](https://arxiv.org/abs/2306.11768)

* **How will generative AI disrupt data science in drug discovery?**[2023]   
 Vert, JP.  
[Nat Biotechnol (2023)](https://doi.org/10.1038/s41587-023-01789-6)

* **Generative Models as an Emerging Paradigm in the Chemical Sciences**[2023]   
 Anstine, Dylan M., and Olexandr Isayev.  
[JACS (2023)](https://pubs.acs.org/doi/10.1021/jacs.2c13467)

* **Chemical language models for de novo drug design: Challenges and opportunities**[2023]  
Grisoni, Francesca.  
[Current Opinion in Structural Biology 79 (2023)](https://doi.org/10.1016/j.sbi.2023.102527)

* **Artificial intelligence in multi-objective drug design**[2023]   
 Luukkonen, Sohvi, Helle W. van den Maagdenberg, Michael TM Emmerich, and Gerard JP van Westen.  
[Current Opinion in Structural Biology 79 (2023)](https://doi.org/10.1016/j.sbi.2023.102537)

* **Integrating structure-based approaches in generative molecular design**[2023]   
 Thomas, Morgan, Andreas Bender, and Chris de Graaf.  
[Current Opinion in Structural Biology 79 (2023)](https://doi.org/10.1016/j.sbi.2023.102559)

* **Open data and algorithms for open science in AI-driven molecular informatics**[2023]  
 Brinkhaus, Henning Otto, Kohulan Rajan, Jonas Schaub, Achim Zielesny, and Christoph Steinbeck.  
[Current Opinion in Structural Biology 79 (2023)](https://doi.org/10.1016/j.sbi.2023.102542)

* **Structure-based drug design with geometric deep learning**[2023]  
  Isert, Clemens, Kenneth Atz, and Gisbert Schneider.  
[Current Opinion in Structural Biology 79 (2023)](https://doi.org/10.1016/j.sbi.2023.102548)

* **MolGenSurvey: A Systematic Survey in Machine Learning Models for Molecule Design**[2022]  
  Du, Yuanqi, Tianfan Fu, Jimeng Sun, and Shengchao Liu.  
[arXiv:2203.14500 (2022)](https://arxiv.org/abs/2203.14500)

* **Deep generative molecular design reshapes drug discovery**[2022]  
  Zeng, Xiangxiang, Fei Wang, Yuan Luo, Seung-gu Kang, Jian Tang, Felice C. Lightstone, Evandro F. Fang, Wendy Cornell, Ruth Nussinov, and Feixiong Cheng.  
[Cell Reports Medicine (2022)](https://doi.org/10.1016/j.xcrm.2022.100794)

* **Structure-based drug discovery with deep learning**[2022]  
  Özçelik, Rıza, Derek van Tilborg, José Jiménez-Luna, and Francesca Grisoni.  
[ChemBioChem (2022)](https://doi.org/10.1002/cbic.202200776)

* **Generative models for molecular discovery: Recent advances and challenges**[2022]  
 Bilodeau, Camille, Wengong Jin, Tommi Jaakkola, Regina Barzilay, and Klavs F. Jensen.  
[Computational Molecular Science 12.5 (2022)](https://doi.org/10.1002/wcms.1608)

* **Assessing Deep Generative Models in Chemical Composition Space**[2022]  
 Türk, Hanna, Elisabetta Landini, Christian Kunkel, Johannes T. Margraf, and Karsten Reuter.  
[Chemistry of Materials 34.21 (2022)](https://doi.org/10.1021/acs.chemmater.2c01860)

* **Generative machine learning for de novo drug discovery: A systematic review**[2022]  
 Martinelli, Dominic.   
[Computers in Biology and Medicine 145 (2022)](https://doi.org/10.1016/j.compbiomed.2022.105403)

* **Docking-based generative approaches in the search for new drug candidates**[2022]  
 Danel, Tomasz, Jan Łęski, Sabina Podlewska, and Igor T. Podolak.   
[Drug Discovery Today (2022)](https://doi.org/10.1016/j.drudis.2022.103439)

* **Advances and Challenges in De Novo Drug Design Using Three-Dimensional Deep Generative Models**[2022]  
  Xie, Weixin, Fanhao Wang, Yibo Li, Luhua Lai, and Jianfeng Pei.   
[J. Chem. Inf. Model. 2022, 62, 10, 2269–2279](https://pubs.acs.org/doi/10.1021/acs.jcim.2c00042)

* **Deep learning to catalyze inverse molecular design**[2022]  
  Alshehri, Abdulelah S., and Fengqi You.   
[Chemical Engineering Journal 444 (2022)](https://doi.org/10.1016/j.cej.2022.136669)

* **AI in 3D compound design**[2022]  
  Hadfield, Thomas E., and Charlotte M. Deane.   
[Current Opinion in Structural Biology 73 (2022)](https://doi.org/10.1016/j.sbi.2021.102326)

* **Deep learning approaches for de novo drug design: An overview**[2021]  
 Wang, Mingyang, Zhe Wang, Huiyong Sun, Jike Wang, Chao Shen, Gaoqi Weng, Xin Chai, Honglin Li, Dongsheng Cao, and Tingjun Hou.  
[Current Opinion in Structural Biology 72 (2022)](https://doi.org/10.1016/j.sbi.2021.10.001)

* **Generative chemistry: drug discovery with deep learning generative models**[2021]  
 Bian, Yuemin, and Xiang-Qun Xie.  
[Journal of Molecular Modeling 27 (2021)](https://link.springer.com/article/10.1007/s00894-021-04674-8)

* **Generative Deep Learning for Targeted Compound Design**[2021]  
 Sousa, Tiago, João Correia, Vítor Pereira, and Miguel Rocha.  
[J. Chem. Inf. Model. 2021, 61, 11, 5343–5361](https://pubs.acs.org/doi/10.1021/acs.jcim.0c01496)

* **Generative Models for De Novo Drug Design**[2021]  
 Tong, Xiaochu, Xiaohong Liu, Xiaoqin Tan, Xutong Li, Jiaxin Jiang, Zhaoping Xiong, Tingyang Xu, Hualiang Jiang, Nan Qiao, and Mingyue Zheng.  
[Journal of Medicinal Chemistry 64.19 (2021)](https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00927)

* **Molecular design in drug discovery: a comprehensive review of deep generative models**[2021]  
 Cheng, Yu, Yongshun Gong, Yuansheng Liu, Bosheng Song, and Quan Zou.   
[Briefings in bioinformatics 22.6 (2021)](https://doi.org/10.1093/bib/bbab344)

* **De novo molecular design and generative models**[2021]  
 Meyers, Joshua, Benedek Fabian, and Nathan Brown.  
[Drug Discovery Today 26.11 (2021)](https://doi.org/10.1016/j.drudis.2021.05.019)

* **Deep learning for molecular design—a review of the state of the art**[2019]  
 Elton, Daniel C., Zois Boukouvalas, Mark D. Fuge, and Peter W. Chung.  
[Molecular Systems Design & Engineering 4.4 (2019)](https://pubs.rsc.org/en/content/articlelanding/2019/me/c9me00039a)

* **Inverse molecular design using machine learning: Generative models for matter engineering**[2018]  
 Sanchez-Lengeling, Benjamin, and Alán Aspuru-Guzik.  
[Science 361.6400 (2018)](https://www.science.org/doi/10.1126/science.aat2663)

## Datasets and Benchmarks

### Datasets

[**DrugBank**](https://go.drugbank.com/)

[**ZINC 15**](https://zinc15.docking.org/)  

[**ZINC 20**](https://zinc20.docking.org/)

[**PubChem**](https://pubchem.ncbi.nlm.nih.gov/)

[**ChEMBL** ](https://www.ebi.ac.uk/chembl/)
  
[**GDB Databases**](https://gdb.unibe.ch/downloads/)

[**ChemSpider**](http://www.chemspider.com/)

[**QM Dataset**](http://quantum-machine.org/datasets/)

[**COCONUT** | Collection of Open Natural Products database](https://coconut.naturalproducts.net/)  

**MolData**  
A Molecular Benchmark for Disease and Target Based Machine Learning  
https://github.com/LumosBio/MolData  
  
  
* **Machine Learning Methods for Small Data Challenges in Molecular Science** [2023]   
 Bozheng Dou, Zailiang Zhu, Ekaterina Merkurjev, Lu Ke, Long Chen, Jian Jiang, Yueying Zhu, Jie Liu, Bengong Zhang, and Guo-Wei Wei   
  [Chem. Rev  (2023)](https://doi.org/10.1021/acs.chemrev.3c00189) 




### Benchmarks



* **Benchmarking Study of Deep Generative Models for Inverse Polymer Design** [2024]  
Yue T, Tao L, Varshney V, Li Y.  
[chemrxiv-2024-gzq4r (2024)](https://doi.org/10.26434/chemrxiv-2024-gzq4r)   

* **RediscMol: Benchmarking Molecular Generation Models in Biological Properties** [2024]  
Weng, Gaoqi, Huifeng Zhao, Dou Nie, Haotian Zhang, Liwei Liu, Tingjun Hou, and Yu Kang.  
[J. Med. Chem. 2024](https://pubs.acs.org/doi/10.1021/acs.jcim.2c01355) |  [code](https://github.com/gaoqiweng/RediscMol) 

* **Generative Models Should at Least Be Able to Design Molecules That Dock Well: A New Benchmark** [2023]  
Ciepliński, Tobiasz, Tomasz Danel, Sabina Podlewska, and Stanisław Jastrzȩbski.  
[J. Chem. Inf. Model. 2023, 63, 11, 3238–3247](https://pubs.acs.org/doi/10.1021/acs.jcim.2c01355) |  [code](https://github.com/cieplinski-tobiasz/smina-docking-benchmark) 

* **Tartarus: A Benchmarking Platform for Realistic And Practical Inverse Molecular Design** [2022]  
Nigam, AkshatKumar, Robert Pollice, Gary Tom, Kjell Jorner, Luca A.  
[arXiv:2209.12487v1](https://arxiv.org/abs/2209.12487) |  [code](https://github.com/aspuru-guzik-group/Tartarus) 


* **Molecular Sets (MOSES): A benchmarking platform for molecular generation models**  [2020]  
Polykovskiy, Daniil, Alexander Zhebrak, Benjamin Sanchez-Lengeling, Sergey Golovanov, Oktai Tatanov, Stanislav Belyaev, Rauf Kurbanov et al.  
  [Frontiers in pharmacology 11 (2020)](https://doi.org/10.3389/fphar.2020.565644) |  [code](https://github.com/molecularsets/moses)   


* **GuacaMol: Benchmarking Models for de Novo Molecular Design**  [2019]  
Brown, Nathan, Marco Fiscato, Marwin HS Segler, and Alain C. Vaucher.  
  [J. Chem. Inf. Model. 2019, 59, 3, 1096–1108](https://pubs.acs.org/doi/10.1021/acs.jcim.8b00839) |  [code](https://github.com/BenevolentAI/guacamol)   



## Drug-likeness and Evaluation metrics

**Drug-likeness** may be defined as a complex balance of various molecular properties and structure features which determine whether particular molecule is similar to the known drugs. These properties, mainly hydrophobicity, electronic distribution, hydrogen bonding characteristics, molecule size and flexibility and of course presence of various pharmacophoric features influence the behavior of molecule in a living organism, including bioavailability, transport properties, affinity to proteins, reactivity, toxicity, metabolic stability and many others.

https://github.com/AspirinCode/DrugAI_Drug-Likeness

### QED
**quantitative estimation of drug-likeness**  

* **Quantifying the chemical beauty of drugs**  [2012]  
  Bickerton, G., Paolini, G., Besnard, J. et al.    
[Nature Chem 4, 90–98 (2012)](https://doi.org/10.1038/nchem.1243) |  [code](https://github.com/AspirinCode/DrugAI_Drug-Likeness) 


### QEPPI
**quantitative estimate of protein-protein interaction targeting drug-likeness**  

* **Quantitative Estimate Index for Early-Stage Screening of Compounds Targeting Protein-Protein Interactions**  [2021]  
  Kosugi, Takatsugu, and Masahito Ohue.    
[International Journal of Molecular Sciences 22.20 (2021)](https://doi.org/10.3390/ijms222010925) |  [code](https://github.com/ohuelab/QEPPI) 

* **Quantitative Estimate of Protein-Protein Interaction Targeting Drug-likeness**  [2021]  
  Kosugi, Takatsugi, and Masahito Ohue.    
[CIBCB. IEEE, (2021)](https://ieeexplore.ieee.org/abstract/document/9562931) |  [code](https://github.com/ohuelab/QEPPI) 


### SAscore
**Estimation of synthetic accessibility score of drug-like molecules based on molecular complexity and fragment contributions**  
[J Cheminform 1, 8 (2009)](http://www.jcheminf.com/content/1/1/8) |  [code](https://github.com/rdkit/rdkit/tree/master/Contrib/SA_Score) 

### RAscore
**Retrosynthetic accessibility score (RAscore) – rapid machine learned synthesizability classification from AI driven retrosynthetic planning**  
[Chemical Science 12.9 (2021)](https://pubs.rsc.org/en/content/articlelanding/2021/sc/d0sc05401a) |  [code](https://github.com/reymond-group/RAscore) 

### Evaluation metrics



* **Spacial Score – A Comprehensive Topological Indicator for Small Molecule Complexity** [2023]   
Krzyzanowski, Adrian, Axel Pahl, Michael Grigalunas, and Herbert Waldmann.   
[J. Med. Chem. (2023)](https://doi.org/10.1021/acs.jmedchem.3c00689) | [chemrxiv-2023-nd1ll](https://chemrxiv.org/engage/chemrxiv/article-details/64257af562fecd2a83add9c2) |  [code](https://github.com/frog2000/Spacial-Score) 

* **An automated scoring function to facilitate and standardize evaluation of goal-directed generative models for de novo molecular design** [2023]   
Thomas, Morgan, Noel M. O'Boyle, Andreas Bender, and Chris De Graaf.   
[chemrxiv-2023-c4867](https://doi.org/10.26434/chemrxiv-2023-c4867) |  [code](https://github.com/MorganCThomas/MolScore) 

* **FCD : Fréchet ChemNet Distance**  
  Fréchet ChemNet Distance: A Metric for Generative Models for Molecules in Drug Discovery
  Preuer, Kristina, Philipp Renz, Thomas Unterthiner, Sepp Hochreiter, and Gunter Klambauer.  
[J. Chem. Inf. Model. 2018, 58, 9, 1736–1741](https://pubs.acs.org/doi/10.1021/acs.jcim.8b00234) |  [code](https://github.com/bioinf-jku/FCD)

* **Perplexity-Based Molecule Ranking and Bias Estimation of Chemical Language Models** [2022]  
  Moret, M., Grisoni, F., Katzberger, P. and Schneider, G.  
[J. Chem. Inf. Model. 2022, 62, 5, 1199–1206](https://pubs.acs.org/doi/10.1021/acs.jcim.2c00079) |  [code](https://github.com/ETHmodlab/CLM_perplexity) 



## Molecular generative validation



* **On the difficulty of validating molecular generative models realistically: a case study on public and proprietary data** [2023]   
Handa, K., Thomas, M.C., Kageyama, M. et al.   
[J Cheminform 15, 112 (2023)](https://doi.org/10.1186/s13321-023-00781-1) 




## Generative AI for Molecular Conformation





### Reviews for Molecular Conformation Generation




* **Prediction of Molecular Conformation Using Deep Generative Neural Networks** [2023]   
Xu, Congsheng, Yi Lu, Xiaomei Deng, and Peiyuan Yu.   
[Chinese Journal of Chemistry(2023)](https://doi.org/10.1002/cjoc.202300269)





### Benchmark for Molecular Conformer Ensembles



* **Learning Over Molecular Conformer Ensembles: Datasets and Benchmarks** [2023]   
Zhu, Yanqiao, Jeehyun Hwang, Keir Adams, Zhen Liu, Bozhao Nan, Brock Stenfors, Yuanqi Du et al.   
[NeurIPS 2023 AI for Science Workshop. 2023 (2023)](https://openreview.net/forum?id=kFiMXnLH9x) |  [code](https://github.com/SXKDZ/MARCEL) 






### VAE-based Molecular Conformation Generation




* **Deep-Learning-Assisted Enhanced Sampling for Exploring Molecular Conformational Changes** [2023]   
Haohao Fu, Han Liu, Jingya Xing, Tong Zhao, Xueguang Shao, and Wensheng Cai.   
[J. Phys. Chem. B (2023)](https://doi.org/10.1021/acs.jpcb.3c05284) 

* **An End-to-End Framework for Molecular Conformation Generation via Bilevel Programming** [2021]   
Xu, Minkai, Wujie Wang, Shitong Luo, Chence Shi, Yoshua Bengio, Rafael Gomez-Bombarelli, and Jian Tang.   
[International Conference on Machine Learning. PMLR (2021)](http://proceedings.mlr.press/v139/xu21f.html) |  [code](https://github.com/MinkaiXu/ConfVAE-ICML21) 






### GAN-based Molecular Conformation Generation








* **COSMIC: Molecular Conformation Space Modeling in Internal Coordinates with an Adversarial Framework** [2024]   
Kuznetsov, Maksim, Fedor Ryabov, Roman Schutski, Rim Shayakhmetov, Yen-Chu Lin, Alex Aliper, and Daniil Polykovskiy.   
[J. Chem. Inf. Model. (2024)](https://doi.org/10.1021/acs.jcim.3c00989) |  [code](https://github.com/insilicomedicine/COSMIC)  









### Energy-based Molecular Conformation Generation






* **Energy-inspired molecular conformation optimization** [2022]   
Guan, Jiaqi, Wesley Wei Qian, Wei-Ying Ma, Jianzhu Ma, and Jian Peng.   
[International Conference on Learning Representations. (2022)](https://openreview.net/forum?id=7QfLW-XZTl) |  [code](https://github.com/guanjq/confopt_officialf) 








### Diffusion-based Molecular Conformation Generation





* **Diffusion-based generative AI for exploring transition states from 2D molecular graphs** [204]   
Kim, S., Woo, J. & Kim, W.Y.   
[Nat Commun 15, 341 (2024)](https://doi.org/10.1038/s41467-023-44629-6) |  [code](https://github.com/seonghann/tsdiff)

* **Physics-informed generative model for drug-like molecule conformers** [204]   
David C. Williams, Neil Imana.   
[	arXiv:2403.07925. (2024)](https://arxiv.org/abs/2403.07925v1) |  [code](https://github.com/nobiastx/diffusion-conformer)   

* **DynamicsDiffusion: Generating and Rare Event Sampling of Molecular Dynamic Trajectories Using Diffusion Models** [2023]   
Petersen, Magnus, Gemma Roig, and Roberto Covino.   
[NeurIPS 2023 AI4Science  (2023)](https://openreview.net/forum?id=pwYCCq4xAf) 

* **Generating Molecular Conformer Fields** [2023]   
Yuyang Wang, Ahmed Elhag, Navdeep Jaitly, Joshua Susskind, Miguel Bautista.   
[NeurIPS 2023 Generative AI and Biology (GenBio) Workshop (2023)]https://openreview.net/forum?id=Od1KtMeAYo) 

* **On Accelerating Diffusion-based Molecular Conformation Generation in SE(3)-invariant Space** [2023]   
Zhou, Z., Liu, R. and Yu, T.   
[arXiv:2310.04915 (2023))](https://arxiv.org/abs/2310.04915) 

* **Molecular Conformation Generation via Shifting Scores** [2023]   
Zhou, Zihan, Ruiying Liu, Chaolong Ying, Ruimao Zhang, and Tianshu Yu.   
[arXiv:2309.09985 (2023)](https://arxiv.org/abs/2309.09985) 

* **EC-Conf: An Ultra-fast Diffusion Model for Molecular Conformation Generation with Equivariant Consistency** [2023]   
Fan, Zhiguang, Yuedong Yang, Mingyuan Xu, and Hongming Chen.   
[arXiv:2308.00237 (2023)](https://arxiv.org/abs/2308.00237) 

* **Torsional diffusion for molecular conformer generation** [2022]   
Jing, Bowen, Gabriele Corso, Jeffrey Chang, Regina Barzilay, and Tommi Jaakkola.   
[NeurIPS. (2022)](https://proceedings.neurips.cc/paper_files/paper/2022/hash/994545b2308bbbbc97e3e687ea9e464f-Abstract-Conference.html) |  [code](https://github.com/gcorso/torsional-diffusionf) 

* **GeoDiff: A Geometric Diffusion Model for Molecular Conformation Generation** [2022]   
Xu, Minkai, Lantao Yu, Yang Song, Chence Shi, Stefano Ermon, and Jian Tang.   
[International Conference on Learning Representations. (2022)](https://openreview.net/forum?id=PzcvxEMzvQC) |  [code](https://github.com/MinkaiXu/GeoDiff) 




### RL-based Molecular Conformation Generation







* **Conformer-RL: A deep reinforcement learning library for conformer generation** [2022]   
Jiang, Runxuan, Tarun Gogineni, Joshua Kammeraad, Yifei He, Ambuj Tewari, and Paul M. Zimmerman.   
[Journal of Computational Chemistry 43.27 (2022)](https://doi.org/10.1002/jcc.26984) |  [code](https://github.com/ZimmermanGroup/conformer-rl) 






### GNN-based Molecular Conformation Generation




* **Leveraging 2D Molecular Graph Pretraining for Improved 3D Conformer Generation with Graph Neural Networks** [2024]   
Jiang, Runxuan, Tarun Gogineni, Joshua Kammeraad, Yifei He, Ambuj Tewari, and Paul M. Zimmerman.   
[Computers & Chemical Engineering (2024)](https://doi.org/10.1016/j.compchemeng.2024.108622) |  [code](https://github.com/m1k2zoo/2D-3DConformerGNN) 








## Deep Learning-based design
* **Reinvent 4: Modern AI–driven generative molecule design** [2024]  
Hannes H. Loeffler, Jiazhen He, Alessandro Tibo, Jon Paul Janet, Alexey Voronov, Lewis H. Mervin & Ola Engkvist  
[Journal of Cheminformatics,16(20) (2024)](https://jcheminf.biomedcentral.com/articles/10.1186/s13321-024-00812-5) | [code](https://github.com/MolecularAI/REINVENT4)


* **Chemistry42: An AI-Driven Platform for Molecular Design and Optimization** [2023]   
Ivanenkov, Yan A., Daniil Polykovskiy, Dmitry Bezrukov, Bogdan Zagribelnyy, Vladimir Aladinskiy, Petrina Kamya, Alex Aliper, Feng Ren, and Alex Zhavoronkov.   
[Journal of Chemical Information and Modeling 63.3 (2023)](https://doi.org/10.1021/acs.jcim.2c01191) |  [web](https://cloud.chemistry42.com/login) 







### RNN-based






* **Prospective de novo drug design with deep interactome learning** [2024]   
Atz, K., Cotos, L., Isert, C. et al.   
[Nat Commun 15, 3408 (2024)](https://doi.org/10.1038/s41467-024-47613-w) |  [code](https://github.com/atzkenneth/dragonfly_gen)

* **CNSMolGen: a bidirectional recurrent neural networks based generative model for de novo central nervous system drug design** [2024]   
Gou, Rongpei, Jingyi Yang, Menghan Guo, Yingjun Chen, and Weiwei Xue.   
[chemrxiv-2024-x4wbl (2024)](https://doi.org/10.26434/chemrxiv-2024-x4wbl) |  [code](https://github.com/xueww/CNSMolGen/) 

* **NovoMol: Recurrent Neural Network for Orally Bioavailable Drug Design and Validation on PDGFRα Receptor** [2023]   
Rao, Ishir.   
[arXiv:2312.01527 (2023)](https://arxiv.org/abs/2312.01527) |  [code](https://github.com/ishirraov/NovoMol) 

* **Generation of focused drug molecule library using recurrent neural network** [2023]   
Zou, Jinping, Long Zhao, and Shaoping Shi.   
[Journal of Molecular Modeling 29.12 (2023)](https://doi.org/10.1007/s00894-023-05772-5) |  [code](https://github.com/JinPing1025/Drug_RNN) 

* **ChemTSv2: Functional molecular design using de novo molecule generator** [2023]   
Ishida, Shoichi, Tanuj Aasawat, Masato Sumita, Michio Katouda, Tatsuya Yoshizawa, Kazuki Yoshizoe, Koji Tsuda, and Kei Terayama.   
[Wiley Interdisciplinary Reviews: Computational Molecular Science (2023)](https://doi.org/10.1002/wcms.1680) |  [code](https://github.com/molecule-generator-collection/ChemTSv2) 

* **Utilizing Reinforcement Learning for de novo Drug Design**  [2023]   
 Svensson, Hampus Gummesson, Christian Tyrchan, Ola Engkvist, and Morteza Haghir Chehreghani.   
  [arXiv:2303.17615 (2023)](https://arxiv.org/abs/2303.17615) |  [code](https://github.com/MolecularAI/SMILES-RL) 

* **De novo drug design based on Stack-RNN with multi-objective reward-weighted sum and reinforcement learning** [2023]   
Hu, P., Zou, J., Yu, J. et al.   
[ J Mol Model 29, 121 (2023)](https://doi.org/10.1007/s00894-023-05523-6) |  [code](https://github.com/PengWeiHu1/mul_RL) 

* **On The Difficulty of Validating Molecular Generative Models Realistically: A Case Study on Public and Proprietary Data** [2023]   
Handa, Koichi, Morgan Thomas, Michiharu Kageyama, Takeshi Iijima, and Andreas Bender.   
[chemrxiv-2023-lbvgn](https://doi.org/10.26434/chemrxiv-2023-lbvgn) |  [code](https://github.com/MolecularAI/Reinvent) 

* **Magicmol: a light-weighted pipeline for drug-like molecule evolution and quick chemical space exploration** [2023]  
Chen, Lin, Qing Shen, and Jungang Lou.  
[BMC Bioinformatics (2023)](https://doi.org/10.1186/s12859-023-05286-0) |  [code](https://github.com/Josefjosda/Magicmol) 

* **Augmented Hill-Climb increases reinforcement learning efficiency for language-based de novo molecule generation** [2022]  
Thomas, M., O’Boyle, N.M., Bender, A. et al.  
[ J Cheminform (2022)](https://doi.org/10.1186/s13321-022-00646-z) |  [code](https://github.com/MorganCThomas/SMILES-RNN) 

* **De novo molecule design with chemical language models** [2022]  
Grisoni, F., Schneider, G.  
[Artificial Intelligence in Drug Design. Methods in Molecular Biology, vol 2390.(2022)](https://doi.org/10.1007/978-1-0716-1787-8_9) |  [code](https://github.com/grisoniFr/de_novo_design_RNN) 

* **Correlated RNN Framework to Quickly Generate Molecules with Desired Properties for Energetic Materials in the Low Data Regime** [2022]  
Li, Chuan, Chenghui Wang, Ming Sun, Yan Zeng, Yuan Yuan, Qiaolin Gou, Guangchuan Wang, Yanzhi Guo, and Xuemei Pu.   
[J. Chem. Inf. Model. (2022)](https://doi.org/10.1021/acs.jcim.2c00997) |  [code](https://github.com/wangchenghuidream/RNNMGM) 

* **Optimizing Recurrent Neural Network Architectures for De Novo Drug Design** [2021]  
Santos, B. P., Abbasi, M., Pereira, T., Ribeiro, B., & Arrais, J. P.  
[Paper](https://ieeexplore.ieee.org/document/9474742) |  [code](https://github.com/larngroup/RNN-Drug-Generation) 

* **A recurrent neural network (RNN) that generates drug-like molecules for drug discovery** [2021]  
[code](https://github.com/shiwentao00/Molecule-RNN) 

* **A molecule generative model used interaction fingerprint (docking pose) as constraints** [2021]  
[code](https://github.com/jeah-z/IFP-RNN) 

* **Bidirectional Molecule Generation with Recurrent Neural Networks** [2020]  
Grisoni, F., Moret, M., Lingwood, R., & Schneider, G.  
[J. Chem. Inf. Model. (2020)](https://pubs.acs.org/doi/10.1021/acs.jcim.9b00943) |  [code](https://github.com/robinlingwood/BIMODAL) 

* **Direct steering of de novo molecular generation with descriptor conditional recurrent neural networks** [2019]  
Kotsias, PC., Arús-Pous, J., Chen, H. *et al.*   
[Nat Mach Intell 2, 254–265 (2020)](https://www.nature.com/articles/s42256-020-0174-5) |  [code](https://github.com/pcko1/Deep-Drug-Coder) 

* **ChemTS: An Efficient Python Library for de novo Molecular Generation** [2017]  
Yang, X., Zhang, J., Yoshizoe, K., Terayama, K., & Tsuda, K.  
[Science and Technology of Advanced Materials (2017)](https://www.tandfonline.com/doi/full/10.1080/14686996.2017.1401424) |  [code](https://github.com/tsudalab/ChemTS) 








### LSTM-based



* **Prospective de novo drug design with deep interactome learning** [2024]   
Atz, K., Cotos, L., Isert, C. et al.   
[Nat Commun 15, 3408 (2024)](https://doi.org/10.1038/s41467-024-47613-w) |  [code](https://github.com/atzkenneth/dragonfly_gen)

* **Computational Drug Discovery on HIV Virus with a Customized LSTM Variational Autoencoder Deep Learning Architecture** [2023]   
  Kutsal, Mucahit, Ferhat Ucar, and Nida Kati.   
  [CPT: Pharmacometrics & Systems Pharmacology. (2023)](https://doi.org/10.1002/psp4.13085) |  [code](https://github.com/McahitKutsal/lstm-drug-discovery) 

* **Structured State-Space Sequence Models for De Novo Drug Design** [2023]   
  Özçelik R, de Ruiter S, Grisoni F.   
  [chemrxiv-2023-jwmf3. (2023)](https://doi.org/10.26434/chemrxiv-2023-jwmf3) |  [code](https://github.com/molML/s4-for-de-novo-drug-design) 

* **Integrating synthetic accessibility with AI-based generative drug design**  [2023]   
 Parrot, M., Tajmouati, H., da Silva, V.B.R. et al.   
  [J Cheminform 15, 83 (2023)](https://doi.org/10.1186/s13321-023-00742-8)  |  [code](https://github.com/iktos/generation-under-synthetic-constraint/) 

* **Deep interactome learning for de novo drug design**  [2023]   
 Atz K, Cotos Muñoz L, Isert C, Håkansson M, Focht D, Nippa DF, et al.   
  [chemrxiv-2023-cbq9k (2023)](https://doi.org/10.26434/chemrxiv-2023-cbq9k) 

* **Deep learning driven de novo drug design based on gastric proton pump structures** [2023]   
  Abe, K., Ozako, M., Inukai, M. et al.   
  [Commun Biol 6, 956 (2023)](https://doi.org/10.1038/s42003-023-05334-8) |  [code](https://doi.org/10.1038/s42003-023-05334-8) 

* **Artificial Intelligence for Prediction of Biological Activities and Generation of molecular hits using Stereochemical Information** [2023]   
  Pereira, Tiago O., Maryam Abbasi, Rita I. Oliveira, Romina A. Guedes, Jorge AR Salvador, and Joel P. Arrais.   
  [Research Square.  (2023)](https://doi.org/10.21203/rs.3.rs-2499317/v1) |  [code](https://github.com/larngroup/targeted_generation_stereo) 

* **LOGICS: Learning optimal generative distribution for designing de novo chemical structures** [2023]   
  Bae, B., Bae, H. & Nam, H.   
  [J Cheminform 15, 77 (2023)](https://doi.org/10.1186/s13321-023-00747-3) |  [code](https://github.com/GIST-CSBL/LOGICS) 

* **Leveraging molecular structure and bioactivity with chemical language models for de novo drug design** [2023]  
  Kotsias, PC., Arús-Pous, J., Chen, H. *et al.*  
  [Nat Commun 14, 114 (2023)](https://www.nature.com/articles/s41467-022-35692-6) |  [code](https://github.com/ETHmodlab/hybridCLMs/tree/v1.0) 

* **SMILES-based CharLSTM with finetuning and goal-directed generation via policy gradient** [2022]  

  [code](https://github.com/gmattedi/Smiles-LSTM) 

* **DeLA-Drug: A Deep Learning Algorithm for Automated Design of Druglike Analogues** [2022]  
  Creanza, T. M., Lamanna, G., Delre, P., Contino, M., Corriero, N., Saviano, M., ... & Ancona, N.   
  [J. Chem. Inf. Model. (2022)](https://pubs.acs.org/doi/10.1021/acs.jcim.2c00205) |  [Web](http://www.ba.ic.cnr.it/softwareic/deladrugportal/) 

* **De novo design and bioactivity prediction of SARS-CoV-2 main protease inhibitors using recurrent neural network-based transfer learning** [2021]  
Santana, M.V.S., Silva-Jr, F.P.  
  [BMC Chemistry 15, 8 (2021)](https://bmcchem.biomedcentral.com/articles/10.1186/s13065-021-00737-2) |  [code](https://github.com/marcossantanaioc/De_novo_design_SARSCOV2) 

* **Generative Recurrent Networks for De Novo Drug Design** [2018]  
  Gupta, A., Müller, A. T., Huisman, B. J., Fuchs, J. A., Schneider, P., & Schneider, G.   
  [Mol Inform. 2018](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5836943/) |  [code](https://github.com/topazape/LSTM_Chem) 

* **Generative Recurrent Neural Networks for De Novo Drug Design** [2017]  
  Gupta, Anvita, et al.   
  [Mol Inform. 2018](https://onlinelibrary.wiley.com/doi/10.1002/minf.201700111) |  [code](https://github.com/SilviaAmAm/MolBot) 




### Autoregressive-models



* **PocketFlow is a data-and-knowledge-driven structure-based molecular generative model** [2024]   
Shengyong Yang, Yuanyuan Jiang, Guo Zhang et al.  
[Nat Mach Intell (2024)](https://doi.org/10.1038/s42256-024-00808-8) |  [Research Square. PREPRINT. (2023)](https://www.researchsquare.com/article/rs-3077992/v1) |  [code](https://github.com/Saoge123/PocketFlow) 

* **De Novo Molecule Design Towards Biased Properties via a Deep Generative Framework and Iterative Transfer Learning** [2024]   
Sattari, Kianoosh, Dawei Li, Bhupalee Kalita, Yunchao Xie, Fatemeh Barmaleki Lighvan, Olexandr Isayev, and Jian Lin.  
[Digital Discovery (2024)](https://pubs.rsc.org/en/content/articlehtml/2024/dd/d3dd00210a) |  [code](https://github.com/linresearchgroup/RRCGAN_Molecules_Ehl) 

* **Autoregressive fragment-based diffusion for pocket-aware ligand design** [2023]   
 Ghorbani, Mahdi, Leo Gendelev, Paul Beroza, and Michael Keiser.    
  [NeurIPS 2023 Generative AI and Biology (GenBio) Workshop.  (2023)](https://openreview.net/forum?id=E3HN48zjam) |  [code](https://github.com/ghorbanimahdi73/autofragdiff) 

* **Learning on topological surface and geometric structure for 3D molecular generation** [2023]   
Zhang, Odin, Tianyue Wang, Gaoqi Weng, Dejun Jiang, Ning Wang, Xiaorui Wang, Huifeng Zhao et al.  
[Nat Comput Sci (2023)](https://doi.org/10.1038/s43588-023-00530-2) |  [code](https://github.com/HaotianZhangAI4Science/SurfGen) 

* **ResGen is a pocket-aware 3D molecular generation model based on parallel multiscale modelling** [2023]   
Zhang, O., Zhang, J., Jin, J. et al.   
[Nat Mach Intell (2023)](https://doi.org/10.1038/s42256-023-00712-7) |  [code](https://github.com/HaotianZhangAI4Science/ResGen) 

* **FFLOM: A Flow-Based Autoregressive Model for Fragment-to-Lead Optimization** [2023]   
Jieyu Jin, Dong Wang, Guqin Shi, Jingxiao Bao, Jike Wang, Haotian Zhang, Peichen Pan, Dan Li, Xiaojun Yao, Huanxiang Liu, Tingjun Hou, and Yu Kang  
[J. Med. Chem. (2023)](https://doi.org/10.1021/acs.jmedchem.3c01009) |  [code](https://github.com/JenniferKim09/FFLOM) 

* **Domain-Agnostic Molecular Generation with Self-feedback** [2023]  
 Yin Fang, Ningyu Zhang, Zhuo Chen, Xiaohui Fan, Huajun Chen   
  [arXiv:2301.11259v3](https://arxiv.org/abs/2301.11259) |  [code](https://github.com/zjunlp/MolGen) 

* **GraphAF: a Flow-based Autoregressive Model for Molecular Graph Generation** [2020]  
  Shi, C., Xu, M., Zhu, Z., Zhang, W., Zhang, M., & Tang, J.   
    [ICLR (2020)](https://openreview.net/forum?id=S1esMkHYPr) |[arXiv:2001.09382](https://arxiv.org/abs/2001.09382) |  [code](https://github.com/DeepGraphLearning/GraphAF) 








###  Transformer-based


* **BindGPT: A Scalable Framework for 3D Molecular Design via Language Modeling and Reinforcement Learning**  [2024]   
 Zholus, Artem, Maksim Kuznetsov, Roman Schutski, Rim Shayakhmetov, Daniil Polykovskiy, Sarath Chandar, and Alex Zhavoronkov.   
  [arXiv:2406.03686 (2024)](https://arxiv.org/abs/2406.03686)   

* **Exploring Novel Fentanyl Analogues Using a Graph-Based Transformer Model**  [2024]   
 Zhang, Guangle, Yuan Zhang, Ling Li, Jiaying Zhou, Honglin Chen, Jinwen Ji, Yanru Li, Yue Cao, Zhihui Xu, and Cong Pian.   
  [Interdisciplinary Sciences: Computational Life Sciences (2024)](https://doi.org/10.1007/s12539-024-00623-0) |  [code](https://github.com/xueyuanyuan0410/fentanyl_data) 

* **TenGAN: Pure Transformer Encoders Make an Efficient Discrete GAN for De Novo Molecular Generation**  [2024]   
 Li, Chen, and Yoshihiro Yamanishi.   
  [ International Conference on Artificial Intelligence and Statistics. PMLR (2024)](https://proceedings.mlr.press/v238/li24d.html)   

* **DockingGA: enhancing targeted molecule generation using transformer neural network and genetic algorithm with docking simulation**  [2024]   
 Changnan Gao, Wenjie Bao, Shuang Wang, Jianyang Zheng, Lulu Wang, Yongqi Ren, Linfang Jiao, Jianmin Wang, Xun Wang.   
  [Briefings in Functional Genomics (2024)](https://doi.org/10.1093/bfgp/elae011) |  [code](https://github.com/Chinafor/DockingGA) 

* **Gotta be SAFE: A New Framework for Molecular Design** [2024]   
  Noutahi, Emmanuel, Cristian Gabellini, Michael Craig, Jonathan SC Lim, and Prudencio Tossou.   
  [Digital Discovery (2024)](https://doi.org/10.1039/D4DD00019F) |  [arXiv:2310.10773 (2023)](https://arxiv.org/abs/2310.10773) |  [code](https://github.com/datamol-io/safe)

* **Enhancing molecular design efficiency: Uniting language models and generative networks with genetic algorithms**  [2024]   
 Bhowmik, Debsindhu, Pei Zhang, Zachary Fox, Stephan Irle, and John Gounley.   
  [Patterns (2024)](https://doi.org/10.1016/j.patter.2024.100947) |  [code](https://zenodo.org/records/8387351) 

* **ChemSpaceAL: An Efficient Active Learning Methodology Applied to Protein-Specific Molecular Generation**  [2024]   
 Kyro, Gregory W., Anton Morgunov, Rafael I. Brent, and Victor S. Batista.   
  [J. Chem. Inf. Model. (2024)](https://doi.org/10.1021/acs.jcim.3c01456) |  [code](https://github.com/batistagroup/ChemSpaceAL) 

* **Evaluation of Reinforcement Learning in Transformer-based Molecular Design**  [2024]   
 He J, Tibo A, Janet JP, Nittinger E, Tyrchan C, Czechtizky W, et al.   
  [chemrxiv-2024-r9ljm (2024)](https://doi.org/10.26434/chemrxiv-2024-r9ljm) |  [code](https://github.com/MolecularAI/transformer_rl) 

* **Dual-Space Optimization: Improved Molecule Sequence Design by Latent Prompt Transformer** [2024]   
  Deqian Kong and Yuhao Huang and Jianwen Xie and Edouardo Honig and Ming Xu and Shuanghong Xue and Pei Lin and Sanping Zhou and Sheng Zhong and Nanning Zheng and Ying Nian Wu.   
  [	arXiv:2402.17179 (2024)](https://arxiv.org/abs/2402.17179) 

* **A novel molecule generative model of VAE combined with Transformer** [2024]   
  Yasuhiro Yoshikai and Tadahaya Mizuno and Shumpei Nemoto and Hiroyuki Kusuhara.   
  [	arXiv:2402.11950 (2024)](https://arxiv.org/abs/2402.11950) |  [code](https://github.com/mizuno-group/TransformerVAE) 

* **GexMolGen: Cross-modal Generation of Hit-like Molecules via Large Language Model Encoding of Gene Expression Signatures** [2024]   
  Cheng, Jia-Bei, Xiaoyong Pan, Kaiyuan Yang, Shenghao Cao, Bin Liu, Qingran Yan, and Ye Yuan.   
  [bioRxiv (2024)](https://www.biorxiv.org/content/10.1101/2023.11.11.566725v4) |  [code](https://github.com/Bunnybeibei/GexMolGen) 

* **Local Scaffold Diversity-Contributed Generator for Discovering Potential NLRP3 Inhibitors** [2024]   
  Weichen Bo, Yangqin Duan, Yurong Zou, Ziyan Ma, Tao Yang, Peng Wang, Tao Guo, Zhiyuan Fu, Jianmin Wang, Linchuan Fan, Jie liu, Taijin Wang, and Lijuan Chen.   
  [J. Chem. Inf. Model. (2024)](https://doi.org/10.1021/acs.jcim.3c01818) |  [code](https://github.com/wichen-2022/LSDC) 

* **Target-aware Molecule Generation for Drug Design Using a Chemical Language Model** [2024]   
  Xia, Yingce, Kehan Wu, Pan Deng, Renhe Liu, Yuan Zhang, Han Guo, Yumeng Cui et al.   
  [bioRxiv (2024)](https://doi.org/10.1101/2024.01.08.574635) 

* **Accelerating Discovery of Novel and Bioactive Ligands With Pharmacophore-Informed Generative Models** [2024]   
  Xie, Weixin, Jianhang Zhang, Qin Xie, Chaojun Gong, Youjun Xu, Luhua Lai, and Jianfeng Pei.   
  [arXiv:2401.01059 (2024)](https://arxiv.org/abs/2401.01059) |  [code](http://gitlab.iipharma.cn/zhangjh/transpharmer-repo) 

* **A self-improvable Polymer Discovery Framework Based on Conditional Generative Model** [2023]   
  Xiangyun Lei and Weike Ye and Zhenze Yang and Daniel Schweigert and Ha-Kyung Kwon and Arash Khajeh.   
  [	arXiv:2312.04013.  (2023)](https://arxiv.org/abs/2312.04013) 

* **LLamol: A Dynamic Multi-Conditional Generative Transformer for De Novo Molecular Design** [2023]   
  Dobberstein, Niklas, Astrid Maass, and Jan Hamaekers.   
  [arXiv:2311.14407.  (2023)](https://arxiv.org/abs/2311.14407) |  [code](https://github.com/Fraunhofer-SCAI/llamol) 

* **GraphGPT: A Graph Enhanced Generative Pretrained Transformer for Conditioned Molecular Generation** [2023]   
  Lu, Hao, Zhiqiang Wei, Xuze Wang, Kun Zhang, and Hao Liu.   
  [International Journal of Molecular Sciences 24.23 (2023)](https://doi.org/10.3390/ijms242316761) |  [code](https://github.com/luhao27/GraphGPT) 

* **PROTACable is an Integrative Computational Pipeline of 3-D Modeling and Deep Learning to Automate the De Novo Design of PROTACs** [2023]   
  Hazem Mslati, Francesco Gentile, Mohit Pandey, Fuqiang Ban, Artem Cherkasov.   
  [bioRxiv 2023.11.20.567951. (2023)](https://doi.org/10.1101/2023.11.20.567951) |  [code](https://github.com/giaguaro/PROTACable) 

* **Molecular generation strategy and optimization based on A2C reinforcement learning in de novo drug design** [2023]   
  Wang, Qian, Zhiqiang Wei, Xiaotong Hu, Zhuoya Wang, Yujie Dong, and Hao Liu.   
  [Bioinformatics: btad693.  (2023)](https://doi.org/10.1093/bioinformatics/btad693) |  [code](https://github.com/wq-sunshine/MomdTDSRL) 

* **Cross-modal Generation of Hit-like Molecules via Foundation Model Encoding of Gene Expression Signatures** [2023]   
  Jiabei Cheng, Xiaoyong Pan, Kaiyuan Yang, Shenghao Cao, Bin Liu, Ye Yuan.   
  [bioRxiv 2023.11.11.566725.  (2023)](https://doi.org/10.1101/2023.11.11.566725) |  [code](https://github.com/Bunnybeibei/GexMolGen) 

* **REINVENT4: Modern AI–Driven Generative Molecule Design** [2023]   
  Loeffler H, He J, Tibo A, Janet JP, Voronov A, Mervin L, et al.   
  [chemrxiv-2023-xt65x  (2023)](https://doi.org/10.26434/chemrxiv-2023-xt65x) |  [code](https://github.com/MolecularAI/REINVENT4) 

* **Optimization of binding affinities in chemical space with transformer and deep reinforcement learning** [2023]   
  Xu, Xiaopeng, Juexiao Zhou, Chen Zhu, Qing Zhan, Zhongxiao Li, Ruochi Zhang, Yu Wang, Xingyu Liao, and Xin Gao.   
  [chemrxiv-2023-7v4sw (2023)](https://doi.org/10.26434/chemrxiv-2023-7v4sw) |  [code](https://github.com/charlesxu90/sgpt) 

* **Searching for High-Value Molecules Using Reinforcement Learning and Transformers** [2023]   
  Raj Ghugare and Santiago Miret and Adriana Hugessen and Mariano Phielipp and Glen Berseth.   
  [arXiv:2310.02902 (2023)](https://arxiv.org/abs/2310.02902)

* **Molecular De Novo Design through Transformer-based Reinforcement Learning** [2023]   
  Feng, Tao, Pengcheng Xu, Tianfan Fu, Siddhartha Laghuvarapu, and Jimeng Sun.   
  [arXiv:2310.05365 (2023)](https://arxiv.org/abs/2310.05365)

* **Probabilistic generative transformer language models for generative design of molecules** [2023]   
  Wei, L., Fu, N., Song, Y. et al.   
  [J Cheminform 15, 88 (2023)](https://doi.org/10.1186/s13321-023-00759-z) |  [code](https://github.com/usccolumbia/GMTransformer) 

* **De Novo Drug Design with Joint Transformers** [2023]   
  Adam Izdebski and Ewelina Węglarz-Tomczak and Ewa Szczurek and Jakub M. Tomczak   
  [	arXiv:2310.02066. (2023)](https://arxiv.org/abs/2310.02066) 

* **Structured State-Space Sequence Models for De Novo Drug Design** [2023]   
  Özçelik R, de Ruiter S, Grisoni F.   
  [chemrxiv-2023-jwmf3. (2023)](https://doi.org/10.26434/chemrxiv-2023-jwmf3) |  [code](https://github.com/molML/s4-for-de-novo-drug-design) 

* **De Novo Generation of Chemical Structures of Inhibitor and Activator Candidates for Therapeutic Target Proteins by a Transformer-Based Variational Autoencoder and Bayesian Optimization** [2023]   
  Yuki Matsukiyo, Chikashige Yamanaka, and Yoshihiro Yamanishi.   
  [J. Chem. Inf. Model. (2023)](https://doi.org/10.1021/acs.jcim.3c00824) |  [code](https://yamanishi.cs.i.nagoya-u.ac.jp/triompheboa/) 

* **A Reverse-Transformer GAN Generates Scaffold-Constrained Molecules with Property Optimization.** [2023]   
Li, C., Yamanishi, Y.    
[ECML PKDD (2023)](https://doi.org/10.1007/978-3-031-43412-9_19) |  [code](https://github.com/naruto7283/SpotGAN) 

* **ChemSpaceAL: An Efficient Active Learning Methodology Applied to Protein-Specific Molecular Generation**  [2023]   
 Kyro, Gregory W., Anton Morgunov, Rafael I. Brent, and Victor S. Batista.   
  [arXiv:2309.05853 (2023)](https://arxiv.org/abs/2309.05853) |  [code](https://github.com/batistagroup/ChemSpaceAL/) 

* **Efficient 3D Molecular Design with an E(3) Invariant Transformer VAE**  [2023]   
 Dollar, Orion, Nisarg Joshi, Jim Pfaendtner, and David AC Beck.   
  [The Journal of Physical Chemistry A (2023)](https://doi.org/10.1021/acs.jpca.3c04188) |  [code](https://github.com/oriondollar/vagrant_en) 

* **Lingo3DMol: Generation of a Pocket-based 3D Molecule using a Language Model**  [2023]   
 Wang, Lvwei, Zaiyun Lin, Yanhao Zhu, Rong Bai, Wei Feng, Huting Wang, Jielong Zhou, Wei Peng, Bo Huang, and Wenbiao Zhou.   
  [arXiv:2305.10133 (2023)](https://arxiv.org/abs/2305.10133) |  [code](https://github.com/stonewiseAIDrugDesign/Lingo3DMol) 

* **FSM-DDTR: End-to-end feedback strategy for multi-objective De Novo drug design using transformers**  [2023]   
 Monteiro, Nelson RC, Tiago O. Pereira, Ana Catarina D. Machado, José L. Oliveira, Maryam Abbasi, and Joel P. Arrais.   
  [Computers in Biology and Medicine (2023)](https://doi.org/10.1016/j.compbiomed.2023.107285) |  [code](https://github.com/larngroup/FSM-DDTR) 

* **Macrocyclization of linear molecules by deep learning to facilitate macrocyclic drug candidates discovery**  [2023]   
 Diao, Y., Liu, D., Ge, H. et al.   
  [Nat Commun 14, 4552 (2023)](https://doi.org/10.1038/s41467-023-40219-8) |  [code](https://github.com/yydiao1025/Macformer) 

* **De novo drug design based on patient gene expression profiles via deep learning**  [2023]   
 Yamanaka, Chikashige, Shunya Uki, Kazuma Kaitoh, Michio Iwata, and Yoshihiro Yamanishi.   
  [Molecular Informatics (2023)](https://doi.org/10.1002/minf.202300064) |  [code](https://www.dropbox.com/s/vg3nxcio799h4ex/software.zip?dl=0) 

* **Transformer-based deep learning method for optimizing ADMET properties of lead compounds** [2023]   
 Yang, Lijuan, Chao Jin, Guanghui Yang, Zhitong Bing, Liang Huang, Yuzhen Niu, and Lei Yang.    
  [Physical Chemistry Chemical Physics 25.3 (2023)](https://doi.org/10.1039/D2CP05332B) 

* **Sequence-based drug design as a concept in computational drug design**   [2023]   
 Chen, L., Fan, Z., Chang, J. et al.    
  [Nat Commun 14, 4217 (2023)]( https://doi.org/10.1038/s41467-023-39856-w)  |  [code](https://github.com/lifanchen-simm/transformerCPI2.0/) 

* **DrugGPT: A GPT-based Strategy for Designing Potential Ligands Targeting Specific Proteins** [2023]   
 Yuesen Li, Chengyi Gao, Xin Song, Xiangyu Wang,  View ORCID ProfileYungang Xu, Suxia Han    
  [bioRxiv (2023)](https://doi.org/10.1101/2023.06.29.543848) |  [code](https://github.com/LIYUESEN/druggpt) 

* **PrefixMol: Target- and Chemistry-aware Molecule Design via Prefix Embedding** [2023]   
 Gao, Zhangyang, Yuqi Hu, Cheng Tan, and Stan Z. Li.    
  [arXiv:2302.07120 (2023)](https://arxiv.org/abs/2302.07120) |  [code](https://github.com/A4Bio/PrefixMolf) 

* **Adaptive language model training for molecular designs** [2023]  
 Andrew E. Blanchard, Debsindhu Bhowmik, Zachary Fox, John Gounley, Jens Glaser, Belinda S. Akpa & Stephan Irle.  
  [ J Cheminform 15, 59 (2023)](https://doi.org/10.1186/s13321-023-00719-7) |  [code](https://code.ornl.gov/candle/mlmol%20in%20the%20adaptive-lm%20directory) 

* **CMGN: a conditional molecular generation net to design target-specific molecules with desired properties** [2023]   
 Yang, Minjian, Hanyu Sun, Xue Liu, Xi Xue, Yafeng Deng, and Xiaojian Wang.    
  [Briefings in Bioinformatics, 2023;, bbad185](https://doi.org/10.1093/bib/bbad185) |  [code](https://github.com/WJmodels/CMGN) 

* **cMolGPT: A Conditional Generative Pre-Trained Transformer for Target-Specific De Novo Molecular Generation** [2023]   
 Wang, Ye, Honggang Zhao, Simone Sciabola, and Wenlu Wang.    
  [Molecules 2023, 28(11), 4430](https://doi.org/10.3390/molecules28114430) |  [code](https://github.com/VV123/cMolGPT) 

* **Molecule generation using transformers and policy gradient reinforcement learning** [2023]   
 Mazuz, E., Shtar, G., Shapira, B. et al.    
  [Sci Rep 13, 8799 (2023)](https://doi.org/10.1038/s41598-023-35648-w) |  [code](https://github.com/eyalmazuz/MolGen) 

* **iupacGPT: IUPAC-based large-scale molecular pre-trained model for property prediction and molecule generation** [2023]   
 Jiashun Mao,, Jianmin  Wang, Kwang-Hwi Cho, Kyoung Tai No    
  [chemrxiv-2023-5kjvh](https://doi.org/10.26434/chemrxiv-2023-5kjvh) |  [code](https://github.com/AspirinCode/iupacGPT) 

* **Molecular Generation with Reduced Labeling through Constraint Architecture** [2023]   
 Wang, Jike, Yundian Zeng, Huiyong Sun, Junmei Wang, Xiaorui Wang, Ruofan Jin, Mingyang Wang et al.    
  [J. Chem. Inf. Model. (2023)](https://pubs.acs.org/doi/full/10.1021/acs.jcim.3c00579) |  [code](https://github.com/jkwang93/Frag-G_M) 

* **Generative Discovery of Novel Chemical Designs using Diffusion Modeling and Transformer Deep Neural Networks with Application to Deep Eutectic Solvents** [2023]   
 Luu, Rachel K., Marcin Wysokowski, and Markus J. Buehler.    
  [arXiv:2304.12400v1](https://arxiv.org/abs/2304.12400) |  [code](https://github.com/lamm-mit/MoleculeDiffusionTransformer) 

* **Regression Transformer enables concurrent sequence regression and generation for molecular language modelling** [2023]   
 Born, J., Manica, M.    
  [Nat Mach Intell 5, 432–444 (2023)](https://doi.org/10.1038/s42256-023-00639-z) |  [code](https://github.com/GT4SD/gt4sd-core/tree/main/examples/regression_transformer) 

* **Transformer-based molecular generative model for antiviral drug design** [2023]  
 mao, jiashun; wang, jianming; zeb, amir; Cho, Kwang-Hwi; jin, haiyan; Kim, Jongwan; Lee, Onju; Wang, Yunyun; No, Kyoung Tai.    
  [J. Chem. Inf. Model. (2023)](https://doi.org/10.1021/acs.jcim.3c00536) |  [code](https://github.com/AspirinCode/TransAntivirus) 

* **Target Specific De Novo Design of Drug Candidate Molecules with Graph Transformer-based Generative Adversarial Networks** [2023]   
  Ünlü, Atabey, Elif Çevrim, Ahmet Sarıgün, Hayriye Çelikbilek, Heval Ataş Güvenilir, Altay Koyaş, Deniz Cansen Kahraman, Ahmet Rifaioğlu, and Abdurrahman Olğaç.   
  [arXiv:2302.07868v5](https://arxiv.org/abs/2302.07868)

* **DrugEx v3: scaffold-constrained drug design with graph transformer-based reinforcement learning** [2023]  
  Liu, X., Ye, K., van Vlijmen, H.W.T. et al.   
  [J Cheminform 15, 24 (2023)](https://doi.org/10.1186/s13321-023-00694-z) |  [code](https://github.com/CDDLeiden/DrugEx) 

* **Explore drug-like space with deep generative models** [2023]  
  Wang, Jianmin, et al.  
  [Methods (2023)](https://doi.org/10.1016/j.ymeth.2023.01.004) |  [code](https://github.com/AspirinCode/drug-likeness_space) 

* **Large-scale chemical language representations capture molecular structure and properties** [2022]  
  Ross, J., Belgodere, B., Chenthamarakshan, V., Padhi, I., Mroueh, Y., & Das, P.   
  [Nat Mach Intell 4, 1256–1264 (2022)](https://www.nature.com/articles/s42256-022-00580-7) |  [code](https://github.com/IBM/molformer) 

* **AlphaDrug: protein target specific de novo molecular generation** [2022]  
  Qian, Hao, Cheng Lin, Dengwei Zhao, Shikui Tu, and Lei Xu.  
  [ PNAS Nexus (2022)](https://academic.oup.com/pnasnexus/article/1/4/pgac227/6751929) |  [code](https://github.com/CMACH508/AlphaDrug) 

* **Can We Quickly Learn to “Translate” Bioactive Molecules with Transformer Models?** [2022]  
  Bagal, V., Aggarwal, R., Vinod, P. K., & Priyakumar, U. D.  
  [chemrxiv-2022-gln27](https://doi.org/10.26434/chemrxiv-2022-gln27)

* **MolGPT: Molecular Generation Using a Transformer-Decoder Model** [2022]  
  Bagal, V., Aggarwal, R., Vinod, P. K., & Priyakumar, U. D.  
  [J. Chem. Inf. Model. 2022, 62, 9, 2064–2076](https://pubs.acs.org/doi/10.1021/acs.jcim.1c00600) |  [code](https://github.com/devalab/molgpt) 
  
* **Tailoring Molecules for Protein Pockets: a Transformer-based Generative Solution for Structured-based Drug Design** [2022]  
  Wu, K., Xia, Y., Fan, Y., Deng, P., Liu, H., Wu, L., ... & Liu, T. Y.  
  [arXiv.2209.06158](https://arxiv.org/abs/2209.06158) |  [code](https://github.com/HankerWu/TamGent) 
 
 * **Exploiting pretrained biochemical language models for targeted drug design** [2022]  
  Uludoğan, Gökçe, Elif Ozkirimli, Kutlu O. Ulgen, Nilgün Karalı, and Arzucan Özgür.  
  [Bioinformatics (2022)](https://doi.org/10.1093/bioinformatics/btac482) |  [code](https://github.com/boun-tabi/biochemical-lms-for-drug-design) 
 
* **A Transformer-based Generative Model for De Novo Molecular Design** [2022]  
  Wang, Wenlu, et al.  
  [arXiv:2210.08749v2](https://arxiv.org/abs/2210.08749)

* **Translation between Molecules and Natural Language** [2022]  
  Edwards, C., Lai, T., Ros, K., Honke, G., & Ji, H.  
  [arXiv:2204.11817v3](https://arxiv.org/abs/2204.11817) |  [code](https://github.com/blender-nlp/MolT5) 

* **Regression Transformer enables concurrent sequence regression and generation for molecular language modeling** [2022]  
  Born, Jannis and Manica, Matteo  
  [arXiv:2202.01338v3](https://arxiv.org/abs/2202.01338) |  [code](https://github.com/IBM/regression-transformer) 

* **Generative Pre-Training from Molecules** [2021]  
  Adilov, Sanjar.  
  [J. Chem. Inf. Model. 2022, 62, 9, 2064–2076](https://pubs.acs.org/doi/10.1021/acs.jcim.1c00600) |  [code](https://github.com/sanjaradylov/smiles-gpt) 

* **Transformers for Molecular Graph Generation** [2021]  
  Cofala, Tim, and Oliver Kramer.  
  [ESANN 2021 ](https://www.esann.org/sites/default/files/proceedings/2021/ES2021-112.pdf) |  [code](https://gitlab.uni-oldenburg.de/gies6280/molegent) 

* **Spatial Generation of Molecules with Transformers** [2021]  
  Cofala, Tim, and Oliver Kramer.  
  [IJCNN52387.2021.9533439 (2021)](https://ieeexplore.ieee.org/abstract/document/9533439) |  [code](https://gitlab.uni-oldenburg.de/gies6280/molegent) 

* **Generative Chemical Transformer: Neural Machine Learning of Molecular Geometric Structures from Chemical Language via Attentio** [2021]  
  Hyunseung Kim, Jonggeol Na*, and Won Bo Lee*.  
  [J. Chem. Inf. Model. 2021, 61, 12, 5804–5814](https://pubs.acs.org/doi/abs/10.1021/acs.jcim.1c01289) |  [code](https://github.com/Hyunseung-Kim/molGCT) 

* **C5T5: Controllable Generation of Organic Molecules with Transformer** [2021]  
  Rothchild, D., Tamkin, A., Yu, J., Misra, U., & Gonzalez, J.  
  [arXiv:2108.10307v1](https://arxiv.org/abs/2108.10307) |  [code](https://github.com/dhroth/c5t5) 

* **Molecular optimization by capturing chemist’s intuition using deep neural networks** [2021]  
  He, J., You, H., Sandström, E. et al.   
  [J Cheminform 13, 26 (2021)](https://doi.org/10.1186/s13321-021-00497-0) |  [code](https://github.com/MolecularAI/deep-molecular-optimization) 

* **Transformer neural network for protein-specific de novo drug generation as a machine translation problem** [2021]  
  Grechishnikova, Daria.   
  [Sci Rep 11, 321 (2021)](https://www.nature.com/articles/s41598-020-79682-4) |  [code](https://github.com/dariagrechishnikova/molecule_structure_generation) 

* **Transmol: repurposing a language model for molecular generation** [2021]  
  Grechishnikova, Daria.   
  [RSC advances. 2021;11(42):25921-32.](https://pubs.rsc.org/en/content/articlelanding/2021/ra/d1ra03086h) |  [code](https://gitlab.com/cheml.io/public/transmol) 

* **Attention-based generative models for de novo molecular design** [2021]  
  Dollar, O., Joshi, N., Beck, D.A. and Pfaendtner, J.,  
  [Chemical Science 12.24 (2021)](https://pubs.rsc.org/en/content/articlelanding/2021/sc/d1sc01050f) |  [code](https://github.com/oriondollar/TransVAE) 











### VAE-based






* **Structure-Based Drug Design with a Deep Hierarchical Generative Model** [2024]   
  Weller, Jesse A., and Remo Rohs.   
  [J. Chem. Inf. Model. (2024)](https://pubs.acs.org/doi/full/10.1021/acs.jcim.4c01193) |  [code](https://github.com/jssweller/DrugHIVE) 

* **Leveraging Active Subspaces to Capture Epistemic Model Uncertainty in Deep Generative Models for Molecular Design** [2024]   
  Abeer, A. N. M., Sanket Jantre, Nathan M. Urban, and Byung-Jun Yoon.   
  [arXiv:2405.00202 (2024)](https://arxiv.org/abs/2405.00202)  

* **GxVAEs: Two Joint VAEs Generate Hit Molecules from Gene Expression Profiles** [2024]   
  Li, Chen, and Yoshihiro Yamanishi.   
  [Proceedings of the AAAI Conference on Artificial Intelligence. Vol. 38. No. 12. (2024)](https://doi.org/10.1609/aaai.v38i12.29248) |  [code](https://github.com/naruto7283/GxVAEs) 

* **3D molecular generative framework for interaction-guided drug design** [2024]   
  Zhung, W., Kim, H. & Kim, W.Y.   
  [Nat Commun 15, 2688 (2024)](https://doi.org/10.1038/s41467-024-47011-2) |  [code](https://github.com/ACE-KAIST/DeepICL) 

* **Attention Based Molecule Generation via Hierarchical Variational Autoencoder** [2024]   
  Divahar Sivanesan.   
  [	arXiv:2402.16854. (2024)](https://arxiv.org/abs/2402.16854) 

* **A novel molecule generative model of VAE combined with Transformer** [2024]   
  Yasuhiro Yoshikai and Tadahaya Mizuno and Shumpei Nemoto and Hiroyuki Kusuhara.   
  [	arXiv:2402.11950 (2024)](https://arxiv.org/abs/2402.11950) |  [code](https://github.com/mizuno-group/TransformerVAE) 

* **De Novo Generation and Identification of Novel Compounds with Drug Efficacy Based on Machine Learning** [2024]   
  He, Dakuo, Qing Liu, Yan Mi, Qingqi Meng, Libin Xu, Chunyu Hou, Jinpeng Wang et al.   
  [Advanced Science (2024)](https://doi.org/10.1002/advs.202307245) 

* **Computational Drug Discovery on HIV Virus with a Customized LSTM Variational Autoencoder Deep Learning Architecture** [2023]   
  Kutsal, Mucahit, Ferhat Ucar, and Nida Kati.   
  [CPT: Pharmacometrics & Systems Pharmacology. (2023)](https://doi.org/10.1002/psp4.13085) |  [code](https://github.com/McahitKutsal/lstm-drug-discovery) 

* **NRC-VABS: Normalized Reparameterized Conditional Variational Autoencoder with applied beam search in latent space for drug molecule design** [2023]   
  Bhadwal, Arun Singh, Kamal Kumar, and Neeraj Kumar.   
  [Expert Systems with Applications. (2023)](https://doi.org/10.1016/j.eswa.2023.122396) 

* **Target-Aware Variational Auto-Encoders for Ligand Generation with Multi-Modal Protein Modeling** [2023]   
  Ngo, Khang, and Truong Son Hy.   
  [NeurIPS 2023 Generative AI and Biology (GenBio) Workshop. (2023)](https://openreview.net/forum?id=4k926QVVM4) |  [code](https://github.com/HySonLab/Ligand_Generation) 

* **Interaction-aware 3D Molecular Generative Framework for Generalizable Structure-based Drug Design** [2023]   
  Woo Youn Kim, Wonho Zhung, and Hyeongwoo Kim.   
  [Research Square. (2023)](https://www.researchsquare.com/article/rs-3388359/v1) |  [code](https://github.com/ACE-KAIST/DeepICL) 

* **Application of variational graph encoders as an effective generalist algorithm in computer-aided drug design** [2023]   
  Lam, H.Y.I., Pincket, R., Han, H. et al.   
  [Nat Mach Intell 5, 754–764 (2023)](https://doi.org/10.1038/s42256-023-00683-9) |  [code](https://github.com/Chokyotager/NotYetAnotherNightshade) 

* **De Novo Generation of Chemical Structures of Inhibitor and Activator Candidates for Therapeutic Target Proteins by a Transformer-Based Variational Autoencoder and Bayesian Optimization** [2023]   
  Yuki Matsukiyo, Chikashige Yamanaka, and Yoshihiro Yamanishi.   
  [J. Chem. Inf. Model. (2023)](https://doi.org/10.1021/acs.jcim.3c00824) |  [code](https://yamanishi.cs.i.nagoya-u.ac.jp/triompheboa/) 

* **ReBADD-SE: Multi-objective molecular optimisation using SELFIES fragment and off-policy self-critical sequence training** [2023]   
  Choi, Jonghwan, Sangmin Seo, Seungyeon Choi, Shengmin Piao, Chihyun Park, Sung Jin Ryu, Byung Ju Kim, and Sanghyun Park.   
  [Computers in Biology and Medicine 157 (2023)](https://doi.org/10.1016/j.compbiomed.2023.106721) |  [code](https://github.com/mathcom/ReBADD-SE) 

* **Efficient 3D Molecular Design with an E(3) Invariant Transformer VAE**  [2023]   
 Dollar, Orion, Nisarg Joshi, Jim Pfaendtner, and David AC Beck.   
  [The Journal of Physical Chemistry A (2023)](https://doi.org/10.1021/acs.jpca.3c04188) |  [code](https://github.com/oriondollar/vagrant_en) 

* **Multi-constraint molecular generation using sparsely labelled training data for localized high-concentration electrolyte diluent screening**  [2023]   
 Mailoa, Jonathan P., Xin Li, Jiezhong Qiu, and Shengyu Zhang.   
  [Digital Discovery (2023)](https://doi.org/10.1039/D3DD00064H) |  [code](https://github.com/jpmailoa/ConGen)  |  [Dataset](https://github.com/jpmailoa/ConGen_Dataset) 

* **Multi-objective Molecular Optimization for Opioid Use Disorder Treatment Using Generative Network Complex**  [2023]   
 Feng, Hongsong, Rui Wang, Chang-Guo Zhan, and Guo-Wei Wei.   
  [J. Med. Chem. (2023)](https://doi.org/10.1021/acs.jmedchem.3c01053) |  [code](https://weilab.math.msu.edu/DataLibrary/2D/) 

* **ScaffoldGVAE: Scaffold Generation and Hopping of Drug Molecules via a Variational Autoencoder Based on Multi-View Graph Neural Networks**  [2023]   
 Hu, Chao, Song Li, Chenxing Yang, Jun Chen, Yi Xiong, Guisheng Fan, Hao Liu, and Liang Hong.   
  [J Cheminform 15, 91 (2023)](https://doi.org/10.1186/s13321-023-00766-0) |  [Research Square. (2023)](https://www.researchsquare.com/article/rs-3254116/v1) |  [code](https://github.com/ecust-hc/ScaffoldGVAE) 

* **Deep Generative Design of Porous Organic Cages via a Variational Autoencoder**  [2023]   
 Jiajun Zhou, Austin Mroz, Kim Jelfs*.   
  [chemrxiv (2023)](https://doi.org/10.26434/chemrxiv-2023-ggnz0) |  [code](https://github.com/JiajunZhou96/Cage-VAE) 

* **Target-aware Variational Auto-encoders for Ligand Generation with Multimodal Protein Representation Learning**  [2023]   
 Nhat Khang Ngo, Truong Son Hy.   
  [bioRxiv. (2023)](https://doi.org/10.1101/2023.08.10.552868) |  [code](https://github.com/HySonLab/Ligand_Generation) 

* **De Novo Design of κ-Opioid Receptor Antagonists Using a Generative Deep-Learning Framework**  [2023]   
 Salas-Estrada, Leslie, Davide Provasi, Xing Qiu, H. Umit Kaniskan, Xi-Ping Huang, Jeffrey DiBerto, Joao Marcelo Lamim Ribeiro, Jian Jin, Bryan L. Roth, and Marta Filizola.   
  [J. Chem. Inf. Model. (2023)](https://doi.org/10.1021/acs.jcim.3c00651) |  [code](https://github.com/insilicomedicine/GENTRL) 

* **Deep generative model of constructing chemical latent space for large molecular structures with 3D complexity**  [2023]   
 Ochiai, Toshiki, Tensei Inukai, Manato Akiyama, Kairi Furui, Masahito Ohue, Nobuaki Matsumori, Shinsuke Inuki et al.   
  [Commun Chem 6, 249 (2023)](https://doi.org/10.1038/s42004-023-01054-6) |  [chemrxiv (2023)](https://doi.org/10.26434/chemrxiv-2023-pjl0w-v2) |  [code](https://github.com/toshikiochiai/NPVAE) 

* **De novo drug design based on patient gene expression profiles via deep learning**  [2023]   
 Yamanaka, Chikashige, Shunya Uki, Kazuma Kaitoh, Michio Iwata, and Yoshihiro Yamanishi.   
  [Molecular Informatics (2023)](https://doi.org/10.1002/minf.202300064) |  [code](https://www.dropbox.com/s/vg3nxcio799h4ex/software.zip?dl=0) 

* **Construction of order-independent molecular fragments space with vector quantised graph autoencoder**  [2023]   
 Akhmetshin, Timur and Lin, Albert and Madzhidov, Timur and Varnek, Alexandre   
  [chemrxiv-2023-5zmvw](https://doi.org/10.26434/chemrxiv-2023-5zmvw) |  [code](https://github.com/Laboratoire-de-Chemoinformatique/VQGAE) 

* **De Novo Design of κ-Opioid Receptor Antagonists Using a Generative Deep Learning Framework** [2023]   
 Salas-Estrada, Leslie, Davide Provasi, Xing Qiu, H. Umit Kaniskan, Xi-Ping Huang, Jeffrey DiBerto, Joao Marcelo Lamim Ribeiro, Jian Jin, Bryan L. Roth, and Marta Filizola.   
  [bioRxiv (2023)](https://doi.org/10.1101/2023.04.25.537995) |  [code](https://github.com/insilicomedicine/GENTRL) 

* **De Novo Design of Molecules with Multiaction Potential from Differential Gene Expression using Variational Autoencoder** [2023]   
 Salas-Estrada, Leslie, Davide Provasi, Xing Qiu, H. Umit Kaniskan, Xi-Ping Huang, Jeffrey DiBerto, Joao Marcelo Lamim Ribeiro, Jian Jin, Bryan L. Roth, and Marta Filizola.   
  [bioRxiv (2023)](https://doi.org/10.1101/2023.04.25.537995) |  [code](https://github.com/insilicomedicine/GENTRL) 

* **De Novo Design of Molecules with Multiaction Potential from Differential Gene Expression using Variational Autoencoder** [2023]   
 Pravalphruekul, Nutaya, Maytus Piriyajitakonkij, Phond Phunchongharn, and Supanida Piyayotai.   
  [J. Chem. Inf. Model.  (2023)](https://pubs.acs.org/doi/full/10.1021/acs.jcim.3c00355) |  [code](https://github.com/ChemEXL/BiCEV) 

* **Chemical Design with GPU-based Ising Machine** [2023]   
 Mao, Zetian, Yoshiki Matsuda, Ryo Tamura, and Koji Tsuda.   
  [Digital Discovery (2023)](https://doi.org/10.1039/D3DD00047H) |  [code](https://github.com/tsudalab/bVAE-IM) 

* **Accelerating drug target inhibitor discovery with a deep generative foundation model** [2023]  
 Vijil Chenthamarakshan et al.    
  [Sci. Adv.9,eadg7865(2023)](https://www.science.org/doi/10.1126/sciadv.adg7865) |  [code](https://zenodo.org/record/7863805) 

* **De Novo Design of Molecules with Multiaction Potential from Differential Gene Expression using Variational Autoencoder** [2023]  
 Nutaya Pravalphruekul, Maytus Piriyajitakonkij, Phond Phunchongharn, and Supanida Piyayotai.   
  [J. Chem. Inf. Model. 2023](https://doi.org/10.1021/acs.jcim.3c00355) |  [code](https://github.com/ChemEXL/BiCEV) 

* **A Protein-Ligand Interaction-focused 3D Molecular Generative Framework for Generalizable Structure-based Drug Design** [2023]  
 Zhung W, Kim H, Kim WY.   
  [chemrxiv-2023-jsjwx](https://doi.org/10.26434/chemrxiv-2023-jsjwx) |  [code](https://github.com/ACE-KAIST/DeepICL) 

* **VGAE-MCTS: a New Molecular Generative Model combining Variational Graph Auto-Encoder and Monte Carlo Tree Search** [2023]  
 Iwata, Hiroaki, Taichi Nakai, Takuto Koyama, Shigeyuki Matsumoto, Ryosuke Kojima, and Yasushi Okuno.   
   [J. Chem. Inf. Model. (2023)](https://doi.org/10.1021/acs.jcim.3c01220) | [chemrxiv-2023-q8419-v2](https://doi.org/10.26434/chemrxiv-2023-q8419-v2)  |  [code](https://github.com/clinfo/VGAE-MCTS) 

* **Deep Generation Model Guided by the Docking Score for Active Molecular Design** [2023]  
 Yang, Yuwei, Chang-Yu Hsieh, Yu Kang, Tingjun Hou, Huanxiang Liu, and Xiaojun Yao.   
  [J. Chem. Inf. Model. 2023, 63, 10, 2983–2991](https://pubs.acs.org/doi/full/10.1021/acs.jcim.3c00572) |  [code](https://github.com/jaechanglim/CVAE) 

* **Direct De Novo Molecule Generation Using Probabilistic Diverse Variational Autoencoder** [2023]  
 Singh Bhadwal, Arun, and Kamal Kumar.     
  [Computer Vision and Machine Intelligence. Lecture Notes in Networks and Systems, vol 586. Springer (2023)](https://link.springer.com/chapter/10.1007/978-981-19-7867-8_2) 

* **MoVAE: A Variational AutoEncoder for Molecular Graph Generation** [2023]   
 Lin, Zerun, Yuhan Zhang, Lixin Duan, Le Ou-Yang, and Peilin Zhao.   
  [Society for Industrial and Applied Mathematics, 2023.](https://doi.org/10.1137/1.9781611977653.ch58) 

* **Gex2SGen: Designing Drug-like Molecules from Desired Gene Expression Signatures** [2023]   
 Das, Dibyajyoti, Broto Chakrabarty, Rajgopal Srinivasan, and Arijit Roy.   
  [J. Chem. Inf. Model. (2023)](https://doi.org/10.1021/acs.jcim.2c01301)

* **COMA: efficient structure-constrained molecular generation using contractive and margin losses** [2023]  
 Choi, J., Seo, S. & Park, S.     
  [J Cheminform 15, 8 (2023)](https://doi.org/10.1186/s13321-023-00679-y) |  [code](https://github.com/mathcom/COMA) 

* **Design of potent antimalarials with generative chemistry** [2022]   
  Godinez, W.J., Ma, E.J., Chao, A.T. et al.  
  [Nat Mach Intell 4, 180–186 (2022)](https://doi.org/10.1038/s42256-022-00448-w)  |  [code](https://github.com/Novartis/JAEGER) 

* **Accelerating Bayesian Optimization for Biological Sequence Design with Denoising Autoencoders** [2022]  
  Stanton, S., Maddox, W., Gruver, N., Maffettone, P., Delaney, E., Greenside, P., & Wilson, A. G.
  [PMLR 162:20459-20478, 2022](https://arxiv.org/abs/2203.12742)

* **Conditional β-VAE for De Novo Molecular Generation** [2022]  
  Richards, Ryan J., and Austen M. Groener.   
  [arXiv:2205.01592v1 ](https://arxiv.org/abs/2205.01592) 

* **MGCVAE: Multi-Objective Inverse Design via Molecular Graph Conditional Variational Autoencoder** [2022]  
  Lee, Myeonghun, and Kyoungmin Min.  
  [J. Chem. Inf. Model. 2022, 62, 12, 2943–2950](https://pubs.acs.org/doi/10.1021/acs.jcim.2c00487) |  [code](https://github.com/mhlee216/MGCVAE) 

* **RELATION: A Deep Generative Model for Structure-Based De Novo Drug Design** [2022]  
  Wang, M., Hsieh, C.Y., Wang, J., Wang, D., Weng, G., Shen, C., Yao, X., Bing, Z., Li, H., Cao, D. and Hou, T.,  
  [J. Med. Chem. 2022, 65, 13, 9478–9492](https://pubs.acs.org/doi/10.1021/acs.jmedchem.2c00732) |  [code](https://github.com/micahwang/RELATION) 

* **3DLinker: An E(3) Equivariant Variational Autoencoder for Molecular Linker Design** [2022]  
  Huang, Yinan, Xingang Peng, Jianzhu Ma, and Muhan Zhang.   
  [Paper]https://arxiv.org/abs/2205.07309) |  [code](https://github.com/GraphPKU/3DLinker) 

* **Molecule Generation by Principal Subgraph Mining and Assembling** [2022]  
  Kong, X., Huang, W., Tan, Z., & Liu, Y.  
  [NeurIPS 2022](https://openreview.net/forum?id=ATfARCRmM-a) |[arXiv:2106.15098v4](https://arxiv.org/abs/2106.15098) |  [code](https://github.com/THUNLP-MT/PS-VAE) 
  
* **LIMO: Latent Inceptionism for Targeted Molecule Generation** [2022]  
  Eckmann, Peter, Kunyang Sun, Bo Zhao, Mudong Feng, Michael K. Gilson, and Rose Yu.  
  [ICML (2022)](https://proceedings.mlr.press/v162/eckmann22a/eckmann22a.pdf) | [arXiv:2206.09010v1](https://arxiv.org/abs/2206.09010) |  [code](https://github.com/rose-stl-lab/limo) 

* **Predicting chemical structure using reinforcement learning with a stack-augmented conditional variational autoencoder** [2022]  
  Kim, H., Ko, S., Kim, B.J. *et al.*  
  [J Cheminform 14, 83 (2022)](https://jcheminf.biomedcentral.com/articles/10.1186/s13321-022-00666-9) |  [code](https://github.com/HwanheeKim813/stack_CVAE) 

* **Geometry-Based Molecular Generation With Deep Constrained Variational Autoencoder** [2022]   
  Li, Chunyan, Junfeng Yao, Wei Wei, Zhangming Niu, Xiangxiang Zeng, Jin Li, and Jianmin Wang.   
  [IEEE Transactions on Neural Networks and Learning Systems (2022)](https://ieeexplore.ieee.org/abstract/document/9714718) |  [code](https://github.com/anny0316/GEOM-CVAE)

* **High-Dimensional Bayesian Optimisation with Variational Autoencoders and Deep Metric Learning** [2021]
  Grosnit, A., Tutunov, R., Maraval, A.M., Griffiths, R-R., Cowen-Rivers, A.I. et al.
  [arXiv:2106.03609v3](https://arxiv.org/abs/2106.03609) | [code](https://github.com/huawei-noah/HEBO/tree/master/T-LBO)

* **Inverse design of nanoporous crystalline reticular materials with deep generative models.** [2021]  
  Yao, Z., Sánchez-Lengeling, B., Bobbitt, N.S. et al.  
  [Nat Mach Intell 3, 76–86 (2021)](https://doi.org/10.1038/s42256-020-00271-1) |  [code](https://github.com/zhenpengyao/Supramolecular_VAE) 

* **Attention-based generative models for de novo molecular design** [2021]  
  Dollar, O., Joshi, N., Beck, D.A. and Pfaendtner, J.,  
  [Chemical Science 12.24 (2021)](https://pubs.rsc.org/en/content/articlelanding/2021/sc/d1sc01050f) |  [code](https://github.com/oriondollar/TransVAE) 

* **Toward efficient generation, correction, and properties control of unique drug-like structures** [2021]  
  Druchok, Maksym, Dzvenymyra Yarish, Oleksandr Gurbych, and Mykola Maksymenko.  
  [Journal of Computational Chemistry 42.11 (2021)](https://onlinelibrary.wiley.com/doi/10.1002/jcc.26494) |  [code](https://github.com/SoftServeInc/novel-molecule-generation) 

* **Compressed graph representation for scalable molecular graph generation** [2020]  
  Kwon, Youngchun, Dongseon Lee, Youn-Suk Choi, Kyoham Shin, and Seokho Kang.  
  [J Cheminform 12, 58 (2020)](https://jcheminf.biomedcentral.com/articles/10.1186/s13321-020-00463-2) |  [code](https://github.com/seokhokang/graphvae_compress) 

* **Deep learning enables rapid identification of potent DDR1 kinase inhibitors** [2019]   
  Zhavoronkov, A., Ivanenkov, Y.A., Aliper, A. et al.   
  [Nat Biotechnol 37, 1038–1040 (2019)](https://doi.org/10.1038/s41587-019-0224-x) |  [code](https://github.com/insilicomedicine/gentrl) 

* **Fragment Graphical Variational AutoEncoding for Screening Molecules with Small Data** [2019]  
  Armitage, John, Leszek J. Spalek, Malgorzata Nguyen, Mark Nikolka, Ian E. Jacobs, Lorena Marañón, Iyad Nasrallah et al.  
  [arXiv:1910.13325v2](https://arxiv.org/abs/1910.13325) |  [code](https://github.com/OE-FET/FraGVAE) 

* **Molecular generative model based on conditional variational autoencoder for de novo molecular design** [2018]  
  Lim, J., Ryu, S., Kim, J. W., & Kim, W. Y.   
  [J Cheminform 10, 31 (2018)](https://jcheminf.biomedcentral.com/articles/10.1186/s13321-018-0286-7) |  [code](https://github.com/jaechanglim/CVAE)

* **Constrained Bayesian optimization for automatic chemical design using variational autoencoders** [2017]  
  Griffiths, R-R., Hernández-Lobato, J. M.  
  [Chemical Science 11, 2 (2020)](https://pubs.rsc.org/en/content/articlehtml/2019/sc/c9sc04026a) | [arXiv:1709.05501v6](https://arxiv.org/abs/1709.05501) | [code](https://github.com/Ryan-Rhys/Constrained-Bayesian-Optimisation-for-Automatic-Chemical-Design)

* **Automatic chemical design using a data-driven continuous representation of molecules** [2017]  
  Gómez-Bombarelli, R., Wei, J. N., Duvenaud, D., Hernández-Lobato, J. M., Sánchez-Lengeling, B., Sheberla, D., ... & Aspuru-Guzik, A.   
  [ACS Cent. Sci. 2018](https://pubs.acs.org/doi/10.1021/acscentsci.7b00572) | [arXiv:1610.02415v3](https://arxiv.org/abs/1610.02415) |  [code](https://github.com/tuantla80/VAE-Molecular-Generation) 









### GAN-based


* **TenGAN: Pure Transformer Encoders Make an Efficient Discrete GAN for De Novo Molecular Generation**  [2024]   
 Li, Chen, and Yoshihiro Yamanishi.   
  [ International Conference on Artificial Intelligence and Statistics. PMLR (2024)](https://proceedings.mlr.press/v238/li24d.html)   

* **Enhancing molecular design efficiency: Uniting language models and generative networks with genetic algorithms**  [2024]   
 Bhowmik, Debsindhu, Pei Zhang, Zachary Fox, Stephan Irle, and John Gounley.   
  [Patterns (2024)](https://doi.org/10.1016/j.patter.2024.100947) |  [code](https://zenodo.org/records/8387351) 

* **Automated Generation and Analysis of Molecular Images Using Generative Artificial Intelligence Models** [2024]   
Zhiwen Zhu, Jiayi Lu, Shaoxuan Yuan, Yu He, Fengru Zheng, Hao Jiang, Yuyi Yan, Qiang Sun.  
[J. Phys. Chem. Lett. (2024)](https://doi.org/10.1021/acs.jpclett.3c03504)  

* **De Novo Molecule Design Towards Biased Properties via a Deep Generative Framework and Iterative Transfer Learning** [2024]   
Sattari, Kianoosh, Dawei Li, Bhupalee Kalita, Yunchao Xie, Fatemeh Barmaleki Lighvan, Olexandr Isayev, and Jian Lin.  
[Digital Discovery (2024)](https://pubs.rsc.org/en/content/articlehtml/2024/dd/d3dd00210a) |  [code](https://github.com/linresearchgroup/RRCGAN_Molecules_Ehl) 

* **STAGAN: An approach for improve the stability of molecular graph generation based on generative adversarial networks** [2023]   
Zou, Jinping, Jialin Yu, Pengwei Hu, Long Zhao, and Shaoping Shi.    
[Computers in Biology and Medicine (2023)](https://doi.org/10.1016/j.compbiomed.2023.107691) |  [code](https://github.com/JinPing1025/STAGAN) 

* **An interface-based molecular generative framework for protein-protein interaction inhibitors** [2023]   
Jianmin Wang, Jiashun Mao, Chunyan Li, Hongxin Xiang, Xun Wang, Shuang Wang, Zixu Wang, Yangyang Chen, Yuquan Li, Heqi Sun, Kyoung Tai No, Tao Song, Xiangxiang Zeng    
[bioRxiv (2023)](https://doi.org/10.1101/2023.10.10.557742) |  [code](https://github.com/AspirinCode/GENiPPI) 

* **A Reverse-Transformer GAN Generates Scaffold-Constrained Molecules with Property Optimization.** [2023]   
Li, C., Yamanishi, Y.    
[ECML PKDD (2023)](https://doi.org/10.1007/978-3-031-43412-9_19) |  [code](https://github.com/naruto7283/SpotGAN) 

* **Mol-Zero-GAN: Zero-Shot Adaptation of Molecular Generative Adversarial Network for Specific Protein Targets** [2023]   
Ravipas Aphikulvanich*， Natapol Pornputtapong， Duangdao Wichadakul   
[Paper](https://doi.org/10.26434/chemrxiv-2023-lv2m1) |  [code](https://github.com/cucpbioinfo/Mol-Zero-GAN) 

* **De Novo Design of Molecules Towards Biased Properties via a Deep Generative Framework and Iterative Transfer Learning** [2023]   
Sattari, Kianoosh, Dawei Li, Yunchao Xie, Olexandr Isayev, and Jian Lin.   
[Paper](https://doi.org/10.26434/chemrxiv-2023-0zv2f-v2) |  [code](https://github.com/linresearchgroup/RRCGAN_Molecules) 

* **MolFilterGAN: a progressively augmented generative adversarial network for triaging AI-designed molecules** [2023]   
Liu, X., Zhang, W., Tong, X. et al.   
[J Cheminform 15, 42 (2023)](https://doi.org/10.1186/s13321-023-00711-1) |  [code](https://github.com/MolFilterGAN/MolFilterGAN) 

* **Deep generative model for drug design from protein target sequence** [2023]   
Yangyang Chen, Zixu Wang, Lei Wang, Jianmin Wang, Pengyong Li, Dongsheng Cao, Xiangxiang Zeng, Xiucai Ye & Tetsuya Sakurai.   
[J Cheminform 15, 38 (2023)](https://doi.org/10.1186/s13321-023-00702-2) |  [code](https://github.com/viko-3/TargetGAN) 

* **Cell morphology-guided de novo hit design by conditioning GANs on phenotypic image features** [2022]  
Zapata, Paula A. Marin, Oscar Méndez-Lucio, Tuan Le, Carsten Jörn Beese, Jörg Wichard, David Rouquié, and Djork-Arné Clevert.  
[Digital Discovery (2023)](https://doi.org/10.1039/D2DD00081D) |  [code](https://github.com/Bayer-Group/CPMolGAN) 

* **Generating 3D molecules conditional on receptor binding sites with deep generative models** [2022]  
Ragoza, Matthew, Tomohide Masuda, and David Ryan Koes.  
[Chemical science. 2022;13(9):2701-13.](https://pubs.rsc.org/en/content/articlelanding/2022/sc/d1sc05976a) |  [code](https://github.com/mattragoza/liGAN) 

* **Designing optimized drug candidates with Generative Adversarial Network** [2022]  
Abbasi, M., Santos, B.P., Pereira, T.C. et al.  
[J Cheminform 14, 40 (2022)](https://jcheminf.biomedcentral.com/articles/10.1186/s13321-022-00623-6) |  [code](https://github.com/larngroup/GAN-Drug-Generator) 

* **De novo molecular design with deep molecular generative models for PPI inhibitors** [2022]  
Jianmin Wang, Yanyi Chu, Jiashun Mao, Hyeon-Nae Jeon, Haiyan Jin, Amir Zeb, Yuil Jang, Kwang-Hwi Cho, Tao Song, Kyoung Tai No.  
[Briefings in Bioinformatics,July 2022, bbac285,](https://doi.org/10.1093/bib/bbac285) |  [code](https://github.com/AspirinCode/iPPIGAN) 

* **Improvement on Generative Adversarial Network for Targeted Drug Design** [2021]  
Santos, B. P., Abbasi, M., Pereira, T., Ribeiro, B., & Arrais, J.  
[ESANN.(2021)](https://www.cisuc.uc.pt/download-file/16987/IR5glkL7JLe3EZrI59BI) 

* **Generative Adversarial Networks for De Novo Molecular Design** [2021]  
Lee, Y.J., Kahng, H. and Kim, S.B.,  
[Molecular Informatics 40.10 (2021)](https://doi.org/10.1002/minf.202100045) |  [code](https://github.com/dudwojae/SMILES-MaskGAN) 

* **De-novo generation of novel phenotypically active molecules for Chagas disease from biological signatures using AI-driven generative chemistry** [2021]   
Pikusa, Michal, Olivier René, Sarah Williams, Yen-Liang Chen, Eric Martin, William J. Godinez, Srinivasa PS Rao, W. Armand Guiguemde, and Florian Nigsch.   
[bioRxiv (2021)](https://www.biorxiv.org/content/10.1101/2021.12.10.472084v1) |  [code](https://github.com/Novartis/pqsar2cpd) 

* **Mol-CycleGAN: a generative model for molecular optimization** [2020]  
Maziarka, Łukasz, Agnieszka Pocha, Jan Kaczmarczyk, Krzysztof Rataj, Tomasz Danel, and Michał Warchoł  
[J Cheminform 12, 2 (2020)](https://jcheminf.biomedcentral.com/articles/10.1186/s13321-019-0404-1) |  [code](https://github.com/ardigen/mol-cycle-gan) 

* **MolGAN: An implicit generative model for small molecular graph** [2018]  
De Cao, N. and Kipf, T.,  
[arXiv:1805.11973 (2018)](https://arxiv.org/abs/1805.11973) |  [code](https://github.com/yongqyu/MolGAN-pytorch) 

* **Objective-Reinforced Generative Adversarial Networks (ORGAN) for Sequence Generation Models** [2017]  
Guimaraes, G.L., Sanchez-Lengeling, B., Outeiral, C., Farias, P.L.C. and Aspuru-Guzik, A.,  
[ arXiv:1705.10843 (2017)](https://arxiv.org/abs/1705.10843) |  [code](https://github.com/gablg1/ORGAN) 






### Flow-based





* **Efficient 3D Molecular Generation with Flow Matching and Scale Optimal Transport** [2024]   
Irwin, Ross, Alessandro Tibo, Jon-Paul Janet, and Simon Olsson.  
[	arXiv:2406.07266 (2024)](https://arxiv.org/abs/2406.07266)  

* **RGFN: Synthesizable Molecular Generation Using GFlowNets** [2024]   
Koziarski, Michal, Andrei Rekesh, Dmytro Shevchuk, Almer van der Sloot, Piotr Gai'nski, Yoshua Bengio, Cheng-Hao Liu, Mike Tyers and Robert A. Batey.  
[	arXiv:2406.08506 (2024)](https://arxiv.org/abs/2406.08506)  

* **Mixed Continuous and Categorical Flow Matching for 3D De Novo Molecule Generation** [2024]   
Dunn, Ian, and David Ryan Koes.  
[arXiv:2404.19739 (2024)](https://arxiv.org/abs/2404.19739) |  [code](https://github.com/dunni3/FlowMol) 

* **PocketFlow is a data-and-knowledge-driven structure-based molecular generative model** [2024]   
Shengyong Yang, Yuanyuan Jiang, Guo Zhang et al.  
[Nat Mach Intell (2024)](https://doi.org/10.1038/s42256-024-00808-8) |  [Research Square. PREPRINT. (2023)](https://www.researchsquare.com/article/rs-3077992/v1) |  [code](https://github.com/Saoge123/PocketFlow) 

* **High-Temperature Polymer Dielectrics Designed Using an Invertible Molecular Graph Generative Model** [2023]   
Di-Fan Liu, Yong-Xin Zhang, Wen-Zhuo Dong, Qi-Kun Feng, Shao-Long Zhong, and Zhi-Min Dang.   
[J. Chem. Inf. Model. (2023)](https://doi.org/10.1021/acs.jcim.3c01572) |  [code](https://github.com/pfnet-research/graph-nvp) 

* **TacoGFN: Target Conditioned GFlowNet for Structure-Based Drug Design** [2023]   
Tony Shen, Mohit Pandey, Martin Ester.  
[arXiv:2310.03223. (2023)](https://arxiv.org/abs/2310.03223v1) 

* **PocketFlow: an autoregressive flow model incorporated with chemical knowledge for generating drug-like molecules inside protein pockets** [2023]   
Shengyong Yang, Yuanyuan Jiang, Guo Zhang et al.  
[Research Square. PREPRINT. (2023)](https://www.researchsquare.com/article/rs-3077992/v1) |  [code](https://github.com/Saoge123/PocketFlow) 

* **FFLOM: A Flow-Based Autoregressive Model for Fragment-to-Lead Optimization** [2023]   
Jieyu Jin, Dong Wang, Guqin Shi, Jingxiao Bao, Jike Wang, Haotian Zhang, Peichen Pan, Dan Li, Xiaojun Yao, Huanxiang Liu, Tingjun Hou, and Yu Kang  
[J. Med. Chem. (2023)](https://doi.org/10.1021/acs.jmedchem.3c01009) |  [code](https://github.com/JenniferKim09/FFLOM) 

* **Semi-Equivariant conditional normalizing flows, with applications to target-aware molecule generation** [2023]  
Rozenberg, Eyal, and Daniel Freedman.   
[Machine Learning: Science and Technology (2023)](https://iopscience.iop.org/article/10.1088/2632-2153/ace58c/meta) |  [arXiv:2304.06779 (2023)](https://arxiv.org/abs/2304.06779) 

* **Multi-view deep learning based molecule design and structural optimization accelerates the SARS-CoV-2 inhibitor discovery** [2022]  
Chao Pang , Yu Wang , Yi Jiang , Ruheng Wang , Ran Su  , and Leyi Wei.  
[arXiv:2212.01575 (2022)](https://arxiv.org/abs/2212.01575) |  [code](https://github.com/Pang-chao/MEDICO) 

* **Biological Sequence Design with GFlowNets** [2022]  
Jain, M., Bengio, E., Hernandez-Garcia, A., Rector-Brooks, J., Dossou, B.F., Ekbote, C.A., Fu, J., Zhang, T., Kilgour, M., Zhang, D. and Simine, L.  
[International Conference on Machine Learning. PMLR, (2022)](https://proceedings.mlr.press/v162/jain22a.html) |  [code](https://github.com/MJ10/BioSeq-GFN-AL) 

* **FastFlows: Flow-Based Models for Molecular Graph Generation** [2022]  
Frey, N.C., Gadepally, V. and Ramsundar, B.  
[arXiv:2201.12419 (2022)](https://arxiv.org/abs/2201.12419)

* **MoFlow: An Invertible Flow Model for Generating Molecular Graphs** [2020]  
Zang, Chengxi, and Fei Wang.  
[KDD '20 (2020)](https://dl.acm.org/doi/abs/10.1145/3394486.3403104) |  [code](https://github.com/calvin-zcx/moflow) 

* **GraphNVP: an Invertible Flow-based Model for Generating Molecular Graphs** [2020]   
Madhawa, K., Ishiguro, K., Nakago, K. and Abe, M.  
[arXiv:1905.11600 (2019)](https://openreview.net/forum?id=ryxQ6T4YwB)










### Prompt-Based






* **PromptSMILES: Prompting for scaffold decoration and fragment linking in chemical language models** [2024]   
  Thomas, Morgan, Mazen Ahmad, Gary Tresadern, and Gianni de Fabritiis.   
  [chemrxiv-2024-z5jnt (2024)](https://doi.org/10.26434/chemrxiv-2024-z5jnt)  |  [code](https://github.com/compsciencelab/PromptSMILES) 

* **Dual-Space Optimization: Improved Molecule Sequence Design by Latent Prompt Transformer** [2024]   
  Deqian Kong and Yuhao Huang and Jianwen Xie and Edouardo Honig and Ming Xu and Shuanghong Xue and Pei Lin and Sanping Zhou and Sheng Zhong and Nanning Zheng and Ying Nian Wu.   
  [	arXiv:2402.17179 (2024)](https://arxiv.org/abs/2402.17179) 

* **Molecule Design by Latent Prompt Transformer** [2023]   
 Kong, D., Huang, Y., Xie, J. and Wu, Y.N.   
  [arXiv:2310.03253 (2023)](https://arxiv.org/abs/2310.03253)










### Score-Based




* **Exploring Chemical Space with Score-based Out-of-distribution Generation** [2023]   
 Lee, Seul, Jaehyeong Jo, and Sung Ju Hwang.    
  [arXiv:2206.07632v3](https://arxiv.org/abs/2206.07632) | [code](https://github.com/SeulLee05/MOOD) 

* **Score-Based Generative Models for Molecule Generation** [2022]  
Gnaneshwar, Dwaraknath, et al.  
[arXiv:2203.04698 (2022)](https://arxiv.org/abs/2203.04698) 








### Energy-based






* **Molecular design with automated quantum computing-based deep learning and optimization**   [2023]   
Ajagekar, Akshay, and Fengqi You.   
[npj Comput Mater 9, 143 (2023)](https://doi.org/10.1038/s41524-023-01099-0) |  [code](https://github.com/PEESEgroup/qc-camd) 

* **Molecule Design by Latent Space Energy-Based Modeling and Gradual Distribution Shifting**   [2023]   
Deqian Kong, Bo Pang, Tian Han, Ying Nian Wu   
[arXiv:2306.14902v1](https://arxiv.org/abs/2306.14902) |  [code](https://github.com/deqiankong/SGDS) 

* **Energy-based Generative Models for Target-specific Drug Discovery** [2022]  
Li, Junde, Collin Beaudoin, and Swaroop Ghosh.  
[arXiv:2212.02404 (2022)](https://arxiv.org/abs/2212.02404) |  [code](https://github.com/jundeli/TagMol) 

* **MOG: Molecular Out-of-distribution Generation with Energy-based Models** [2021]  
Lee, Seul, Dong Bok Lee, and Sung Ju Hwang.  
[Paper](https://openreview.net/forum?id=qkTEaJ9orc1) 








### Diffusion-based



* **DrugDiff - small molecule diffusion model with flexible guidance towards molecular properties** [204]   
Marie Oestreich, Erinc Merdivan, Michael Lee, Joachim L. Schultze, Marie Piraud, Matthias Becker.   
[bioRxiv 2024.07.17.603873 (2024)](https://doi.org/10.1101/2024.07.17.603873) |  [code](https://github.com/MarieOestreich/DrugDiff)

* **MolCRAFT: Structure-Based Drug Design in Continuous Parameter Space** [2024]   
 Qu, Yanru, Keyue Qiu, Yuxuan Song, Jingjing Gong, Jiawei Han, Mingyue Zheng, Hao Zhou, and Wei-Ying Ma.    
  [ICML (2024)](https://openreview.net/pdf/5026a68b7e1fc7485c73ca919d8dea934394f0d0.pdf)  |  [code](https://github.com/AlgoMole/MolCRAFT) 

* **PILOT: Equivariant diffusion for pocket conditioned de novo ligand generation with multi-objective guidance via importance sampling**   [2024]   
 Cremer, Julian, Tuan Le, Frank Noé, Djork-Arné Clevert, and Kristof T. Schütt.    
  [arXiv:2405.14925 (2024)](https://arxiv.org/abs/2405.14925)  

* **Diff-Shape: A Novel Constrained Diffusion Model for Shape based De Novo Drug Design**   [2024]   
 Lin, Jie, Mingyuan Xu, and Hongming Chen.   
  [chemrxiv-2024-km0h1 (2024)](https://doi.org/10.26434/chemrxiv-2024-km0h1)  

* **A Property-Guided Diffusion Model For Generating Molecular Graphs**   [2024]   
 Ma, Changsheng, Taicheng Guo, Qiang Yang, Xiuying Chen, Xin Gao, Shangsong Liang, Nitesh Chawla, and Xiangliang Zhang.    
  [ICASSP 2024-2024 IEEE International Conference on Acoustics, Speech and Signal Processing (ICASSP). IEEE (2024)](https://doi.org/10.1109/ICASSP48485.2024.10447350)  

* **A unified conditional diffusion framework for dual protein targets based bioactive molecule generation**   [2024]   
 Huang, Lei, Zheng Yuan, Huihui Yan, Rong Sheng, Linjing Liu, Fuzhou Wang, Weidun Xie et al.    
  [IEEE Transactions on Artificial Intelligence (2024)](https://doi.org/10.1109/TAI.2024.3387402)  |  [arXiv:2306.13957 (2023)](https://arxiv.org/abs/2306.13957) 

* **Equivariant 3D-conditional diffusion model for molecular linker design** [2024]   
 Igashov, I., Stärk, H., Vignac, C. et al.    
  [Nat Mach Intell (2024)](https://doi.org/10.1038/s42256-024-00815-9)  |  [code](https://github.com/igashov/DiffLinker) 

* **Sculpting Molecules in Text-3D Space: A Flexible Substructure Aware Framework for Text-Oriented Molecular Optimization** [2024]   
 Zhang, Kaiwei, Yange Lin, Guangcheng Wu, Yuxiang Ren, Xuecang Zhang, Bo Wang, and Xiao-Yu Zhang.    
  [Research Square (2024)](https://www.researchsquare.com/article/rs-4023429/v1)  

* **AUTODIFF: Autoregressive Diffusion Modeling for Structure-based Drug Design** [2024]   
 Li, Xinze, Penglei Wang, Tianfan Fu, Wenhao Gao, Chengtao Li, Leilei Shi, and Junhong Liu.    
  [arXiv:2404.02003 (2024)](https://arxiv.org/html/2404.02003v1)  

* **MolSnapper: Conditioning Diffusion for Structure Based Drug Design** [2024]   
 Ziv, Yael, Brian Marsden, and Charlotte Deane.    
  [bioRxiv (2024)](https://doi.org/10.1101/2024.03.28.586278)  |  [code](https://github.com/oxpig/MolSnapper) 

* **De Novo Molecule Generation with Graph Latent Diffusion Model** [2024]   
 Wang, Conghao, Hiok Hian Ong, Shunsuke Chiba, and Jagath C. Rajapakse.    
  [ICASSP 2024-2024 IEEE International Conference on Acoustics, Speech and Signal Processing (ICASSP). IEEE (2024)](https://doi.org/10.1109/ICASSP48485.2024.10447480)  

* **A dual diffusion model enables 3D molecule generation and lead optimization based on target pockets** [2024]   
 Huang, L., Xu, T., Yu, Y. et al.    
  [Nat Commun 15, 2657 (2024)](https://doi.org/10.1038/s41467-024-46569-1)  |  [code](https://github.com/Layne-Huang/PMDM) 

* **3M-Diffusion: Latent Multi-Modal Diffusion for Text-Guided Generation of Molecular Graphs** [2024]   
 Huaisheng Zhu, Teng Xiao, Vasant G Honavar.    
  [	arXiv:2403.07179. (2024)](https://arxiv.org/abs/2403.07179)  |  [code](https://github.com/huaishengzhu/3MDiffusion) 

* **DiffDec: Structure-Aware Scaffold Decoration with an End-to-End Diffusion Model** [2024]   
 Xie, Junjie, Sheng Chen, Jinping Lei, and Yuedong Yang.    
  [J. Chem. Inf. Model. (2024)](https://doi.org/10.1021/acs.jcim.3c01466)  |  [code](https://github.com/biomed-AI/DiffDec) 

* **Functional-Group-Based Diffusion for Pocket-Specific Molecule Generation and Elaboration** [2024]   
 Lin, Haitao, Yufei Huang, Odin Zhang, Yunfan Liu, Lirong Wu, Siyuan Li, Zhiyuan Chen, and Stan Z. Li.    
  [Advances in Neural Information Processing Systems 36 (2024)](https://proceedings.neurips.cc/paper_files/paper/2023/hash/6cdd4ce9330025967dd1ed0bed3010f5-Abstract-Conference.html)  

* **Binding-Adaptive Diffusion Models for Structure-Based Drug Design** [2024]   
 Zhilin Huang, Ling Yang, Zaixi Zhang, Xiangxin Zhou, Yu Bao, Xiawu Zheng, Yuwei Yang, Yu Wang, Wenming Yang.    
  [AAAI 2024 (2024)](https://arxiv.org/abs/2402.18583)  |  [code](https://github.com/YangLing0818/BindDM) 

* **Field-based Molecule Generation** [2024]   
 Dumitrescu, Alexandru, Dani Korpela, Markus Heinonen, Yogesh Verma, Valerii Iakovlev, Vikas Garg, and Harri Lähdesmäki.    
  [arXiv:2402.15864 (2024)](https://arxiv.org/abs/2402.15864) 

* **Text-Guided Molecule Generation with Diffusion Language Model** [2024]   
 Gong, Haisong, Qiang Liu, Shu Wu, and Liang Wang.    
  [arXiv:2402.13040 (2024)](https://arxiv.org/abs/2402.13040)  |  [code](https://github.com/Deno-V/tgm-dlm) 

* **Inverse Molecular Design with Multi-Conditional Diffusion Guidance** [2024]   
 Liu, Gang, Jiaxin Xu, Tengfei Luo, and Meng Jiang.    
  [arXiv:2401.13858 (2024)](https://arxiv.org/abs/2401.13858)  |  [code](https://github.com/liugangcode/MCD) 

* **Navigating the Design Space of Equivariant Diffusion-Based Generative Models for De Novo 3D Molecule Generation** [2024]   
 Le, Tuan, Julian Cremer, Frank Noé, Djork-Arné Clevert, and Kristof Schütt.    
  [International Conference on Learning Representations (ICLR).  (2024)](https://openreview.net/pdf?id=kzGuiRXZrQ)  |  [code](https://github.com/tuanle618/eqgat-diff) 

* **KGDiff: towards explainable target-aware molecule generation with knowledge guidance** [2023]   
 Hao Qian, Wenjing Huang, Shikui Tu, Lei Xu.    
  [Briefings in Bioinformatics.  (2023)](https://doi.org/10.1093/bib/bbad435)  |  [code](https://github.com/CMACH508/KGDiff) 

* **STRIDE: Structure-guided Generation for Inverse Design of Molecules** [2023]   
 Zaman, Shehtab, Denis Akhiyarov, Mauricio Araya-Polo, and Kenneth Chiu.    
  [NeurIPS 2023 AI for Science Workshop.  (2023)](https://openreview.net/forum?id=DqJThcBJ6P&noteId=JjvFlUIseN) 

* **LinkerNet: Fragment Poses and Linker Co-Design with 3D Equivariant Diffusion** [2023]   
 Guan, Jiaqi, Xingang Peng, PeiQi Jiang, Yunan Luo, Jian Peng, and Jianzhu Ma    
  [NeurIPS 2023.  (2023)](https://openreview.net/forum?id=6EaLIw3W7c) |  [code](https://github.com/guanjq/LinkerNet) 

* **Autoregressive fragment-based diffusion for pocket-aware ligand design** [2023]   
 Ghorbani, Mahdi, Leo Gendelev, Paul Beroza, and Michael Keiser.    
  [NeurIPS 2023 Generative AI and Biology (GenBio) Workshop.  (2023)](https://openreview.net/forum?id=E3HN48zjam) |  [code](https://github.com/ghorbanimahdi73/autofragdiff) 

* **Diffusing on Two Levels and Optimizing for Multiple Properties: A Novel Approach to Generating Molecules with Desirable Properties** [2023]   
 Guo, Siyuan, Jihong Guan, and Shuigeng Zhou.    
  [arXiv:2310.04463 (2023)](https://arxiv.org/abs/2310.04463) 

* **DiffDec: Structure-Aware Scaffold Decoration with an End-to-End Diffusion** [2023]   
 Xie, Junjie, Sheng Chen, Jinping Lei, and Yuedong Yang.    
  [bioRxiv (2023)](https://doi.org/10.1101/2023.10.08.561377) 

* **Generative Design of inorganic compounds using deep diffusion language models** [2023]   
 Rongzhi Dong and Nihang Fu and dirisuriya M. D. Siriwardane and Jianjun Hu.    
  [	arXiv:2310.00475  (2023)](https://arxiv.org/abs/2310.00475) 

* **Navigating the Design Space of Equivariant Diffusion-Based Generative Models for De Novo 3D Molecule Generation** [2023]   
 Tuan Le and Julian Cremer and Frank No'e and Djork-Arn'e Clevert and Kristof Schutt.    
  [arXiv:2309.17296v1  (2023)](https://doi.org/10.48550/arXiv.2309.17296) 

* **Guided Diffusion for molecular generation with interaction prompt** [2023]   
 Wu Song, Peng Wu, Huabin Du, Yingchao Yan, Chen Bai    
  [bioRxiv  (2023)](https://doi.org/10.1101/2023.09.11.557141) |  [data](https://bits.csb.pitt.edu/files/crossdock2020/) 

* **Shape-conditioned 3D Molecule Generation via Equivariant Diffusion Models** [2023]   
 Chen, Ziqi, Bo Peng, Srinivasan Parthasarathy, and Xia Ning    
  [arXiv:2308.11890 (2023)](https://arxiv.org/abs/2308.11890) 

* **DiffSeqMol: A Non-Autoregressive Diffusion-Based Approach for Molecular Sequence Generation and Optimization** [2023]   
 Zixu Wang, Yangyang Chen*, Xiucai Ye.    
  [chemrxiv-2023-ltr9v-v2.  (2023)](https://doi.org/10.26434/chemrxiv-2023-ltr9v-v2) |  [code](https://github.com/viko-3/DiffSeqMol) 

* **MolDiff: Addressing the Atom-Bond Inconsistency Problem in 3D Molecule Diffusion Generation** [2023]   
 Peng, Xingang, Jiaqi Guan, Qiang Liu, and Jianzhu Ma.    
  [ICML  (2023)](https://proceedings.mlr.press/v202/peng23b.html) |  [code](https://github.com/pengxingang/MolDiff) 

* **DiffHopp: A Graph Diffusion Model for Novel Drug Design via Scaffold Hopping** [2023]   
 Torge, Jos, Charles Harris, Simon V. Mathis, and Pietro Lió.    
  [ICML(2023)](https://icml-compbio.github.io/2023/papers/WCBICML2023_paper69.pdf) |  [code](https://github.com/jostorge/diffusion-hopping) 

* **Coarse-to-Fine: a Hierarchical Diffusion Model for Molecule Generation in 3D** [2023]   
 Qiang, Bo, Yuxuan Song, Minkai Xu, Jingjing Gong, Bowen Gao, Hao Zhou, Wei-Ying Ma, and Yanyan Lan.    
  [ICML  (2023)](https://proceedings.mlr.press/v202/qiang23a.html) |  [code](https://github.com/qiangbo1222/HierDiff) 

* **DecompDiff: Diffusion Models with Decomposed Priors for Structure-Based Drug Design** [2023]   
 Guan, Jiaqi, Xiangxin Zhou, Yuwei Yang, Yu Bao, Jian Peng, Jianzhu Ma, Qiang Liu, Liang Wang, and Quanquan Gu.    
  [ICML (2023)](https://openreview.net/forum?id=9qy9DizMlr) |  [code](https://github.com/bytedance/DecompDiff) 

* **Learning Joint 2D & 3D Diffusion Models for Complete Molecule Generation** [2023]   
 Huang, Han, Leilei Sun, Bowen Du, and Weifeng Lv.    
  [arXiv:2305.12347 (2023)](https://arxiv.org/abs/2305.12347) |  [code](https://github.com/GRAPH-0/JODO) 

* **Conditional Diffusion Based on Discrete Graph Structures for Molecular Graph Generation** [2023]   
 Huang, Han, Leilei Sun, Bowen Du, and Weifeng Lv.    
  [arXiv:2301.00427 (2023)](https://arxiv.org/abs/2301.00427) |  [code](https://github.com/GRAPH-0/CDGS) 

* **SILVR: Guided Diffusion for Molecule Generation** [2023]   
 Runcie, Nicholas T., and Antonia SJS Mey.    
  [J. Chem. Inf. Model. (2023)](https://pubs.acs.org/doi/10.1021/acs.jcim.3c00667) |  [arXiv:2304.10905v1](https://arxiv.org/abs/2304.10905) |  [code](https://github.com/meyresearch/SILVR) 

* **Guided Diffusion for Inverse Molecular Design** [2023]   
 Weiss, Tomer, Luca Cosmo, Eduardo Mayo Yanes, Sabyasachi Chakraborty, Alex M. Bronstein, and Renana Gershoni-Poranne.    
  [Nat Comput Sci (2023)](https://doi.org/10.1038/s43588-023-00532-0) |   [chemrxiv-2023-z8ltp](https://doi.org/10.26434/chemrxiv-2023-z8ltp) |  [code](https://github.com/tomer196/GaUDI) 

* **Generative Discovery of Novel Chemical Designs using Diffusion Modeling and Transformer Deep Neural Networks with Application to Deep Eutectic Solvents** [2023]   
 Luu, Rachel K., Marcin Wysokowski, and Markus J. Buehler.    
  [arXiv:2304.12400v1](https://arxiv.org/abs/2304.12400) |  [code](https://github.com/lamm-mit/MoleculeDiffusionTransformer) 

* **Geometric Latent Diffusion Models for 3D Molecule Generation** [2023]   
  Minkai Xu, Alexander Powers, Ron Dror, Stefano Ermon, Jure Leskovec   
  [arXiv:2305.01140v1](https://arxiv.org/abs/2305.01140) |  [code](https://github.com/MinkaiXu/GeoLDMf) 

* **3D Equivariant Diffusion for Target-Aware Molecule Generation and Affinity Prediction** [2023]   
  Guan, Jiaqi, Wesley Wei Qian, Xingang Peng, Yufeng Su, Jian Peng, and Jianzhu Ma.   
  [ICLR (2023)](https://openreview.net/forum?id=kJqXEPXMsE0) |  [code](https://github.com/guanjq/targetdiff) 

* **Structure-based Drug Design with Equivariant Diffusion Models** [2023]  
Schneuing, A., Du, Y., Harris, C., Jamasb, A., Igashov, I., Du, W., ... & Correia, B.   
[arXiv:2210.13695 (2022)](https://openreview.net/forum?id=uKmuzIuVl8z) |  [code](https://github.com/arneschneuing/DiffSBDD) 

* **Equivariant 3D-Conditional Diffusion Models for Molecular Linker Desig** [2023]  
Igashov, I., Stärk, H., Vignac, C., Satorras, V.G., Frossard, P., Welling, M., Bronstein, M. and Correia, B.,   
[arXiv:2210.05274 (2022)](https://openreview.net/forum?id=cnsHSSLnHVV) |  [code](https://github.com/igashov/DiffLinker) 

* **MiDi: Mixed Graph and 3D Denoising Diffusion for Molecule Generation** [2023]  
Vignac, Clement, Nagham Osman, Laura Toni, and Pascal Frossard.  
[arXiv:2302.09048 (2023)](https://arxiv.org/abs/2302.09048) |  [code](https://github.com/cvignac/MiDi) 

* **Geometry-Complete Diffusion for 3D Molecule Generation** [2023]  
Morehead, Alex, and Jianlin Cheng.  
[arXiv:2302.04313 (2023)](https://arxiv.org/abs/2302.04313) |  [code](https://github.com/BioinfoMachineLearning/bio-diffusion) 

* **MDM: Molecular Diffusion Model for 3D Molecule Generation** [2022]  
Huang, Lei, Hengtong Zhang, Tingyang Xu, and Ka-Chun Wong.  
[arXiv:2209.05710 (2022)](https://arxiv.org/abs/2209.05710)

* **Diffusion-based Molecule Generation with Informative Prior Bridges** [2022]  
Lemeng Wu, Chengyue Gong, Xingchao Liu, Mao Ye, Qiang Liu  
[NeurIPS (2022)](https://openreview.net/forum?id=QagNEt9k8Vi)

* **Equivariant Diffusion for Molecule Generation in 3D** [2022]  
Hoogeboom, Emiel, Vıctor Garcia Satorras, Clément Vignac, and Max Welling.  
[International Conference on Machine Learning. PMLR, (2022)](https://proceedings.mlr.press/v162/hoogeboom22a.html) |  [code](https://github.com/ehoogeboom/e3_diffusion_for_molecules) 









### RL-based


* **BindGPT: A Scalable Framework for 3D Molecular Design via Language Modeling and Reinforcement Learning**  [2024]   
 Zholus, Artem, Maksim Kuznetsov, Roman Schutski, Rim Shayakhmetov, Daniil Polykovskiy, Sarath Chandar, and Alex Zhavoronkov.   
  [arXiv:2406.03686 (2024)](https://arxiv.org/abs/2406.03686)  

* **De novo drug design as GPT language modeling: large chemistry models with supervised and reinforcement learning**  [2024]   
 Ye, Gavin.   
  [ Journal of Computer-Aided Molecular Design 38.1 (2024)](https://link.springer.com/article/10.1007/s10822-024-00559-z) |  [code](https://huggingface.co/Coconut104/EfficacyGPT-DrugDesign) 

* **Augmented Memory: Sample-Efficient Generative Molecular Design with Reinforcement Learning**  [2024]   
 Guo, Jeff, and Philippe Schwaller.   
  [JACS Au (2024)](https://doi.org/10.1021/jacsau.4c00066) |  [code](https://github.com/schwallergroup/augmented_memory) 

* **Mol-AIR: Molecular Reinforcement Learning with Adaptive Intrinsic Rewards for Goal-directed Molecular Generation**  [2024]   
 Park, Jinyeong, Jaegyoon Ahn, Jonghwan Choi, and Jibum Kim.   
  [arXiv:2403.20109 (2024)](https://arxiv.org/abs/2403.20109) |  [code](https://github.com/DevSlem/Mol-AIR) 

* **Evaluation of Reinforcement Learning in Transformer-based Molecular Design**  [2024]   
 He J, Tibo A, Janet JP, Nittinger E, Tyrchan C, Czechtizky W, et al.   
  [chemrxiv-2024-r9ljm (2024)](https://doi.org/10.26434/chemrxiv-2024-r9ljm) |  [code](https://github.com/MolecularAI/transformer_rl) 

* **Structure-Based Drug Design via 3D Molecular Generative Pre-training and Sampling** [2024]   
 Yang, Yuwei, Siqi Ouyang, Xueyu Hu, Meihua Dang, Mingyue Zheng, Hao Zhou, and Lei Li.    
  [arXiv:2402.14315 (2024)](https://arxiv.org/abs/2402.14315) 

* **Sample Efficient Reinforcement Learning with Active Learning for Molecular Design**  [2024]   
 Janet, Jon Paul, Michael Dodds, Jeff Guo, Thomas Löhr, Alessandro Tibo, and Ola Engkvist.   
  [Chemical Science (2024)](https://doi.org/10.1039/D3SC04653B) |  [code](https://www.rsc.org/suppdata/d3/sc/d3sc04653b/d3sc04653b2.pdf) 

* **FREED++: Improving RL Agents for Fragment-Based Molecule Generation by Thorough Reproduction** [2024]   
  Telepov, Alexander, Artem Tsypin, Kuzma Khrabrov, Sergey Yakukhnov, Pavel Strashnov, Petr Zhilyaev, Egor Rumiantsev et al.    
  [arXiv:2401.09840 (2024)](https://arxiv.org/abs/2401.09840) |  [code](https://github.com/AIRI-Institute/FFREED) 

* **Local Scaffold Diversity-Contributed Generator for Discovering Potential NLRP3 Inhibitors** [2024]   
  Weichen Bo, Yangqin Duan, Yurong Zou, Ziyan Ma, Tao Yang, Peng Wang, Tao Guo, Zhiyuan Fu, Jianmin Wang, Linchuan Fan, Jie liu, Taijin Wang, and Lijuan Chen.   
  [J. Chem. Inf. Model. (2024)](https://doi.org/10.1021/acs.jcim.3c01818) |  [code](https://github.com/wichen-2022/LSDC) 

* **Using Generative Modeling to Endow with Potency Initially Inert Compounds with Good Bioavailability and Low Toxicity** [2024]   
  Robert I. Horne, Jared Wilson-Godber, Alicia González Díaz, Z. Faidon Brotzakis, Srijit Seal, Rebecca C. Gregory, Andrea Possenti, Sean Chia, and Michele Vendruscolo.  
  [J. Chem. Inf. Model.  (2024)](https://doi.org/10.1021/acs.jcim.3c01777) | [code](https://github.com/Jaredwg2000/MolDQN_CNS)  

* **Goal-directed molecule generation with fine-tuning by policy gradient** [2024]   
  Sha, Chunli, and Fei Zhu.   
  [Expert Systems with Applications (2024)](https://doi.org/10.1016/j.eswa.2023.123127)  

* **GRELinker: A Graph-based Generative Model for Molecular Linker Design with Reinforcement and Curriculum Learning** [2024]   
  Zhang, Hao, Jinchao Huang, Junjie Xie, Weifeng Huang, Yuedong Yang, Mingyuan Xu, Jinping Lei, and Hongming Chen.  
  [J. Chem. Inf. Model.  (2024)](https://doi.org/10.1021/acs.jcim.3c01700) | [code](https://github.com/howzh728/GRELinker)  

* **Molecular generation strategy and optimization based on A2C reinforcement learning in de novo drug design** [2023]   
  Wang, Qian, Zhiqiang Wei, Xiaotong Hu, Zhuoya Wang, Yujie Dong, and Hao Liu.   
  [Bioinformatics: btad693.  (2023)](https://doi.org/10.1093/bioinformatics/btad693) |  [code](https://github.com/wq-sunshine/MomdTDSRL) 

* **Tree-Invent: A Novel Multipurpose Molecular Generative Model Constrained with a Topological Tree** [2023]   
  Mingyuan Xu, Hongming Chen.   
  [J. Chem. Inf. Model.  (2023)](https://doi.org/10.1021/acs.jcim.3c01626) |  [code](https://github.com/MingyuanXu/Tree-Invent) 

* **De novo Drug Design using Reinforcement Learning with Multiple GPT Agents** [2023]   
  Hu, Xiuyuan, Guoqing Liu, Yang Zhao, and Hao Zhang.   
  [NeurIPS 2023  (2023)](https://openreview.net/forum?id=1B6YKnHYBb) |  [code](https://github.com/HXYfighter/MolRL-MGPT) 

* **REINVENT4: Modern AI–Driven Generative Molecule Design** [2023]   
  Loeffler H, He J, Tibo A, Janet JP, Voronov A, Mervin L, et al.   
  [chemrxiv-2023-xt65x  (2023)](https://doi.org/10.26434/chemrxiv-2023-xt65x) |  [code](https://github.com/MolecularAI/REINVENT4) 

* **Optimization of binding affinities in chemical space with transformer and deep reinforcement learning** [2023]   
  Xu, Xiaopeng, Juexiao Zhou, Chen Zhu, Qing Zhan, Zhongxiao Li, Ruochi Zhang, Yu Wang, Xingyu Liao, and Xin Gao.   
  [chemrxiv-2023-7v4sw (2023)](https://doi.org/10.26434/chemrxiv-2023-7v4sw) |  [code](https://github.com/charlesxu90/sgpt) 

* **A flexible data-free framework for structure-based de novo drug design with reinforcement learning** [2023]   
  Hongyan Du, Dejun Jiang, Odin Zhang, Zhenxing Wu, Junbo Gao, Xujun Zhang, Xiaorui Wang, Yafeng Deng, Yu Kang, Dan Li, Peichen Pan, Chang-Yu Hsieh, Tingjun Hou.   
  [Chemical Science (2023)](https://doi.org/10.1039/D3SC04091G) |  [code](https://github.com/Brian-hongyan/3D-MCTS) 

* **Searching for High-Value Molecules Using Reinforcement Learning and Transformers** [2023]   
  Raj Ghugare and Santiago Miret and Adriana Hugessen and Mariano Phielipp and Glen Berseth.   
  [arXiv:2310.02902 (2023)](https://arxiv.org/abs/2310.02902)

* **Molecular De Novo Design through Transformer-based Reinforcement Learning** [2023]   
  Feng, Tao, Pengcheng Xu, Tianfan Fu, Siddhartha Laghuvarapu, and Jimeng Sun.   
  [arXiv:2310.05365 (2023)](https://arxiv.org/abs/2310.05365)

* **Integrating synthetic accessibility with AI-based generative drug design**  [2023]   
 Parrot, M., Tajmouati, H., da Silva, V.B.R. et al.   
  [J Cheminform 15, 83 (2023)](https://doi.org/10.1186/s13321-023-00742-8)  |  [code](https://github.com/iktos/generation-under-synthetic-constraint/) 

* **Deep learning driven de novo drug design based on gastric proton pump structures** [2023]   
  Abe, K., Ozako, M., Inukai, M. et al.   
  [Commun Biol 6, 956 (2023)](https://doi.org/10.1038/s42003-023-05334-8) |  [code](https://doi.org/10.1038/s42003-023-05334-8) 

* **3D based generative PROTAC linker design with reinforcement learning** [2023]    
  Li, Baiqing, Ting Ran, and Hongming Chen.   
  [Briefings in Bioinformatics (2023)](https://doi.org/10.1093/bib/bbad323) |  [code](https://github.com/jidushanbojue/Protac-invent) 

* **Exploration and Exploitation Approaches Based on Generative Machine Learning to Identify Potent Small Molecule Inhibitors of α-Synuclein Secondary Nucleation** [2023]    
  Horne, Robert I., Mhd Hussein Murtada, Donghui Huo, Z. Faidon Brotzakis, Rebecca C. Gregory, Andrea Possenti, Sean Chia, and Michele Vendruscolo.   
  [Journal of Chemical Theory and Computation (2023)](https://doi.org/10.1021/acs.jctc.2c01303) |  [code](https://github.com/husseinmur/GraphINVENT-CNS) 

* **ReBADD-SE: Multi-objective molecular optimisation using SELFIES fragment and off-policy self-critical sequence training** [2023]   
  Choi, Jonghwan, Sangmin Seo, Seungyeon Choi, Shengmin Piao, Chihyun Park, Sung Jin Ryu, Byung Ju Kim, and Sanghyun Park.   
  [Computers in Biology and Medicine 157 (2023)](https://doi.org/10.1016/j.compbiomed.2023.106721) |  [code](https://github.com/mathcom/ReBADD-SE) 

* **LOGICS: Learning optimal generative distribution for designing de novo chemical structures** [2023]   
  Bae, B., Bae, H. & Nam, H.   
  [J Cheminform 15, 77 (2023)](https://doi.org/10.1186/s13321-023-00747-3) |  [code](https://github.com/GIST-CSBL/LOGICS) 

* **3D Based Generative PROTAC Linker Design with Reinforcement Learning**  [2023]   
 baiqing li, and Hongming Chen.   
  [chemrxiv-2023-j740w (2023)](https://doi.org/10.26434/chemrxiv-2023-j740w) |  [code](https://github.com/jidushanbojue/Protac-invent) 

* **De Novo Design of κ-Opioid Receptor Antagonists Using a Generative Deep-Learning Framework**  [2023]   
 Salas-Estrada, Leslie, Davide Provasi, Xing Qiu, H. Umit Kaniskan, Xi-Ping Huang, Jeffrey DiBerto, Joao Marcelo Lamim Ribeiro, Jian Jin, Bryan L. Roth, and Marta Filizola.   
  [J. Chem. Inf. Model. (2023)](https://doi.org/10.1021/acs.jcim.3c00651) |  [code](https://github.com/insilicomedicine/GENTRL) 

* **Utilizing Reinforcement Learning for de novo Drug Design**  [2023]   
 Svensson, Hampus Gummesson, Christian Tyrchan, Ola Engkvist, and Morteza Haghir Chehreghani.   
  [arXiv:2303.17615 (2023)](https://arxiv.org/abs/2303.17615) |  [code](https://github.com/MolecularAI/SMILES-RL) 

* **De novo drug design by iterative multiobjective deep reinforcement learning with graph-based molecular quality assessment **  [2023]   
 Fang, Yi, Xiaoyong Pan, and Hong-Bin Shen.   
  [Bioinformatics 39.4 (2023)](https://doi.org/10.1093/bioinformatics/btad157) |  [code](https://github.com/yifang000/QADD) 

* **Generative Organic Electronic Molecular Design via Reinforcement Learning Integration with Quantum Chemistry: Tuning Singlet and Triplet Energy Energy Levels**  [2023]   
 Cheng-Han Li ,Daniel P. Tabor   
  [chemrxiv (2023)](https://doi.org/10.26434/chemrxiv-2023-bgcjg) |  [code](https://github.com/Tabor-Research-Group/reinvent_qc) 

* **De Novo Design of κ-Opioid Receptor Antagonists Using a Generative Deep Learning Framework** [2023]   
 Salas-Estrada, Leslie, Davide Provasi, Xing Qiu, H. Umit Kaniskan, Xi-Ping Huang, Jeffrey DiBerto, Joao Marcelo Lamim Ribeiro, Jian Jin, Bryan L. Roth, and Marta Filizola.   
  [bioRxiv (2023)](https://doi.org/10.1101/2023.04.25.537995) |  [code](https://github.com/insilicomedicine/GENTRL) 

* **Reinforcement Learning-Driven Linker Design via Fast Attention-based Point Cloud Alignment** [2023]   
 Neeser, Rebecca M., Mehmet Akdel, Daniel Kovtun, and Luca Naef.   
  [arXiv:2306.08166 (2023)](https://arxiv.org/abs/2306.08166) |  [code](https://github.com/aivant/ShapeLinker) 

* **De novo drug design based on Stack-RNN with multi-objective reward-weighted sum and reinforcement learning** [2023]   
Hu, P., Zou, J., Yu, J. et al.   
[ J Mol Model 29, 121 (2023)](https://doi.org/10.1007/s00894-023-05523-6) |  [code](https://github.com/PengWeiHu1/mul_RL) 

* **LS-MolGen: Ligand-and-Structure Dual-Driven Deep Reinforcement Learning for Target-Specific Molecular Generation Improves Binding Affinity and Novelty** [2023]  
 Li, Song, Chao Hu, Song Ke, Chenxing Yang, Jun Chen, Yi Xiong, Hao Liu, and Liang Hong.   
  [J. Chem. Inf. Model. （2023）](https://pubs.acs.org/doi/full/10.1021/acs.jcim.3c00587) |  [code](https://github.com/songleee/LS-MolGen) 

* **Molecule generation using transformers and policy gradient reinforcement learning** [2023]   
 Mazuz, E., Shtar, G., Shapira, B. et al.    
  [Sci Rep 13, 8799 (2023)](https://doi.org/10.1038/s41598-023-35648-w) |  [code](https://github.com/eyalmazuz/MolGen) 

* **Artificial Intelligence for Prediction of Biological Activities and Generation of molecular hits using Stereochemical Information** [2023]   
  Pereira, Tiago O., Maryam Abbasi, Rita I. Oliveira, Romina A. Guedes, Jorge AR Salvador, and Joel P. Arrais.   
  [Research Square.  (2023)](https://doi.org/10.21203/rs.3.rs-2499317/v1) |  [code](https://github.com/larngroup/targeted_generation_stereo) 

* **Augmented Memory: Capitalizing on Experience Replay to Accelerate De Novo Molecular Design** [2023]   
 Guo, Jeff, and Philippe Schwaller.    
  [chemrxiv-2023-qmqmq-v3](https://doi.org/10.26434/chemrxiv-2023-qmqmq-v2) |  [code](https://github.com/schwallergroup/augmented_memory) 

* **Generating Potential Protein-Protein Interaction Inhibitor Molecules Based on Physicochemical Properties** [2023]   
 Ohue, Masahito, Yuki Kojima, and Takatsugu Kosugi.   
  [Paper](https://www.preprints.org/manuscript/202305.0704/v1) |  [code](https://github.com/ohuelab/iPPI-REINVENT) 

* **Tree-Invent: A novel molecular generative model constrained with topological tree** [2023]   
 Mingyuan Xu, HongMing Chen.   
  [chemrxiv-2023-m77vk](https://doi.org/10.26434/chemrxiv-2023-m77vk) |  [code](https://github.com/MingyuanXu/Tree-Invent) 

* **Molecular Graph Generation by Decomposition and Reassembling**  [2023]   
 Yamada, Masatsugu, and Mahito Sugiyama.   
  [ACS omega (2023)](https://doi.org/10.1021/acsomega.3c01078) |  [code](https://github.com/Masatsugar/graph-decomposition-reassembling) 

* **De Novo Drug Design by Iterative Multi-Objective Deep Reinforcement Learning with Graph-based Molecular Quality Assessment** [2023]   
 Yi Fang, Xiaoyong Pan, Hong-Bin Shen.    
  [Bioinformatics 39.4 (2023)](https://doi.org/10.1093/bioinformatics/btad157) |  [code](https://github.com/yifang000/QADD) 

* **DrugEx v3: scaffold-constrained drug design with graph transformer-based reinforcement learning** [2023]  
  Liu, X., Ye, K., van Vlijmen, H.W.T. et al.   
  [ J Cheminform 15, 24 (2023)](https://doi.org/10.1186/s13321-023-00694-z) |  [code](https://github.com/CDDLeiden/DrugEx) 

* **COMA: efficient structure-constrained molecular generation using contractive and margin losses** [2023]  
 Choi, J., Seo, S. & Park, S.     
  [J Cheminform 15, 8 (2023)](https://doi.org/10.1186/s13321-023-00679-y) |  [code](https://github.com/mathcom/COMA) 

* **Generative and reinforcement learning approaches for the automated de novo design of bioactive compounds** [2022]  
Korshunova, M., Huang, N., Capuzzi, S. et al.  
[Commun Chem 5, 129 (2022)](https://doi.org/10.1038/s42004-022-00733-0) |  [code](https://github.com/isayevlab/rl_experiments) 

* **Augmented Hill-Climb increases reinforcement learning efficiency for language-based de novo molecule generation** [2022]  
Thomas, M., O’Boyle, N.M., Bender, A. et al.  
[ J Cheminform 14, 68 (2022)](https://doi.org/10.1186/s13321-022-00646-z) |  [code](https://github.com/MorganCThomas/SMILES-RNN) 

* **Predicting chemical structure using reinforcement learning with a stack-augmented conditional variational autoencoder** [2022]  
  Kim, H., Ko, S., Kim, B.J. *et al.*  
  [J Cheminform 14, 83 (2022)](https://jcheminf.biomedcentral.com/articles/10.1186/s13321-022-00666-9) |  [code](https://github.com/HwanheeKim813/stack_CVAE) 

* **De Novo Drug Design Using Reinforcement Learning with Graph-Based Deep Generative Models** [2022]  
Atance, S.R., Diez, J.V., Engkvist, O., Olsson, S. and Mercado, R.  
[J. Chem. Inf. Model. 2022, 62, 20, 4863–4872](https://pubs.acs.org/doi/10.1021/acs.jcim.2c00838) |  [code](https://github.com/olsson-group/RL-GraphINVENT) 

* **DRlinker: Deep Reinforcement Learning for Optimization in Fragment Linking Design** [2022]  
  Tan, Y., Dai, L., Huang, W., Guo, Y., Zheng, S., Lei, J., ... & Yang, Y.   
  [J. Chem. Inf. Model. 2022, 62, 23, 5907–5917](https://pubs.acs.org/doi/10.1021/acs.jcim.2c00982) |  [code](https://github.com/biomed-AI/DRlinker) 

* **Widely Used and Fast De Novo Drug Design by a Protein Sequence-Based Reinforcement Learning Model** [2022]  
Li, Yaqin, Lingli Li, Yongjin Xu, and Yi Yu.  
[bioRxiv (2022)](https://arxiv.org/abs/2209.07405) 

* **Molecular Design Method Using a Reversible Tree Representation of Chemical Compounds and Deep Reinforcement Learning** [2022]  
  Ishitani, R., Kataoka, T. and Rikimaru, K.  
  [J. Chem. Inf. Model. 2022, 62, 17, 4032–4048](https://pubs.acs.org/doi/10.1021/acs.jcim.2c00366) |  [code](https://github.com/pfnet-research/RJT-RL) 

* **Accelerated rational PROTAC design via deep learning and molecular simulations** [2022]  
  Zheng, S., Tan, Y., Wang, Z. et al.  
  [Nat Mach Intell 4, 739–748 (2022)](https://www.nature.com/articles/s42256-022-00527-y) |  [code](https://github.com/biomed-AI/PROTAC-RL) 
  
* **Improving de novo molecular design with curriculum learning** [2022]  
  Guo, J., Fialková, V., Arango, J.D. et al.  
  [Nat Mach Intell 4, 555–563 (2022)](https://doi.org/10.1038/s42256-022-00494-4) |  [code](https://github.com/MolecularAI/ReinventCommunity/blob/master/notebooks/Automated_Curriculum_Learning_Demo.ipynb)  
 
* **De novo design of protein target specific scaffold-based Inhibitors via Reinforcement Learning** [2022]  
  Bontha M, McNaughton A, Knutson C, Pope J, Kumar N.  
  [arXiv:2205.10473 (2022)](https://arxiv.org/abs/2205.10473)

* **Autonomous molecule generation using reinforcement learning and docking to develop potential novel inhibitors** [2022]  
  Jeon, W., Kim, D.  
  [Sci Rep 10, 22104 (2020)](https://doi.org/10.1038/s41598-020-78537-2) |  [code](https://github.com/wsjeon92/morld) 

* **Scalable Fragment-Based 3D Molecular Design with Reinforcement Learning** [2022]  
  Flam-Shepherd, Daniel, Alexander Zhigalin, and Alán Aspuru-Guzik.  
  [arXiv:2202.00658 (2022)](https://arxiv.org/abs/2202.00658) 

* **Hit and Lead Discovery with Explorative RL and Fragment-based Molecule Generation** [2021]  
  Yang, S., Hwang, D., Lee, S., Ryu, S., & Hwang, S. J.  
  [Neural Information Processing Systems 34 (2021)](https://proceedings.neurips.cc/paper/2021/hash/41da609c519d77b29be442f8c1105647-Abstract.html) |  [code](https://github.com/AITRICS/FREED) 

* **Unlocking reinforcement learning for drug design** [2021]  
 
  [code](https://github.com/DarkMatterAI/mrl) 

* **MoleGuLAR: Molecule Generation Using Reinforcement Learning with Alternating Rewards** [2021]  
Goel, Manan, Shampa Raghunathan, Siddhartha Laghuvarapu, and U. Deva Priyakumar.  
  [J. Chem. Inf. Model. 2021, 61, 12, 5815–5826](https://pubs.acs.org/doi/10.1021/acs.jcim.1c01341) |  [code](https://github.com/devalab/MoleGuLAR) 

* **Memory-Assisted Reinforcement Learning for Diverse Molecular De Novo Design** [2020]  
Blaschke T, Engkvist O, Bajorath J, Chen H.  
  [Journal of cheminformatics 12.1 (2020)](https://jcheminf.biomedcentral.com/articles/10.1186/s13321-020-00473-0) |  [code](https://github.com/tblaschke/reinvent-memory) 

* **DeepGraphMolGen, a multi-objective, computational strategy for generating molecules with desirable properties: a graph convolution and reinforcement learning approach** [2020]  
  Khemchandani, Yash, Stephen O’Hagan, Soumitra Samanta, Neil Swainston, Timothy J. Roberts, Danushka Bollegala, and Douglas B. Kell.   
  [J Cheminform 12, 53 (2020)](https://doi.org/10.1186/s13321-020-00454-3) |  [code](https://github.com/dbkgroup/prop_gen) 

* **Reinforcement Learning for Molecular Design Guided by Quantum Mechanics** [2020]  
Simm, G., Pinsler, R. and Hernández-Lobato, J.M.,  
  [nternational Conference on Machine Learning. PMLR (2020)](http://proceedings.mlr.press/v119/simm20b/simm20b.pdf) |  [code](https://github.com/gncs/molgym) 

* **Deep learning enables rapid identification of potent DDR1 kinase inhibitors** [2019]   
  Zhavoronkov, A., Ivanenkov, Y.A., Aliper, A. et al.   
  [Nat Biotechnol 37, 1038–1040 (2019)](https://doi.org/10.1038/s41587-019-0224-x) |  [code](https://github.com/insilicomedicine/gentrl) 

* **Molecular de-novo design through deep reinforcement learning** [2017]  
Olivecrona, M., Blaschke, T., Engkvist, O. et al.  
  [ J Cheminform 9, 48 (2017)](https://doi.org/10.1186/s13321-017-0235-x) |  [code](https://github.com/tblaschke/reinvent) 











### Multi-task DMGs




* **Molecular Language Model as Multi-task Generator** [2023]  
 Fang, Y., Zhang, N., Chen, Z., Fan, X. and Chen, H.    
  [arXiv:2301.11259 (2023)](https://arxiv.org/abs/2301.11259) |  [code](https://github.com/zjunlp/MolGen) 





### Active Learning DMGs



* **Optimal Molecular Design: Generative Active Learning Combining REINVENT with Absolute Binding Free Energy Simulations**  [2024]   
 Loeffler, Hannes, Shunzhou Wan, Marco Klähn, Agastya Bhati, and Peter Coveney.   
  [chemrxiv-2024-sr1v6 (2024)](https://doi.org/10.26434/chemrxiv-2024-sr1v6) |  [code](https://github.com/MolecularAI/REINVENT4) 

* **ChemSpaceAL: An Efficient Active Learning Methodology Applied to Protein-Specific Molecular Generation**  [2024]   
 Kyro, Gregory W., Anton Morgunov, Rafael I. Brent, and Victor S. Batista.   
  [J. Chem. Inf. Model. (2024)](https://doi.org/10.1021/acs.jcim.3c01456) |  [code](https://github.com/batistagroup/ChemSpaceAL) 

* **Sample Efficient Reinforcement Learning with Active Learning for Molecular Design**  [2024]   
 Janet, Jon Paul, Michael Dodds, Jeff Guo, Thomas Löhr, Alessandro Tibo, and Ola Engkvist.   
  [Chemical Science (2024)](https://doi.org/10.1039/D3SC04653B) |  [code](https://www.rsc.org/suppdata/d3/sc/d3sc04653b/d3sc04653b2.pdf) 

* **Streamlining pipeline efficiency: a novel model-agnostic technique for accelerating conditional generative and virtual screening pipelines**  [2023]   
 Viswanathan, K., Goel, M., Laghuvarapu, S. et al.   
  [Sci Rep 13, 21069 (2023)](https://doi.org/10.1038/s41598-023-42952-y) |  [code](https://github.com/devalab/Enhanced-MoleGuLAR) 

* **Traversing Chemical Space with Active Deep Learning**  [2023]   
 Derek van Tilborg, AFrancesca Grisoni*.   
  [chemrxiv-2023-wgl32 (2023)](https://doi.org/10.26434/chemrxiv-2023-wgl32) |  [code](https://github.com/derekvantilborg/traversing_chem_space) 

* **ChemSpaceAL: An Efficient Active Learning Methodology Applied to Protein-Specific Molecular Generation**  [2023]   
 Kyro, Gregory W., Anton Morgunov, Rafael I. Brent, and Victor S. Batista.   
  [arXiv:2309.05853 (2023)](https://arxiv.org/abs/2309.05853) |  [code](https://github.com/batistagroup/ChemSpaceAL/) 





### Monte Carlo Tree Search




* **DrugSynthMC: an atom based generation of drug-like molecules with Monte Carlo Search** [2024]   
  Roucairol, Milo, Alexios Georgiou, Tristan Cazenave, Filippo Prischi, and Olivier E. Pardo   
  [chemrxiv-2024-l2969 (2024)](https://doi.org/10.26434/chemrxiv-2024-l2969) |  [code](https://github.com/RoucairolMilo/DrugSynthMC)

* **Generative AI for designing and validating easily synthesizable and structurally novel antibiotics** [2024]   
  Suzuki, Takamasa, Dian Ma, Nobuaki Yasuo, and Masakazu Sekijima.   
  [Nat Mach Intell 6, 338–353 (2024)](https://doi.org/10.1038/s42256-024-00809-7) |  [code](https://github.com/swansonk14/SyntheMol) 

* **Mothra: Multi-objective de novo Molecular Generation using Monte Carlo Tree Search** [2024]   
  Suzuki, Takamasa, Dian Ma, Nobuaki Yasuo, and Masakazu Sekijima.   
  [chemrxiv-2024-4719t (2024)](https://doi.org/10.26434/chemrxiv-2024-4719t) |  [code](https://github.com/sekijima-lab/Mothra) 

* **A flexible data-free framework for structure-based de novo drug design with reinforcement learning** [2023]   
  Hongyan Du, Dejun Jiang, Odin Zhang, Zhenxing Wu, Junbo Gao, Xujun Zhang, Xiaorui Wang, Yafeng Deng, Yu Kang, Dan Li, Peichen Pan, Chang-Yu Hsieh, Tingjun Hou.   
  [Chemical Science (2023)](https://doi.org/10.1039/D3SC04091G) |  [code](https://github.com/Brian-hongyan/3D-MCTS) 

* **ChemTSv2: Functional molecular design using de novo molecule generator** [2023]   
Ishida, Shoichi, Tanuj Aasawat, Masato Sumita, Michio Katouda, Tatsuya Yoshizawa, Kazuki Yoshizoe, Koji Tsuda, and Kei Terayama.   
[Wiley Interdisciplinary Reviews: Computational Molecular Science (2023)](https://doi.org/10.1002/wcms.1680) |  [code](https://github.com/molecule-generator-collection/ChemTSv2) 

* **VGAE-MCTS: a New Molecular Generative Model combining Variational Graph Auto-Encoder and Monte Carlo Tree Search** [2023]  
 Iwata, Hiroaki, Taichi Nakai, Takuto Koyama, Shigeyuki Matsumoto, Ryosuke Kojima, and Yasushi Okuno.   
  [J. Chem. Inf. Model. (2023)](https://doi.org/10.1021/acs.jcim.3c01220) | [chemrxiv-2023-q8419-v2](https://doi.org/10.26434/chemrxiv-2023-q8419-v2)  |  [code](https://github.com/clinfo/VGAE-MCTS) 

* **A graph-based genetic algorithm and generative model/Monte Carlo tree search for the exploration of chemical space** [2019]   
  Jensen, Jan H.    
  [Chemical science 10.12 (2019)](https://pubs.rsc.org/en/content/articlelanding/2019/sc/c8sc05372c)
  





### Genetic Algorithm-based


* **DockingGA: enhancing targeted molecule generation using transformer neural network and genetic algorithm with docking simulation**  [2024]   
 Changnan Gao, Wenjie Bao, Shuang Wang, Jianyang Zheng, Lulu Wang, Yongqi Ren, Linfang Jiao, Jianmin Wang, Xun Wang.   
  [Briefings in Functional Genomics (2024)](https://doi.org/10.1093/bfgp/elae011) |  [code](https://github.com/Chinafor/DockingGA) 

* **Enhancing molecular design efficiency: Uniting language models and generative networks with genetic algorithms**  [2024]   
 Bhowmik, Debsindhu, Pei Zhang, Zachary Fox, Stephan Irle, and John Gounley.   
  [Patterns (2024)](https://doi.org/10.1016/j.patter.2024.100947) |  [code](https://zenodo.org/records/8387351) 

* **Augmenting Genetic Algorithms with Machine Learning for Inverse Molecular Design** [2024]   
  Kneiding H, Balcells D.   
  [chemrxiv-2024-lcm83. (2024)](https://doi.org/10.26434/chemrxiv-2024-lcm83)   

* **Genetic Algorithm-Based Receptor Ligand: A Genetic Algorithm-Guided Generative Model to Boost the Novelty and Drug-Likeness of Molecules in a Sampling Chemical Space** [2024]   
  Wang, Mingyang, Zhengjian Wu, Jike Wang, Gaoqi Weng, Yu Kang, Peichen Pan, Dan Li et al.   
  [J. Chem. Inf. Model. (2024)](https://doi.org/10.1021/acs.jcim.3c01964) |  [code](https://github.com/micahwang/GARel) 

* **Genetic algorithms are strong baselines for molecule generation** [2023]   
  Austin Tripp and Jos'e Miguel Hern'andez-Lobato.   
  [arXiv:2310.09267 (2023)](https://arxiv.org/abs/2310.09267) 

* **GENERA: A Combined Genetic/Deep-Learning Algorithm for Multiobjective Target-Oriented De Novo Design** [2023]   
  Lamanna, Giuseppe, Pietro Delre, Gilles Marcou, Michele Saviano, Alexandre Varnek, Dragos Horvath, and Giuseppe Felice Mangiatordi.   
  [J. Chem. Inf. Model. (2023)](https://doi.org/10.1021/acs.jcim.3c00963) |  [code](https://github.com/GiuseppeLamanna/GENERA) 

* **AlvaBuilder: A Software for De Novo Molecular Design** [2023]   
  Mauri, Andrea, and Matteo Bertola.  
  [J. Chem. Inf. Model. (2023)](https://doi.org/10.1021/acs.jcim.3c00610) |  [code](https://www.alvascience.com/alvabuilder/) 

* **Reinforced Genetic Algorithm for Structure-based Drug Design** [2022]   
  Fu, Tianfan, Wenhao Gao, Connor Coley, and Jimeng Sun.  
  [Advances in Neural Information Processing Systems 35 (2022)](https://openreview.net/forum?id=Qx6UPW0r9Lf) |  [code](https://github.com/futianfan/reinforced-genetic-algorithm) 

* **Evolutionary design of molecules based on deep learning and a genetic algorithm** [2021]   
  Kwon, Y., Kang, S., Choi, YS. et al.   
  [Sci Rep 11, 17304 (2021)](https://doi.org/10.1038/s41598-021-96812-8) 

* **A graph-based genetic algorithm and generative model/Monte Carlo tree search for the exploration of chemical space** [2019]   
  Jensen, Jan H.    
  [Chemical science 10.12 (2019)](https://pubs.rsc.org/en/content/articlelanding/2019/sc/c8sc05372c) 

* **DENOPTIM: Software for Computational de Novo Design of Organic and Inorganic Molecules** [2019]   
  Marco Foscato, Vishwesh Venkatraman, and Vidar R. Jensen.   
  [J. Chem. Inf. Model. 2019, 59, 10, 4077–4082](https://doi.org/10.1021/acs.jcim.9b00516)






### Evolutionary Algorithm-based






* **Combining Evolutionary Algorithms with Reaction Rules Towards Focused Molecular Design**  [2023]   
 Correia, João, Vítor Pereira, and Miguel Rocha.   
  [Proceedings of the Genetic and Evolutionary Computation Conference (2023)](https://doi.org/10.1145/3583131.3590413) |  [code](https://github.com/BioSystemsUM/ReactEA) 

* **LEADD: Lamarckian evolutionary algorithm for de novo drug design**  [2022]   
 Kerstjens, A., De Winter, H.   
  [J Cheminform 14, 3 (2022)](https://doi.org/10.1186/s13321-022-00582-y) |  [code](https://github.com/UAMCAntwerpen/LEADD) 





### Large Language Model-based


* **Generative design of compounds with desired potency from target protein sequences using a multimodal biochemical language model**  [2024]   
 Chen, H., Bajorath, J.   
  [ J Cheminform 16, 55 (2024)](https://doi.org/10.1186/s13321-024-00852-x) |  [code](https://uni-bonn.sciebo.de/s/Z9O2ZqKoA2cS7B1) 

* **Large Property Models: A New Generative Paradigm for Molecules**  [2024]   
 Jin, Tianfan, Veerupaksh Singla, Hsuan-Hao Hsu, and Brett Savoie.   
  [chemrxiv-2024-v3qww (2024)](https://doi.org/10.26434/chemrxiv-2024-v3qww)  

* **De novo drug design as GPT language modeling: large chemistry models with supervised and reinforcement learning**  [2024]   
 Ye, Gavin.   
  [ Journal of Computer-Aided Molecular Design 38.1 (2024)](https://link.springer.com/article/10.1007/s10822-024-00559-z) |  [code](https://huggingface.co/Coconut104/EfficacyGPT-DrugDesign) 

* **DrugAssist: A Large Language Model for Molecule Optimization**  [2023]   
 Ye, Geyan, Xibao Cai, Houtim Lai, Xing Wang, Junhong Huang, Longyue Wang, Wei Liu, and Xiangxiang Zeng.   
  [arXiv:2401.10334 (2023)](https://arxiv.org/abs/2401.10334) |  [code](https://github.com/blazerye/DrugAssist) 





## Text-driven molecular generation models




* **Text-Guided Molecule Generation with Diffusion Language Model** [2024]   
 Gong, Haisong, Qiang Liu, Shu Wu, and Liang Wang.    
  [arXiv:2402.13040 (2024)](https://arxiv.org/abs/2402.13040)  |  [code](https://github.com/Deno-V/tgm-dlm) 

* **Exploring the potential of AI-Chatbots in organic chemistry: An assessment of ChatGPT and bard** [2023]   
 Hallal, K., Hamdan, R. and Tlais, S.    
  [Computers and Education: Artificial Intelligence (2023)](https://doi.org/10.1016/j.caeai.2023.100170) 

* **Generating Novel Leads for Drug Discovery using LLMs with Logical Feedback** [2023]   
 Shreyas Bhat Brahmavar, Ashwin Srinivasan, Tirtharaj Dash, Sowmya R Krishnan, Lovekesh Vig, Arijit Roy, Raviprasad Aduri    
  [bioRxiv (2023)](https://doi.org/10.1101/2023.09.14.557698) |  [code](https://github.com/Shreyas-Bhat/LMLF) 

* **DrugGPT: A GPT-based Strategy for Designing Potential Ligands Targeting Specific Proteins** [2023]   
 Yuesen Li, Chengyi Gao, Xin Song, Xiangyu Wang,  View ORCID ProfileYungang Xu, Suxia Han    
  [bioRxiv (2023)](https://doi.org/10.1101/2023.06.29.543848) |  [code](https://github.com/LIYUESEN/druggpt) 

* **Interactive Molecular Discovery with Natural Language** [2023]   
 Zheni Zeng, Bangchen Yin, Shipeng Wang, Jiarui Liu, Cheng Yang, Haishen Yao, Xingzhi Sun, Maosong Sun, Guotong Xie, Zhiyuan Liu   
  [arXiv:2306.11976v1](https://arxiv.org/abs/2306.11976) |  [code](https://github.com/Ellenzzn/ChatMol) 

* **Mol-Instructions: A Large-Scale Biomolecular Instruction Dataset for Large Language Models** [2023]  
 Yin Fang, Xiaozhuan Liang, Ningyu Zhang, Kangwei Liu, Rui Huang, Zhuo Chen, Xiaohui Fan, Huajun Chen   
  [arXiv:2306.08018v1](https://arxiv.org/abs/2306.08018) |  [code](https://github.com/zjunlp/Mol-Instructions) 

* **Domain-Agnostic Molecular Generation with Self-feedback** [2023]  
 Yin Fang, Ningyu Zhang, Zhuo Chen, Xiaohui Fan, Huajun Chen   
  [arXiv:2301.11259v3](https://arxiv.org/abs/2301.11259) |  [code](https://github.com/zjunlp/MolGen) 





## Multi-Target based deep molecular generative models

* **Generation of Dual-Target Compounds Using a Transformer Chemical Language Model** [2024]  
 Srinivasan, Sanjana, and Jürgen Bajorath.    
  [chemrxiv-2024-8qj17 (2024)](https://doi.org/10.26434/chemrxiv-2024-8qj17)  

* **Structure-aware dual-target drug design through collaborative learning of pharmacophore combination and molecular simulation** [2024]  
 Sheng Chen, Junjie Xie, Renlong Ye, David Daqiang Xu and Yuedong Yang*.    
  [Chem. Sci., (2024)](https://doi.org/10.1039/D4SC00094C) | [code](https://github.com/biomed-AI/AIxFuse) 

* **De novo generation of multi-target compounds using deep generative chemistry** [2024]  
 Munson, B.P., Chen, M., Bogosian, A. et al.    
  [Nat Commun 15, 3636 (2024)](https://doi.org/10.1038/s41467-024-47120-y) | [code](https://github.com/bpmunson/polygon) 

* **De novo generation of dual-target ligands using adversarial training and reinforcement learning** [2021]  
 Lu, Fengqing, Mufei Li, Xiaoping Min, Chunyan Li, and Xiangxiang Zeng.    
  [Briefings in Bioinformatics 22.6 (2021)](https://doi.org/10.1093/bib/bbab333) | [code](https://github.com/lllfq/DLGN) 

* **Compound dataset and custom code for deep generative multi-target compound design** [2021]  
 Blaschke, Thomas, and Jürgen Bajorath.    
  [Future Science OA 7.6 (2021)](https://doi.org/10.2144/fsoa-2021-0033) |  [code](https://github.com/tblaschke/reinvent-multi-target) 





## Ligand-based deep molecular generative models




* **Tree-Invent: A Novel Multipurpose Molecular Generative Model Constrained with a Topological Tree** [2023]   
  Mingyuan Xu, Hongming Chen.   
  [J. Chem. Inf. Model.  (2023)](https://doi.org/10.1021/acs.jcim.3c01626) |  [code](https://github.com/MingyuanXu/Tree-Invent) 

* **LS-MolGen: Ligand-and-Structure Dual-Driven Deep Reinforcement Learning for Target-Specific Molecular Generation Improves Binding Affinity and Novelty** [2023]  
 Li, Song, Chao Hu, Song Ke, Chenxing Yang, Jun Chen, Yi Xiong, Hao Liu, and Liang Hong.   
  [J. Chem. Inf. Model. （2023）](https://pubs.acs.org/doi/full/10.1021/acs.jcim.3c00587) |  [code](https://github.com/songleee/LS-MolGen) 

* **Regression Transformer enables concurrent sequence regression and generation for molecular language modeling** [2023]  
  Born, Jannis and Manica, Matteo  
  [Nat Mach Intell 5, 432–444 (2023)](https://doi.org/10.1038/s42256-023-00639-z) |  [arXiv:2202.01338v3](https://arxiv.org/abs/2202.01338) |  [code](https://github.com/IBM/regression-transformer) 

* **Domain-Agnostic Molecular Generation with Self-feedback** [2023]  
 Yin Fang, Ningyu Zhang, Zhuo Chen, Xiaohui Fan, Huajun Chen   
  [arXiv:2301.11259v3](https://arxiv.org/abs/2301.11259) |  [code](https://github.com/zjunlp/MolGen) 

* **Transformer-based molecular generative model for antiviral drug design** [2023]  
 mao, jiashun; wang, jianming; zeb, amir; Cho, Kwang-Hwi; jin, haiyan; Kim, Jongwan; Lee, Onju; Wang, Yunyun; No, Kyoung Tai.    
  [Available at SSRN 4345811 (2023)](https://dx.doi.org/10.2139/ssrn.4345811) |  [code](https://github.com/AspirinCode/TransAntivirus) 

* **Leveraging molecular structure and bioactivity with chemical language models for de novo drug design** [2023]  
  Kotsias, PC., Arús-Pous, J., Chen, H. *et al.*  
  [Nat Commun 14, 114 (2023)](https://www.nature.com/articles/s41467-022-35692-6) |  [code](https://github.com/ETHmodlab/hybridCLMs/tree/v1.0) 

* **Explore drug-like space with deep generative models** [2023]  
Wang, Jianmin, et al.   
[Methods (2023)](https://doi.org/10.1016/j.ymeth.2023.01.004) |  [code](https://github.com/AspirinCode/drug-likeness_space) 

* **De novo molecular design with deep molecular generative models for PPI inhibitors** [2022]  
  Jianmin Wang, Yanyi Chu, Jiashun Mao, Hyeon-Nae Jeon, Haiyan Jin, Amir Zeb, Yuil Jang, Kwang-Hwi Cho, Tao Song, Kyoung Tai No.  
  [Briefings in Bioinformatics 23.4 (2022)](https://doi.org/10.1093/bib/bbac285) |  [code](https://github.com/AspirinCode/iPPIGAN) 

* **DeLA-Drug: A Deep Learning Algorithm for Automated Design of Druglike Analogues** [2022]  
  Creanza, T. M., Lamanna, G., Delre, P., Contino, M., Corriero, N., Saviano, M., ... & Ancona, N.  
  [J. Chem. Inf. Model. 2022, 62, 6, 1411–1424](https://pubs.acs.org/doi/10.1021/acs.jcim.2c00205) |  [Web](http://www.ba.ic.cnr.it/softwareic/deladrugportal/) 

* **SMILES-based CharLSTM with finetuning and goal-directed generation via policy gradient** [2022]  

  [code](https://github.com/gmattedi/Smiles-LSTM) 

* **Large-scale chemical language representations capture molecular structure and properties** [2022]  
Ross, J., Belgodere, B., Chenthamarakshan, V., Padhi, I., Mroueh, Y., & Das, P.   
[Nat Mach Intell 4, 1256–1264 (2022)](https://www.nature.com/articles/s42256-022-00580-7) |  [code](https://github.com/IBM/molformer) 

* **Augmented Hill-Climb increases reinforcement learning efficiency for language-based de novo molecule generation** [2022]  
  Thomas, M., O’Boyle, N.M., Bender, A. et al.  
  [J Cheminform 14, 68 (2022)](https://doi.org/10.1186/s13321-022-00646-z) |  [code](https://github.com/MorganCThomas/SMILES-RNN) 

* **De novo molecule design with chemical language models** [2022]  
  Grisoni, F., Schneider, G.  
  [Artificial Intelligence in Drug Design (2022)](https://doi.org/10.1007/978-1-0716-1787-8_9) |  [code](https://github.com/grisoniFr/de_novo_design_RNN) 

* **Can We Quickly Learn to “Translate” Bioactive Molecules with Transformer Models?** [2022]  
  Bagal, V., Aggarwal, R., Vinod, P. K., & Priyakumar, U. D.  
  [J. Chem. Inf. Model. 2023, 63, 6, 1734–1744](https://pubs.acs.org/doi/full/10.1021/acs.jcim.2c01618) | [chemrxiv-2022-gln27](https://doi.org/10.26434/chemrxiv-2022-gln27)

* **MolGPT: Molecular Generation Using a Transformer-Decoder Model** [2022]   
  Bagal, V., Aggarwal, R., Vinod, P. K., & Priyakumar, U. D.  
  [J. Chem. Inf. Model. 2022, 62, 9, 2064–2076](https://pubs.acs.org/doi/10.1021/acs.jcim.1c00600) |  [code](https://github.com/devalab/molgpt) 

* **A Transformer-based Generative Model for De Novo Molecular Design** [2022]  
  Wang, Wenlu, et al.  
  [arXiv:2210.08749 (2022)](https://arxiv.org/abs/2210.08749) 

* **Translation between Molecules and Natural Language** [2022]  
  Edwards, C., Lai, T., Ros, K., Honke, G., & Ji, H.  
  [arXiv:2204.11817 (2022)](https://arxiv.org/abs/2204.11817) |  [code](https://github.com/blender-nlp/MolT5) 

* **Optimizing Recurrent Neural Network Architectures for De Novo Drug Design** [2021]  
  Santos, B. P., Abbasi, M., Pereira, T., Ribeiro, B., & Arrais, J. P.  
  [CBMS. IEEE, (2021)](https://ieeexplore.ieee.org/document/9474742) |  [code](https://github.com/larngroup/RNN-Drug-Generation) 

* **A recurrent neural network (RNN) that generates drug-like molecules for drug discovery** [2021]  
  [code](https://github.com/shiwentao00/Molecule-RNN)  

* **A molecule generative model used interaction fingerprint (docking pose) as constraints** [2021]  
  [code](https://github.com/jeah-z/IFP-RNN) 

* **De novo design and bioactivity prediction of SARS-CoV-2 main protease inhibitors using recurrent neural network-based transfer learning** [2021]  
  Santana, M.V.S., Silva-Jr, F.P.   
  [BMC chemistry 15.1 (2021)](https://bmcchem.biomedcentral.com/articles/10.1186/s13065-021-00737-2) |  [code](https://github.com/marcossantanaioc/De_novo_design_SARSCOV2) 

* **Generative Pre-Training from Molecules** [2021]  
  Adilov, Sanjar.  
  [J. Chem. Inf. Model. 2022, 62, 9, 2064–2076](https://pubs.acs.org/doi/10.1021/acs.jcim.1c00600) |  [code](https://github.com/sanjaradylov/smiles-gpt) 

* **Transformers for Molecular Graph Generation** [2021]  
  Cofala, Tim, and Oliver Kramer.  
  [ESANN. 2021](https://www.esann.org/sites/default/files/proceedings/2021/ES2021-112.pdf) |  [code](https://gitlab.uni-oldenburg.de/gies6280/molegent) 

* **Spatial Generation of Molecules with Transformers** [2021]  
  Cofala, Tim, and Oliver Kramer.  
  [IJCNN. IEEE, (2021)](https://ieeexplore.ieee.org/abstract/document/9533439) |  [code](https://gitlab.uni-oldenburg.de/gies6280/molegent) 

* **Generative Chemical Transformer: Neural Machine Learning of Molecular Geometric Structures from Chemical Language via Attention** [2021]  
  Hyunseung Kim, Jonggeol Na*, and Won Bo Lee*.  
  [J. Chem. Inf. Model. 2021, 61, 12, 5804–5814](https://pubs.acs.org/doi/abs/10.1021/acs.jcim.1c01289) |  [code](https://github.com/Hyunseung-Kim/molGCT) 

* **C5T5: Controllable Generation of Organic Molecules with Transformers** [2021]  
  Rothchild, D., Tamkin, A., Yu, J., Misra, U., & Gonzalez, J.  
  [arXiv:2108.10307 (2021)](https://arxiv.org/abs/2108.10307) |  [code](https://github.com/dhroth/c5t5) 

* **Molecular optimization by capturing chemist’s intuition using deep neural networks** [2021]  
  He, J., You, H., Sandström, E. et al.   
  [ J Cheminform 13, 26 (2021)](https://doi.org/10.1186/s13321-021-00497-0) |  [code](https://github.com/MolecularAI/deep-molecular-optimization) 

* **Transmol: repurposing a language model for molecular generation** [2021]  
  Grechishnikova, Daria.   
  [RSC advances 11.42 (2021)](https://pubs.rsc.org/en/content/articlelanding/2021/ra/d1ra03086h) |  [code](https://gitlab.com/cheml.io/public/transmol) 

* **Attention-based generative models for de novo molecular design** [2021]  
  Dollar, O., Joshi, N., Beck, D.A. and Pfaendtner, J.  
  [Chemical Science 12.24 (2021)](https://pubs.rsc.org/en/content/articlelanding/2021/sc/d1sc01050f) |  [code](https://github.com/oriondollar/TransVAE) 

* **Bidirectional Molecule Generation with Recurrent Neural Networks** [2020]  
  Grisoni, F., Moret, M., Lingwood, R., & Schneider, G.  
  [J. Chem. Inf. Model. 2020, 60, 3, 1175–1183](https://pubs.acs.org/doi/10.1021/acs.jcim.9b00943) |  [code](https://github.com/robinlingwood/BIMODAL) 

* **GraphAF: a Flow-based Autoregressive Model for Molecular Graph Generation** [2020]  
Shi, C., Xu, M., Zhu, Z., Zhang, W., Zhang, M., & Tang, J.   
[arXiv:2001.09382 (2020)](https://arxiv.org/abs/2001.09382) |  [code](https://github.com/DeepGraphLearning/GraphAF) 

* **Direct steering of de novo molecular generation with descriptor conditional recurrent neural networks** [2019]  
  Kotsias, PC., Arús-Pous, J., Chen, H. *et al.*   
  [Nat Mach Intell 2, 254–265 (2020)](https://www.nature.com/articles/s42256-020-0174-5) |  [code](https://github.com/pcko1/Deep-Drug-Coder) 

* **Generative Recurrent Networks for De Novo Drug Design** [2018]  
  Gupta, A., Müller, A. T., Huisman, B. J., Fuchs, J. A., Schneider, P., & Schneider, G.   
  [Molecular informatics 37.1-2 (2018)](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5836943/) |  [code](https://github.com/topazape/LSTM_Chem) 

* **Generative Recurrent Neural Networks for De Novo Drug Design** [2017]  
  Gupta, Anvita, et al.   
  [Molecular informatics 37.1-2 (2018)r](https://onlinelibrary.wiley.com/doi/10.1002/minf.201700111) |  [code](https://github.com/SilviaAmAm/MolBot) 

* **ChemTS: An Efficient Python Library for de novo Molecular Generation** [2017]  
  Yang, X., Zhang, J., Yoshizoe, K., Terayama, K., & Tsuda, K.  
  [Science and technology of advanced materials 18.1 (2017)](https://arxiv.org/abs/1710.00616) |  [arXiv:1710.00616v1](https://arxiv.org/abs/1710.00616) |  [code](https://github.com/tsudalab/ChemTS) 







## Pharmacophore-based deep molecular generative models

* **PGMG: A Pharmacophore-Guided Deep Learning Approach for Bioactive Molecular Generation** [2022]  
  Zhu, Huimin, Renyi Zhou, Jing Tang, and Min Li.  
  [arXiv:2207.00821 (2022)](https://arxiv.org/abs/2207.00821) |  [code](https://github.com/CSUBioGroup/PGMG) 

* **Deep generative design with 3D pharmacophoric constraints** [2021]  
  mrie, Fergus and Hadfield, Thomas E and Bradley, Anthony R and Deane, Charlotte M.  
  [Chemical science 12.43 (2021)](https://pubs.rsc.org/en/content/articlelanding/2021/SC/D1SC02436A) |  [code](https://github.com/XiaotanYu/DEVELOP-ZYH) 













## Structure-based deep molecular generative models


* **Generation of Dual-Target Compounds Using a Transformer Chemical Language Model** [2024]  
 Srinivasan, Sanjana, and Jürgen Bajorath.    
  [chemrxiv-2024-8qj17 (2024)](https://doi.org/10.26434/chemrxiv-2024-8qj17)  

* **Structure-Based Drug Design with a Deep Hierarchical Generative Model** [2024]   
  Weller, Jesse A., and Remo Rohs.   
  [J. Chem. Inf. Model. (2024)](https://pubs.acs.org/doi/full/10.1021/acs.jcim.4c01193) |  [code](https://github.com/jssweller/DrugHIVE) 

* **Structure-aware dual-target drug design through collaborative learning of pharmacophore combination and molecular simulation** [2024]  
 Sheng Chen, Junjie Xie, Renlong Ye, David Daqiang Xu and Yuedong Yang*.    
  [Chem. Sci., (2024)](https://doi.org/10.1039/D4SC00094C) | [code](https://github.com/biomed-AI/AIxFuse) 

* **De novo generation of multi-target compounds using deep generative chemistry** [2024]  
 Munson, B.P., Chen, M., Bogosian, A. et al.    
  [Nat Commun 15, 3636 (2024)](https://doi.org/10.1038/s41467-024-47120-y) | [code](https://github.com/bpmunson/polygon) 

* **Generative design of compounds with desired potency from target protein sequences using a multimodal biochemical language model**  [2024]   
 Chen, H., Bajorath, J.   
  [ J Cheminform 16, 55 (2024)](https://doi.org/10.1186/s13321-024-00852-x) |  [code](https://uni-bonn.sciebo.de/s/Z9O2ZqKoA2cS7B1) 

* **From Theory to Therapy: Reframing SBDD Model Evaluation via Practical Metrics** [2024]   
Gao, Bowen, Haichuan Tan, Yanwen Huang, Minsi Ren, Xiao Huang, Wei-Ying Ma, Ya-Qin Zhang and Yanyan Lan.   
[	arXiv:2406.08980 (2024)](https://arxiv.org/abs/2406.08980) |  [code](https://github.com/bowen-gao/sbdd_practical_evaluation)

* **Structure-based Drug Design Benchmark: Do 3D Methods Really Dominate?** [2024]   
Zheng, Kangyu, Yingzhou Lu, Zaixi Zhang, Zhongwei Wan, Yao Ma, Marinka Zitnik, and Tianfan Fu.   
[arXiv-2406 (2024)](https://arxiv.org/abs/2406.03403) |  [code](https://github.com/zkysfls/2024-sbdd-benchmark)

* **Prospective de novo drug design with deep interactome learning** [2024]   
Atz, K., Cotos, L., Isert, C. et al.   
[Nat Commun 15, 3408 (2024)](https://doi.org/10.1038/s41467-024-47613-w) |  [code](https://github.com/atzkenneth/dragonfly_gen)

* **A unified conditional diffusion framework for dual protein targets based bioactive molecule generation**  [2024]   
 Huang, Lei, Zheng Yuan, Huihui Yan, Rong Sheng, Linjing Liu, Fuzhou Wang, Weidun Xie et al.    
  [IEEE Transactions on Artificial Intelligence (2024)](https://doi.org/10.1109/TAI.2024.3387402)  |  [arXiv:2306.13957 (2023)](https://arxiv.org/abs/2306.13957) 

* **Sculpting Molecules in Text-3D Space: A Flexible Substructure Aware Framework for Text-Oriented Molecular Optimization** [2024]   
 Zhang, Kaiwei, Yange Lin, Guangcheng Wu, Yuxiang Ren, Xuecang Zhang, Bo Wang, and Xiao-Yu Zhang.    
  [Research Square (2024)](https://www.researchsquare.com/article/rs-4023429/v1)  

* **AUTODIFF: Autoregressive Diffusion Modeling for Structure-based Drug Design** [2024]   
 Li, Xinze, Penglei Wang, Tianfan Fu, Wenhao Gao, Chengtao Li, Leilei Shi, and Junhong Liu.    
  [arXiv:2404.02003 (2024)](https://arxiv.org/html/2404.02003v1)  

* **MolSnapper: Conditioning Diffusion for Structure Based Drug Design** [2024]   
 Ziv, Yael, Brian Marsden, and Charlotte Deane.    
  [bioRxiv (2024)](https://doi.org/10.1101/2024.03.28.586278)  |  [code](https://github.com/oxpig/MolSnapper) 

* **3D molecular generative framework for interaction-guided drug design** [2024]   
  Zhung, W., Kim, H. & Kim, W.Y.   
  [Nat Commun 15, 2688 (2024)](https://doi.org/10.1038/s41467-024-47011-2) |  [code](https://github.com/ACE-KAIST/DeepICL) 

* **A dual diffusion model enables 3D molecule generation and lead optimization based on target pockets** [2024]   
 Huang, L., Xu, T., Yu, Y. et al.    
  [Nat Commun 15, 2657 (2024)](https://doi.org/10.1038/s41467-024-46569-1)  |  [code](https://github.com/Layne-Huang/PMDM)   

* **ChemSpaceAL: An Efficient Active Learning Methodology Applied to Protein-Specific Molecular Generation**  [2024]   
 Kyro, Gregory W., Anton Morgunov, Rafael I. Brent, and Victor S. Batista.   
  [J. Chem. Inf. Model. (2024)](https://doi.org/10.1021/acs.jcim.3c01456) |  [code](https://github.com/batistagroup/ChemSpaceAL) 

* **PocketFlow is a data-and-knowledge-driven structure-based molecular generative model** [2024]   
Shengyong Yang, Yuanyuan Jiang, Guo Zhang et al.  
[Nat Mach Intell (2024)](https://doi.org/10.1038/s42256-024-00808-8) |  [Research Square. PREPRINT. (2023)](https://www.researchsquare.com/article/rs-3077992/v1) |  [code](https://github.com/Saoge123/PocketFlow)   

* **Structure-Based Drug Design via 3D Molecular Generative Pre-training and Sampling** [2024]   
 Yang, Yuwei, Siqi Ouyang, Xueyu Hu, Meihua Dang, Mingyue Zheng, Hao Zhou, and Lei Li.    
  [arXiv:2402.14315 (2024)](https://arxiv.org/abs/2402.14315) 

* **Target-aware Molecule Generation for Drug Design Using a Chemical Language Model** [2024]   
  Xia, Yingce, Kehan Wu, Pan Deng, Renhe Liu, Yuan Zhang, Han Guo, Yumeng Cui et al.   
  [bioRxiv (2024)](https://doi.org/10.1101/2024.01.08.574635) 

* **KGDiff: towards explainable target-aware molecule generation with knowledge guidance** [2023]   
 Hao Qian, Wenjing Huang, Shikui Tu, Lei Xu.    
  [Briefings in Bioinformatics.  (2023)](https://doi.org/10.1093/bib/bbad435)  |  [code](https://github.com/CMACH508/KGDiff) 

* **Geometric Deep Learning for Structure-Based Ligand Design** [2023]   
  Alexander S. Powers, Helen H. Yu, Patricia Suriana, Rohan V. Koodli, Tianyu Lu, Joseph M. Paggi, and Ron O. Dror.   
  [ACS Cent. Sci. (2023)](https://doi.org/10.1021/acscentsci.3c00572) 

* **Autoregressive fragment-based diffusion for pocket-aware ligand design** [2023]   
 Ghorbani, Mahdi, Leo Gendelev, Paul Beroza, and Michael Keiser.    
  [NeurIPS 2023 Generative AI and Biology (GenBio) Workshop.  (2023)](https://openreview.net/forum?id=E3HN48zjam) |  [code](https://github.com/ghorbanimahdi73/autofragdiff) 

* **Delta Score: Improving the Binding Assessment of Structure-Based Drug Design Methods** [2023]   
  Minsi Ren, Bowen Gao, Bo Qiang, Yanyan Lan.   
  [NeurIPS 2023 Generative AI and Biology (GenBio) Workshop. (2023)](https://openreview.net/forum?id=iO59l1LFvJ) 

* **Target-Aware Variational Auto-Encoders for Ligand Generation with Multi-Modal Protein Modeling** [2023]   
  Ngo, Khang, and Truong Son Hy.   
  [NeurIPS 2023 Generative AI and Biology (GenBio) Workshop. (2023)](https://openreview.net/forum?id=4k926QVVM4) |  [code](https://github.com/HySonLab/Ligand_Generation) 

* **Conformer Generation for Structure-Based Drug Design: How Many and How Good?** [2023]   
  McNutt, Andrew, Fatimah Bisiriyu, Sophia Song, Ananya Vyas, Geoffrey Hutchison, and David Koes.   
  [J. Chem. Inf. Model. (2023)](https://doi.org/10.1021/acs.jcim.3c01245) |  [code](https://github.com/dkoes/conformer_analysis) 

* **AlphaFold accelerates artificial intelligence powered drug discovery: efficient discovery of a novel CDK20 small molecule inhibitor** [2023]   
  Ren, Feng, Xiao Ding, Min Zheng, Mikhail Korzinkin, Xin Cai, Wei Zhu, Alexey Mantsyzov et al.   
  [Chemical Science 14.6 (2023)](https://pubs.rsc.org/en/content/articlehtml/2023/sc/d2sc05709c)   

* **Interaction-aware 3D Molecular Generative Framework for Generalizable Structure-based Drug Design** [2023]   
  Woo Youn Kim, Wonho Zhung, and Hyeongwoo Kim.   
  [Research Square. (2023)](https://www.researchsquare.com/article/rs-3388359/v1) |  [code](https://github.com/ACE-KAIST/DeepICL) 

* **A flexible data-free framework for structure-based de novo drug design with reinforcement learning** [2023]   
  Hongyan Du, Dejun Jiang, Odin Zhang, Zhenxing Wu, Junbo Gao, Xujun Zhang, Xiaorui Wang, Yafeng Deng, Yu Kang, Dan Li, Peichen Pan, Chang-Yu Hsieh, Tingjun Hou.   
  [Chemical Science (2023)](https://doi.org/10.1039/D3SC04091G) |  [code](https://github.com/Brian-hongyan/3D-MCTS) 

* **An interface-based molecular generative framework for protein-protein interaction inhibitors** [2023]   
Jianmin Wang, Jiashun Mao, Chunyan Li, Hongxin Xiang, Xun Wang, Shuang Wang, Zixu Wang, Yangyang Chen, Yuquan Li, Heqi Sun, Kyoung Tai No, Tao Song, Xiangxiang Zeng    
[bioRxiv (2023)](https://doi.org/10.1101/2023.10.10.557742) |  [code](https://github.com/AspirinCode/GENiPPI) 

* **DiffDec: Structure-Aware Scaffold Decoration with an End-to-End Diffusion** [2023]   
 Xie, Junjie, Sheng Chen, Jinping Lei, and Yuedong Yang.    
  [bioRxiv (2023)](https://doi.org/10.1101/2023.10.08.561377) 

* **Pocket Crafter: A 3D Generative Modeling Based Workflow for the Rapid Generation of Hit Molecules in Drug Discovery** [2023]   
Shen, L., Fang, J., Liu, L., Yang, F., Jenkins, J.L., Kutchukian, P.S. and Wang, H.  
[chemrxiv-2023-3b9p3 (2023)](https://doi.org/10.26434/chemrxiv-2023-3b9p3) 

* **Learning Subpocket Prototypes for Generalizable Structure-based Drug Design** [2023]   
ZHANG Z, Liu Q.  
[ICML'23: Proceedings of the 40th International Conference on Machine Learning (2023)](https://dl.acm.org/doi/10.5555/3618408.3620143) |  [code](https://github.com/zaixizhang/DrugGPS_ICML23) 

* **Learning on topological surface and geometric structure for 3D molecular generation** [2023]   
Zhang, Odin, Tianyue Wang, Gaoqi Weng, Dejun Jiang, Ning Wang, Xiaorui Wang, Huifeng Zhao et al.  
[Nat Comput Sci (2023)](https://doi.org/10.1038/s43588-023-00530-2) |  [code](https://github.com/HaotianZhangAI4Science/SurfGen) 

* **Target-Specific Novel Molecules with their Recipe: Incorporating Synthesizability in the Design Process** [2023]   
  Krishnan, Sowmya Ramaswamy, Navneet Bung, Rajgopal Srinivasan, and Arijit Roy.   
  [chemrxiv-2023-54bss. (2023)](https://doi.org/10.26434/chemrxiv-2023-54bss)

* **TacoGFN: Target Conditioned GFlowNet for Structure-Based Drug Design** [2023]   
Tony Shen, Mohit Pandey, Martin Ester.  
[arXiv:2310.03223. (2023)](https://arxiv.org/abs/2310.03223v1) 

* **Structured State-Space Sequence Models for De Novo Drug Design** [2023]   
  Özçelik R, de Ruiter S, Grisoni F.   
  [chemrxiv-2023-jwmf3. (2023)](https://doi.org/10.26434/chemrxiv-2023-jwmf3) |  [code](https://github.com/molML/s4-for-de-novo-drug-design) 

* **De Novo Generation of Chemical Structures of Inhibitor and Activator Candidates for Therapeutic Target Proteins by a Transformer-Based Variational Autoencoder and Bayesian Optimization** [2023]   
  Yuki Matsukiyo, Chikashige Yamanaka, and Yoshihiro Yamanishi.   
  [J. Chem. Inf. Model. (2023)](https://doi.org/10.1021/acs.jcim.3c00824) |  [code](https://yamanishi.cs.i.nagoya-u.ac.jp/triompheboa/) 

* **Deep interactome learning for de novo drug design**  [2023]   
 Atz K, Cotos Muñoz L, Isert C, Håkansson M, Focht D, Nippa DF, et al.   
  [chemrxiv-2023-cbq9k (2023)](https://doi.org/10.26434/chemrxiv-2023-cbq9k) 

* **ChemSpaceAL: An Efficient Active Learning Methodology Applied to Protein-Specific Molecular Generation**  [2023]   
 Kyro, Gregory W., Anton Morgunov, Rafael I. Brent, and Victor S. Batista.   
  [arXiv:2309.05853 (2023)](https://arxiv.org/abs/2309.05853) |  [code](https://github.com/batistagroup/ChemSpaceAL/) 

* **ResGen is a pocket-aware 3D molecular generation model based on parallel multiscale modelling** [2023]   
Zhang, O., Zhang, J., Jin, J. et al.   
[Nat Mach Intell (2023)](https://doi.org/10.1038/s42256-023-00712-7) |  [code](https://github.com/HaotianZhangAI4Science/ResGen) 

* **Benchmarking Generated Poses: How Rational is Structure-based Drug Design with Generative Models?**  [2023]   
 Harris, Charles, Kieran Didi, Arian R. Jamasb, Chaitanya K. Joshi, Simon V. Mathis, Pietro Lio, and Tom Blundell.   
  [arXiv:2308.07413 (2023)](https://arxiv.org/abs/2308.07413) 

* **Lingo3DMol: Generation of a Pocket-based 3D Molecule using a Language Model**  [2023]   
 Wang, Lvwei, Zaiyun Lin, Yanhao Zhu, Rong Bai, Wei Feng, Huting Wang, Jielong Zhou, Wei Peng, Bo Huang, and Wenbiao Zhou.   
  [arXiv:2305.10133 (2023)](https://arxiv.org/abs/2305.10133) |  [code](https://github.com/stonewiseAIDrugDesign/Lingo3DMol) 

* **Target-aware Variational Auto-encoders for Ligand Generation with Multimodal Protein Representation Learning**  [2023]   
 Nhat Khang Ngo, Truong Son Hy.   
  [bioRxiv. (2023)](https://doi.org/10.1101/2023.08.10.552868) |  [code](https://github.com/HySonLab/Ligand_Generation) 

* **Sequence-based drug design as a concept in computational drug design**   [2023]   
 Chen, L., Fan, Z., Chang, J. et al.    
  [Nat Commun 14, 4217 (2023)]( https://doi.org/10.1038/s41467-023-39856-w)  |  [code](https://github.com/lifanchen-simm/transformerCPI2.0/) 

* **Semi-Equivariant conditional normalizing flows, with applications to target-aware molecule generation** [2023]  
Rozenberg, Eyal, and Daniel Freedman.   
[Machine Learning: Science and Technology (2023)](https://iopscience.iop.org/article/10.1088/2632-2153/ace58c/meta) |  [arXiv:2304.06779 (2023)](https://arxiv.org/abs/2304.06779) 

* **DiffDTM: A conditional structure-free framework for bioactive molecules generation targeted for dual proteins**   [2023]   
 Huang, Lei, Zheng Yuan, Huihui Yan, Rong Sheng, Linjing Liu, Fuzhou Wang, Weidun Xie et al.    
  [arXiv:2306.13957 (2023)](https://arxiv.org/abs/2306.13957) 

* **DrugGPT: A GPT-based Strategy for Designing Potential Ligands Targeting Specific Proteins** [2023]   
 Yuesen Li, Chengyi Gao, Xin Song, Xiangyu Wang,  View ORCID ProfileYungang Xu, Suxia Han    
  [bioRxiv (2023)](https://doi.org/10.1101/2023.06.29.543848) |  [code](https://github.com/LIYUESEN/druggpt) 

* **PrefixMol: Target- and Chemistry-aware Molecule Design via Prefix Embedding** [2023]   
 Gao, Zhangyang, Yuqi Hu, Cheng Tan, and Stan Z. Li.    
  [arXiv:2302.07120 (2023)](https://arxiv.org/abs/2302.07120) |  [code](https://github.com/A4Bio/PrefixMolf) 

* **DecompDiff: Diffusion Models with Decomposed Priors for Structure-Based Drug Design** [2023]   
 Guan, Jiaqi, Xiangxin Zhou, Yuwei Yang, Yu Bao, Jian Peng, Jianzhu Ma, Qiang Liu, Liang Wang, and Quanquan Gu.    
  [ICML (2023)](https://openreview.net/forum?id=9qy9DizMlr) |  [code](https://github.com/bytedance/DecompDiff) 

* **LS-MolGen: Ligand-and-Structure Dual-Driven Deep Reinforcement Learning for Target-Specific Molecular Generation Improves Binding Affinity and Novelty** [2023]  
 Li, Song, Chao Hu, Song Ke, Chenxing Yang, Jun Chen, Yi Xiong, Hao Liu, and Liang Hong.   
  [J. Chem. Inf. Model. （2023）](https://pubs.acs.org/doi/full/10.1021/acs.jcim.3c00587) |  [code](https://github.com/songleee/LS-MolGen) 

* **Accelerating drug target inhibitor discovery with a deep generative foundation model** [2023]  
 Vijil Chenthamarakshan et al.    
  [Sci. Adv.9,eadg7865(2023)](https://www.science.org/doi/10.1126/sciadv.adg7865) |  [code](https://zenodo.org/record/7863805) 

* **A Simple Way to Incorporate Target Structural Information in Molecular Generative Models** [2023]  
 Zhang, Wenyi, Kaiyue Zhang, and Jing Huang.   
  [Journal of Chemical Information and Modeling (2023)](https://pubs.acs.org/doi/10.1021/acs.jcim.3c00293) |  [code](https://github.com/JingHuangLab/SWIT) 

* **A Protein-Ligand Interaction-focused 3D Molecular Generative Framework for Generalizable Structure-based Drug Design** [2023]  
 Zhung W, Kim H, Kim WY.   
  [chemrxiv-2023-jsjwx](https://doi.org/10.26434/chemrxiv-2023-jsjwx) |  [code](https://github.com/ACE-KAIST/DeepICL) 

* **Mol-Zero-GAN: Zero-Shot Adaptation of Molecular Generative Adversarial Network for Specific Protein Targets** [2023]   
Ravipas Aphikulvanich*， Natapol Pornputtapong， Duangdao Wichadakul   
[chemrxiv-2023-lv2m1](https://doi.org/10.26434/chemrxiv-2023-lv2m1) |  [code](https://github.com/cucpbioinfo/Mol-Zero-GAN) 

* **Molecule Generation For Target Protein Binding with Structural Motifs** [2023]   
  Zhang, Zaixi, Yaosen Min, Shuxin Zheng, and Qi Liu.   
  [The Eleventh International Conference on Learning Representations. (2023)](https://openreview.net/forum?id=Rq13idF0F73) |  [code](https://github.com/zaixizhang/FLAG) 

* **Deep generative model for drug design from protein target sequence** [2023]   
Yangyang Chen, Zixu Wang, Lei Wang, Jianmin Wang, Pengyong Li, Dongsheng Cao, Xiangxiang Zeng, Xiucai Ye & Tetsuya Sakurai.   
[J Cheminform 15, 38 (2023)](https://doi.org/10.1186/s13321-023-00702-2) |  [code](https://github.com/viko-3/TargetGAN) 

* **3D Equivariant Diffusion for Target-Aware Molecule Generation and Affinity Prediction** [2023]   
  Guan, Jiaqi, Wesley Wei Qian, Xingang Peng, Yufeng Su, Jian Peng, and Jianzhu Ma.   
  [The Eleventh International Conference on Learning Representations. (2023)](https://openreview.net/forum?id=kJqXEPXMsE0) |  [code](https://github.com/guanjq/targetdiff) 

* **Target Specific De Novo Design of Drug Candidate Molecules with Graph Transformer-based Generative Adversarial Networks** [2023]     
  Ünlü, Atabey, Elif Çevrim, Ahmet Sarıgün, Hayriye Çelikbilek, Heval Ataş Güvenilir, Altay Koyaş, Deniz Cansen Kahraman, Ahmet Rifaioğlu, and Abdurrahman Olğaç.   
  [arXiv:2302.07868 (2023)](https://arxiv.org/abs/2302.07868)

* **Structure-based Drug Design with Equivariant Diffusion Models** [2023]  
Schneuing, A., Du, Y., Harris, C., Jamasb, A., Igashov, I., Du, W., ... & Correia, B.   
[arXiv:2210.13695 (2022)](https://openreview.net/forum?id=uKmuzIuVl8z) |  [code](https://github.com/arneschneuing/DiffSBDD) 

* **Icolos: a workflow manager for structure-based post-processing of de novo generated small molecules**  [2022]   
 Moore, J. Harry, Matthias R. Bauer, Jeff Guo, Atanas Patronov, Ola Engkvist, and Christian Margreitter.   
  [Bioinformatics 38.21 (2022)](https://doi.org/10.1093/bioinformatics/btac614) |  [code](https://github.com/MolecularAI/Icolos) 

* **A multilevel generative framework with hierarchical self-contrasting for bias control and transparency in structure-based ligand design** [2022]   
  Chan, Lucian, Rajendra Kumar, Marcel Verdonk, and Carl Poelking.   
  [Nat Mach Intell 4, 1130–1142 (2022)](https://doi.org/10.1038/s42256-022-00564-7) |  [code](https://github.com/capoe/libpqr) 

* **Reinforced Genetic Algorithm for Structure-based Drug Design** [2022]   
  Fu, Tianfan, Wenhao Gao, Connor Coley, and Jimeng Sun.  
  [Advances in Neural Information Processing Systems 35 (2022)](https://openreview.net/forum?id=Qx6UPW0r9Lf) |  [code](https://github.com/futianfan/reinforced-genetic-algorithm) 

* **Exploiting pretrained biochemical language models for targeted drug design** [2022]  
  Uludoğan, Gökçe, Elif Ozkirimli, Kutlu O. Ulgen, Nilgün Karalı, and Arzucan Özgür.  
  [Bioinformatics 38.Supplement_2 (2022)](https://doi.org/10.1093/bioinformatics/btac482) |  [code](https://github.com/boun-tabi/biochemical-lms-for-drug-design) 

* **RELATION: A Deep Generative Model for Structure-Based De Novo Drug Design** [2022]  
  Wang, M., Hsieh, C.Y., Wang, J., Wang, D., Weng, G., Shen, C., Yao, X., Bing, Z., Li, H., Cao, D. and Hou, T.,  
  [Journal of Medicinal Chemistry 65.13 (2022)](https://pubs.acs.org/doi/10.1021/acs.jmedchem.2c00732) |  [code](https://github.com/micahwang/RELATION) 

* **Tailoring Molecules for Protein Pockets: a Transformer-based Generative Solution for Structured-based Drug Design** [2022]  
  Wu, K., Xia, Y., Fan, Y., Deng, P., Liu, H., Wu, L., ... & Liu, T. Y.  
  [arXiv:2209.06158 (2022)](https://arxiv.org/abs/2209.06158) |  [code](https://github.com/HankerWu/TamGent) 

* **De novo design of protein target specific scaffold-based Inhibitors via Reinforcement Learning** [2022]  
  Bontha M, McNaughton A, Knutson C, Pope J, Kumar N.  
  [arXiv:2205.10473 (2022)](https://openreview.net/forum?id=k-ES3OH7eqp)

* **AlphaDrug: protein target specific de novo molecular generation** [2022]  
  Qian, Hao, Cheng Lin, Dengwei Zhao, Shikui Tu, and Lei Xu.  
  [PNAS Nexus 1.4 (2022)](https://academic.oup.com/pnasnexus/article/1/4/pgac227/6751929) |  [code](https://github.com/CMACH508/AlphaDrug) 

* **LIMO: Latent Inceptionism for Targeted Molecule Generation** [2022]  
  Eckmann, Peter, Kunyang Sun, Bo Zhao, Mudong Feng, Michael K. Gilson, and Rose Yu.  
  [arXiv:2206.09010 (2022)](https://arxiv.org/abs/2206.09010) |  [code](https://github.com/rose-stl-lab/limo) 

* **Pocket2Mol: Efficient Molecular Sampling Based on 3D Protein Pockets** [2022]  
 Peng, Xingang, Shitong Luo, Jiaqi Guan, Qi Xie, Jian Peng, and Jianzhu Ma.  
  [International Conference on Machine Learning. PMLR, (2022)](https://proceedings.mlr.press/v162/peng22b.html) |  [code](https://github.com/pengxingang/Pocket2Mol) 

* **Autonomous molecule generation using reinforcement learning and docking to develop potential novel inhibitors** [2022]  
  Jeon, W., Kim, D.  
  [Sci Rep 10, 22104 (2020)](https://doi.org/10.1038/s41598-020-78537-2) |  [code](https://github.com/wsjeon92/morld) 

* **Target-Focused Library Design by Pocket-Applied Computer Vision and Fragment Deep Generative Linking** [2022]  
  Eguida, Merveille, Christel Schmitt-Valencia, Marcel Hibert, Pascal Villa, and Didier Rognan.  
  [Journal of Medicinal Chemistry 65.20 (2022)](https://pubs.acs.org/doi/10.1021/acs.jmedchem.2c00931) |  [code](https://github.com/kimeguida/POEM) 

* **Incorporating Target-Specific Pharmacophoric Information into Deep Generative Models for Fragment Elaboration** [2022]  
  Hadfield, Thomas E., Fergus Imrie, Andy Merritt, Kristian Birchall, and Charlotte M. Deane.  
  [J. Chem. Inf. Model. 2022, 62, 10, 2280–2292](https://pubs.acs.org/doi/10.1021/acs.jcim.1c01311) |  [code](https://github.com/tomhadfield95/STRIFE) 

* **Fragment-Based Ligand Generation Guided By Geometric Deep Learning On Protein-Ligand Structure** [2022]  
  Powers, Alexander S., Helen H. Yu, Patricia Suriana, and Ron O. Dror.  
  [bioRxiv (2022)](https://www.biorxiv.org/content/10.1101/2022.03.17.484653v2) 

* **Zero-Shot 3D Drug Design by Sketching and Generating** [2022]  
  Long, Siyu, Yi Zhou, Xinyu Dai, and Hao Zhou.   
  [arXiv:2209.13865 (2022)](https://nips.cc/media/neurips-2022/Slides/54457.pdf) |  [code](https://github.com/longlongman/DESERT) 

* **Structure-based de novo drug design using 3D deep generative models** [2021]  
  Li, Yibo, Jianfeng Pei, and Luhua Lai.    
  [Chemical science 12.41 (2021)](https://pubs.rsc.org/en/content/articlelanding/2021/sc/d1sc04444c)

* **Transformer neural network for protein-specific de novo drug generation as a machine translation proble** [2021]  
  Grechishnikova, Daria.   
  [Sci Rep 11, 321 (2021)](https://www.nature.com/articles/s41598-020-79682-4) |  [code](https://github.com/dariagrechishnikova/molecule_structure_generation) 

* **Structure-aware generation of drug-like molecules** [2021]  
  Drotár, P., Jamasb, A.R., Day, B., Cangea, C. and Liò, P.,   
  [arXiv:2111.04107 (2021)](https://arxiv.org/abs/2111.04107) 

* **A 3D Generative Model for Structure-Based Drug Design** [2021]  
  Luo, S., Guan, J., Ma, J., & Peng, J.   
  [Advances in Neural Information Processing Systems 34 (2021)](https://openreview.net/forum?id=yDwfVD_odRo) |  [code](https://github.com/luost26/3D-Generative-SBDD) 

* **Structure-Based de Novo Molecular Generator Combined with Artificial Intelligence and Docking Simulations** [2021]  
  Ma, B., Terayama, K., Matsumoto, S., Isaka, Y., Sasakura, Y., Iwata, H., Araki, M. and Okuno, Y.   
  [J. Chem. Inf. Model. 2021, 61, 7, 3304–3313](https://pubs.acs.org/doi/10.1021/acs.jcim.1c00679) |  [code](https://github.com/clinfo/SBMolGen) 








## Fragment-based deep molecular generative models








### Scaffold-based DMGs


* **REINVENT4: Modern AI–Driven Generative Molecule Design** [2023]   
  Loeffler H, He J, Tibo A, Janet JP, Voronov A, Mervin L, et al.   
  [chemrxiv-2023-xt65x  (2023)](https://doi.org/10.26434/chemrxiv-2023-xt65x) |  [code](https://github.com/MolecularAI/REINVENT4) 

* **DiffDec: Structure-Aware Scaffold Decoration with an End-to-End Diffusion** [2023]   
 Xie, Junjie, Sheng Chen, Jinping Lei, and Yuedong Yang.    
  [bioRxiv (2023)](https://doi.org/10.1101/2023.10.08.561377) 

* **D-SMGE: a pipeline for scaffold-based molecular generation and evaluation**  [2023]   
 Chao Xu, Runduo Liu, Shuheng Huang, Wenchao Li, Zhe Li, Hai-Bin Luo.   
  [Briefings in Bioinformatics. (2023)](https://doi.org/10.1093/bib/bbad327) |  [code](https://github.com/ZheLi-Lab-Collaboration/3D-SMGE) 

* **ScaffoldGVAE: Scaffold Generation and Hopping of Drug Molecules via a Variational Autoencoder Based on Multi-View Graph Neural Networks**  [2023]   
 Hu, Chao, Song Li, Chenxing Yang, Jun Chen, Yi Xiong, Guisheng Fan, Hao Liu, and Liang Hong.   
  [J Cheminform 15, 91 (2023)](https://doi.org/10.1186/s13321-023-00766-0) |  [Research Square. (2023)](https://www.researchsquare.com/article/rs-3254116/v1) |  [code](https://github.com/ecust-hc/ScaffoldGVAE) 

* **DiffHopp: A Graph Diffusion Model for Novel Drug Design via Scaffold Hopping** [2023]   
 Torge, Jos, Charles Harris, Simon V. Mathis, and Pietro Lió.    
  [ICML (2023)](https://icml-compbio.github.io/2023/papers/WCBICML2023_paper69.pdf) |  [code](https://github.com/jostorge/diffusion-hopping) 

* **DrugEx v3: scaffold-constrained drug design with graph transformer-based reinforcement learning** [2023]  
  Liu, X., Ye, K., van Vlijmen, H.W.T. et al.    
  [J Cheminform 15, 24 (2023)](https://doi.org/10.1186/s13321-023-00694-z) |  [code](https://github.com/CDDLeiden/DrugEx) 

* **Sc2Mol: a scaffold-based two-step molecule generator with variational autoencoder and transformer** [2023]  
  Zhirui Liao, Lei Xie, Hiroshi Mamitsuka, Shanfeng Zhu.  
  [Bioinformatics 39.1 (2023)](https://doi.org/10.1093/bioinformatics/btac814) |  [code](https://github.com/zhiruiliao/Sc2Mol) 

* **De novo design of protein target specific scaffold-based Inhibitors via Reinforcement Learning** [2022]  
  Bontha M, McNaughton A, Knutson C, Pope J, Kumar N.  
  [arXiv:2205.10473 (2022)](https://openreview.net/forum?id=k-ES3OH7eqp)

* **LibINVENT: Reaction-based Generative Scaffold Decoration for in Silico Library Design** [2022]  
  Fialková, V., Zhao, J., Papadopoulos, K., Engkvist, O., Bjerrum, E.J., Kogej, T. and Patronov, A   
  [J. Chem. Inf. Model. 2022, 62, 9, 2046–2063](https://pubs.acs.org/doi/10.1021/acs.jcim.1c00469) |  [code](https://github.com/MolecularAI/Lib-INVENT) 

* **Learning to Extend Molecular Scaffolds with Structural Motifs** [2022]  
  Maziarz, Krzysztof, Henry Jackson-Flux, Pashmina Cameron, Finton Sirockin, Nadine Schneider, Nikolaus Stiefl, Marwin Segler, and Marc Brockschmidt.   
  [arXiv:2103.03864 (2021)](https://openreview.net/forum?id=ZTsoE8G3GG)

* **Deep scaffold hopping with multimodal transformer neural networks** [2021]  
Zheng, Shuangjia, Zengrong Lei, Haitao Ai, Hongming Chen, Daiguo Deng, and Yuedong Yang.  
  [J Cheminform 13, 87 (2021)](https://doi.org/10.1186/s13321-021-00565-5) |  [code](https://github.com/prokia/deepHops) 

* **Kinase Inhibitor Scaffold Hopping with Deep Learning Approaches** [2021]  
Hu, Lizhao, Yuyao Yang, Shuangjia Zheng, Jun Xu, Ting Ran, and Hongming Chen.  
  [J. Chem. Inf. Model. 2021, 61, 10, 4900–4912](https://pubs.acs.org/doi/10.1021/acs.jcim.1c00608) |  [code](https://github.com/YuYaoYang2333/SyntaLinker) 

* **3D-Scaffold: A Deep Learning Framework to Generate 3D Coordinates of Drug-like Molecules with Desired Scaffolds** [2021]  
Joshi, Rajendra P., Niklas WA Gebauer, Mridula Bontha, Mercedeh Khazaieli, Rhema M. James, James B. Brown, and Neeraj Kumar.  
  [J. Phys. Chem. B 2021, 125, 44, 12166–12176](https://pubs.acs.org/doi/10.1021/acs.jpcb.1c06437) |  [code](https://github.com/PNNL-CompBio/3D_Scaffold) 

* **SMILES-Based Deep Generative Scaffold Decorator for De-Novo Drug Design** [2020]  
Arús-Pous, Josep, Atanas Patronov, Esben Jannik Bjerrum, Christian Tyrchan, Jean-Louis Reymond, Hongming Chen, and Ola Engkvist.  
  [J Cheminform 12, 38 (2020)](https://doi.org/10.1186/s13321-020-00441-8) | [chemrxiv.11638383.v1](https://doi.org/10.26434/chemrxiv.11638383.v1) |  [code](https://github.com/undeadpixel/reinvent-scaffold-decorator) 

* **Scaffold-based molecular design with a graph generative model** [2020]  
Lim, Jaechang, Sang-Yeon Hwang, Seokhyun Moon, Seungsu Kim, and Woo Youn Kim.  
  [Chemical science 11.4 (2020)](https://pubs.rsc.org/en/content/articlelanding/2020/SC/C9SC04503A) |  [code](https://github.com/jaechanglim/GGM) 






### Motifs-based DMGs



* **Learning Subpocket Prototypes for Generalizable Structure-based Drug Design** [2023]   
ZHANG Z, Liu Q.  
[ICML'23: Proceedings of the 40th International Conference on Machine Learning (2023)](https://dl.acm.org/doi/10.5555/3618408.3620143) |  [code](https://github.com/zaixizhang/DrugGPS_ICML23) 

* **MAGNet: Motif-Agnostic Generation of Molecules from Shapes** [2023]   
  Hetzel, Leon, Johanna Sommer, Bastian Rieck, Fabian Theis, and Stephan Günnemann.   
  [arXiv:2305.19303 (2023)](https://arxiv.org/abs/2305.19303) 

* **Molecule Generation For Target Protein Binding with Structural Motifs** [2023]   
  Zhang, Zaixi, Yaosen Min, Shuxin Zheng, and Qi Liu.   
  [The Eleventh International Conference on Learning Representations. (2023)](https://openreview.net/forum?id=Rq13idF0F73) |  [code](https://github.com/zaixizhang/FLAG) 

* **De Novo Molecular Generation via Connection-aware Motif Mining** [2023]   
  Zijie Geng, Shufang Xie, Yingce Xia, Lijun Wu, Tao Qin, Jie Wang, Yongdong Zhang, Feng Wu, Tie-Yan Liu   
  [arXiv:2302.01129 (2023)](https://openreview.net/pdf?id=Q_Jexl8-qDihttps://openreview.net/pdf?id=Q_Jexl8-qDi) |  [code](https://github.com/MIRALab-USTC/AI4Sci-MiCaM) 

* **Learning to Extend Molecular Scaffolds with Structural Motifs** [2022]  
  Maziarz, Krzysztof, Henry Jackson-Flux, Pashmina Cameron, Finton Sirockin, Nadine Schneider, Nikolaus Stiefl, Marwin Segler, and Marc Brockschmidt.   
  [arXiv:2103.03864 (2021)](https://openreview.net/forum?id=ZTsoE8G3GG)

* **Hierarchical generation of molecular graphs using structural motifs** [2020]  
  Jin, Wengong, Regina Barzilay, and Tommi Jaakkola.   
  [International conference on machine learning. PMLR, (2020)](https://dl.acm.org/doi/abs/10.5555/3524938.3525387) |  [code](https://github.com/wengong-jin/hgraph2graph) 









### Fragment-based DMGs

* **t-SMILES: a fragment-based molecular representation framework for de novo ligand design** [2024]   
  Wu, JN., Wang, T., Chen, Y. et al.    
  [Nat Commun 15, 4993 (2024)](https://doi.org/10.1038/s41467-024-49388-6) |  [code](https://github.com/juanniwu/t-SMILES) 

* **Gotta be SAFE: A New Framework for Molecular Design** [2024]   
  Noutahi, Emmanuel, Cristian Gabellini, Michael Craig, Jonathan SC Lim, and Prudencio Tossou.   
  [Digital Discovery (2024)](https://doi.org/10.1039/D4DD00019F) |  [arXiv:2310.10773 (2023)](https://arxiv.org/abs/2310.10773) |  [code](https://github.com/datamol-io/safe)

* **FREED++: Improving RL Agents for Fragment-Based Molecule Generation by Thorough Reproduction** [2024]   
  Telepov, Alexander, Artem Tsypin, Kuzma Khrabrov, Sergey Yakukhnov, Pavel Strashnov, Petr Zhilyaev, Egor Rumiantsev et al.    
  [arXiv:2401.09840 (2024)](https://arxiv.org/abs/2401.09840) |  [code](https://github.com/AIRI-Institute/FFREED) 

* **Geometric Deep Learning for Structure-Based Ligand Design** [2023]   
  Alexander S. Powers, Helen H. Yu, Patricia Suriana, Rohan V. Koodli, Tianyu Lu, Joseph M. Paggi, and Ron O. Dror.   
  [ACS Cent. Sci. (2023)](https://doi.org/10.1021/acscentsci.3c00572) 

* **Autoregressive fragment-based diffusion for pocket-aware ligand design** [2023]   
 Ghorbani, Mahdi, Leo Gendelev, Paul Beroza, and Michael Keiser.    
  [NeurIPS 2023 Generative AI and Biology (GenBio) Workshop.  (2023)](https://openreview.net/forum?id=E3HN48zjam) |  [code](https://github.com/ghorbanimahdi73/autofragdiff) 

* **A flexible data-free framework for structure-based de novo drug design with reinforcement learning** [2023]   
  Hongyan Du, Dejun Jiang, Odin Zhang, Zhenxing Wu, Junbo Gao, Xujun Zhang, Xiaorui Wang, Yafeng Deng, Yu Kang, Dan Li, Peichen Pan, Chang-Yu Hsieh, Tingjun Hou.   
  [Chemical Science (2023)](https://doi.org/10.1039/D3SC04091G) |  [code](https://github.com/Brian-hongyan/3D-MCTS) 

* **Interpretable Fragment-Based Molecule Design with Self-Learning Entropic Population Annealing**  [2023]   
 Li, J., Sumita, M., Tamura, R. and Tsuda, K.   
  [Advanced Intelligent Systems (2023)](https://doi.org/10.1002/aisy.202300189) |  [code](https://github.com/tsudalab/MolSLEPA) 

* **Expanding Bioactive Fragment Space with the Generated Database GDB-13s**  [2023]   
 Buehler, Ye, and Jean-Louis Reymond.   
  [J. Chem. Inf. Model. (2023)](https://pubs.acs.org/doi/10.1021/acs.jcim.3c01096) |  [code](https://github.com/Ye-Buehler/Molecule_Breakdown_Model) 

* **ReBADD-SE: Multi-objective molecular optimisation using SELFIES fragment and off-policy self-critical sequence training** [2023]   
  Choi, Jonghwan, Sangmin Seo, Seungyeon Choi, Shengmin Piao, Chihyun Park, Sung Jin Ryu, Byung Ju Kim, and Sanghyun Park.   
  [Computers in Biology and Medicine 157 (2023)](https://doi.org/10.1016/j.compbiomed.2023.106721) |  [code](https://github.com/mathcom/ReBADD-SE) 

* **ReBADD-SE: Multi-objective molecular optimisation using SELFIES fragment and off-policy self-critical sequence training** [2023]   
  Choi, Jonghwan, Sangmin Seo, Seungyeon Choi, Shengmin Piao, Chihyun Park, Sung Jin Ryu, Byung Ju Kim, and Sanghyun Park.   
  [Computers in Biology and Medicine 157 (2023)](https://doi.org/10.1016/j.compbiomed.2023.106721) |  [code](https://github.com/mathcom/ReBADD-SE) 

* **Integrating Reaction Schemes, Reagent Databases, and Virtual Libraries into Fragment-Based Design by Reinforcement Learning**  [2023]   
 Sauer, Susanne, Hans Matter, Gerhard Hessler, and Christoph Grebner.   
  [J. Chem. Inf. Model. (2023)](https://doi.org/10.1021/acs.jcim.3c00735) |  [code](https://github.com/Sanofi-Public/IDD-papers-fragrl) 

* **Construction of order-independent molecular fragments space with vector quantised graph autoencoder**  [2023]   
 Akhmetshin, Timur and Lin, Albert and Madzhidov, Timur and Varnek, Alexandre   
  [chemrxiv-2023-5zmvw](https://doi.org/10.26434/chemrxiv-2023-5zmvw) |  [code](https://github.com/Laboratoire-de-Chemoinformatique/VQGAE) 

* **Fragment-based Molecule Design with Self-learning Entropic Population Annealing** [2023]   
   [code](https://github.com/tsudalab/MolSLEPA) 

* **Molecular Generation with Reduced Labeling through Constraint Architecture** [2023]   
 Wang, Jike, Yundian Zeng, Huiyong Sun, Junmei Wang, Xiaorui Wang, Ruofan Jin, Mingyang Wang et al.    
  [J. Chem. Inf. Model. 2023, 63, 11, 3319–3327](https://pubs.acs.org/doi/full/10.1021/acs.jcim.3c00579) |  [code](https://github.com/jkwang93/Frag-G_M) 

* **Tree-Invent: A novel molecular generative model constrained with topological tree** [2023]   
 Mingyuan Xu, HongMing Chen.   
  [chemrxiv-2023-m77vk](https://doi.org/10.26434/chemrxiv-2023-m77vk) |  [code](https://github.com/MingyuanXu/Tree-Invent) 

* **MacFrag: segmenting large-scale molecules to obtain diverse fragments with high qualities** [2023]  
  Yanyan Diao, Feng Hu, Zihao Shen, Honglin Li*.  
  [Bioinformatics (2023)](https://doi.org/10.1093/bioinformatics/btad012) |  [code](https://github.com/yydiao1025/MacFrag) 

* **Fragment-based Deep Molecular Generation using Hierarchical Chemical Graph Representation and Multi-Resolution Graph Variational Autoencoder** [2023]  
  Gao, Zhenxiang, Xinyu Wang, Blake Blumenfeld Gaines, Xuetao Shi, Jinbo Bi, and Minghu Song.  
  [Molecular Informatics (2023)](https://doi.org/10.1002/minf.202200215)

* **Fragment-based t-SMILES for de novo molecular generation** [2023]  
  Wu, Juan-Ni, Tong Wang, Yue Chen, Li-Juan Tang, Hai-Long Wu, and Ru-Qin Yu.  
  [arXiv:2301.01829 (2023)](https://arxiv.org/abs/2301.01829) |  [code](https://github.com/juanniwu/t-SMILES) 

* **Target-Focused Library Design by Pocket-Applied Computer Vision and Fragment Deep Generative Linking** [2022]  
  Eguida, Merveille, Christel Schmitt-Valencia, Marcel Hibert, Pascal Villa, and Didier Rognan.  
  [Journal of Medicinal Chemistry 65.20 (2022): 13771-13783](https://pubs.acs.org/doi/10.1021/acs.jmedchem.2c00931) |  [code](https://github.com/kimeguida/POEM) 

* **Incorporating Target-Specific Pharmacophoric Information into Deep Generative Models for Fragment Elaboration** [2022]  
  Hadfield, Thomas E., Fergus Imrie, Andy Merritt, Kristian Birchall, and Charlotte M. Deane.  
  [J. Chem. Inf. Model. 2022, 62, 10, 2280–2292](https://pubs.acs.org/doi/10.1021/acs.jcim.1c01311) |  [code](https://github.com/tomhadfield95/STRIFE) 

* **Fragment-Based Ligand Generation Guided By Geometric Deep Learning On Protein-Ligand Structure** [2022]  
  Powers, Alexander S., Helen H. Yu, Patricia Suriana, and Ron O. Dror.  
  [bioRxiv (2022)](https://www.biorxiv.org/content/10.1101/2022.03.17.484653v2) 

* **FAME: Fragment-based Conditional Molecular Generation for Phenotypic Drug Discovery** [2022]  
  Pham, Thai-Hoang, Lei Xie, and Ping Zhang.  
  [SDM. Society for Industrial and Applied Mathematics, (2022)](https://www.biorxiv.org/content/10.1101/2022.01.21.477292v1) 

* **Scalable Fragment-Based 3D Molecular Design with Reinforcement Learning** [2022]  
  Flam-Shepherd, Daniel, Alexander Zhigalin, and Alán Aspuru-Guzik.  
  [arXiv:2202.00658 (2022)](https://arxiv.org/abs/2202.00658) 

* **Hit and Lead Discovery with Explorative RL and Fragment-based Molecule Generation** [2021]  
  Yang, S., Hwang, D., Lee, S., Ryu, S., & Hwang, S. J.  
  [Advances in Neural Information Processing Systems 34 (2021)](https://arxiv.org/abs/2110.01219) |  [code](https://github.com/AITRICS/FREED) 

* **Automated Generation of Novel Fragments Using Screening Data, a Dual SMILES Autoencoder, Transfer Learning and Syntax Correction** [2021]  
  Bilsland, Alan E., Kirsten McAulay, Ryan West, Angelo Pugliese, and Justin Bower.  
  [J. Chem. Inf. Model. 2021, 61, 6, 2547–2559](https://pubs.acs.org/doi/10.1021/acs.jcim.0c01226) |  [code](https://github.com/abilsland/fragmentEncoder) 

* **A Deep Generative Model for Fragment-Based Molecule Generation** [2020]  
  Podda, Marco, Davide Bacciu, and Alessio Micheli.  
  [International Conference on Artificial Intelligence and Statistics. PMLR, (2020)](http://proceedings.mlr.press/v108/) |  [code](https://github.com/hengwei-chan/fragment-based-de-novo) 

* **Multi-Objective Molecule Generation using Interpretable Substructures** [2020]  
  Jin, Wengong, Regina Barzilay, and Tommi Jaakkola.   
  [International conference on machine learning. PMLR, (2020)](http://proceedings.mlr.press/v119/jin20b.html) |  [code](https://github.com/wengong-jin/multiobj-rationale) 

* **Fragment Graphical Variational AutoEncoding for Screening Molecules with Small Data** [2019]  
  Armitage, John, Leszek J. Spalek, Malgorzata Nguyen, Mark Nikolka, Ian E. Jacobs, Lorena Marañón, Iyad Nasrallah et al.  
  [arXiv:1910.13325 (2019)](https://arxiv.org/abs/1910.13325) |  [code](https://github.com/OE-FET/FraGVAE) 




### Linkers-based DMGs


* **Equivariant 3D-conditional diffusion model for molecular linker design** [2024]   
 Igashov, I., Stärk, H., Vignac, C. et al.    
  [Nat Mach Intell (2024)](https://doi.org/10.1038/s42256-024-00815-9)  |  [code](https://github.com/igashov/DiffLinker) 

* **GRELinker: A Graph-based Generative Model for Molecular Linker Design with Reinforcement and Curriculum Learning** [2024]   
  Zhang, Hao, Jinchao Huang, Junjie Xie, Weifeng Huang, Yuedong Yang, Mingyuan Xu, Jinping Lei, and Hongming Chen.  
  [J. Chem. Inf. Model.  (2024)](https://doi.org/10.1021/acs.jcim.3c01700) | [code](https://github.com/howzh728/GRELinker)  

* **LinkerNet: Fragment Poses and Linker Co-Design with 3D Equivariant Diffusion** [2023]   
 Guan, Jiaqi, Xingang Peng, PeiQi Jiang, Yunan Luo, Jian Peng, and Jianzhu Ma    
  [NeurIPS 2023.  (2023)](https://openreview.net/forum?id=6EaLIw3W7c) |  [code](https://github.com/guanjq/LinkerNet) 

* **3D Based Generative PROTAC Linker Design with Reinforcement Learning**  [2023]   
 baiqing li, and Hongming Chen.   
  [chemrxiv-2023-j740w (2023)](https://doi.org/10.26434/chemrxiv-2023-j740w) |  [code](https://github.com/jidushanbojue/Protac-invent) 

* **Reinforcement Learning-Driven Linker Design via Fast Attention-based Point Cloud Alignment** [2023]   
 Neeser, Rebecca M., Mehmet Akdel, Daniel Kovtun, and Luca Naef.   
  [arXiv:2306.08166 (2023)](https://arxiv.org/abs/2306.08166) |  [code](https://github.com/aivant/ShapeLinker) 

* **Fragment Linker Prediction Using the Deep Encoder-Decoder Network for PROTACs Drug Design** [2023]  
Kao, Chien-Ting, Chieh-Te Lin, Cheng-Li Chou, and Chu-Chung Lin.   
[J. Chem. Inf. Model. 2023, 63, 10, 2918–2927](https://pubs.acs.org/doi/full/10.1021/acs.jcim.2c01287) |  [code](https://github.com/AnHorn/AIMLinker) 

* **Equivariant 3D-Conditional Diffusion Models for Molecular Linker Desig** [2023]  
Igashov, I., Stärk, H., Vignac, C., Satorras, V.G., Frossard, P., Welling, M., Bronstein, M. and Correia, B.,   
[arXiv:2210.05274 (2022)](https://arxiv.org/abs/2210.05274) |  [code](https://github.com/igashov/DiffLinker) 

* **DRlinker: Deep Reinforcement Learning for Optimization in Fragment Linking Design** [2022]  
  Tan, Y., Dai, L., Huang, W., Guo, Y., Zheng, S., Lei, J., ... & Yang, Y.  
  [J. Chem. Inf. Model. 2022, 62, 23, 5907–5917](https://pubs.acs.org/doi/10.1021/acs.jcim.2c00982) |  [code](https://github.com/biomed-AI/DRlinker) 

* **3DLinker: An E(3) Equivariant Variational Autoencoder for Molecular Linker Design** [2022]  
  Huang, Yinan, Xingang Peng, Jianzhu Ma, and Muhan Zhang.   
  [arXiv:2205.07309 (2022)](https://arxiv.org/abs/2205.07309) |  [code](https://github.com/GraphPKU/3DLinker) 

* **SyntaLinker-Hybrid: A deep learning approach for target specific drug design** [2022]  
  Feng, Yu, Yuyao Yang, Wenbin Deng, Hongming Chen, and Ting Ran.    
  [Artificial Intelligence in the Life Sciences 2 (2022)](https://doi.org/10.1016/j.ailsci.2022.100035) 

* **Deep Generative Models for 3D Linker Design** [2020]  
  Imrie, Fergus, Anthony R. Bradley, Mihaela van der Schaar, and Charlotte M. Deane.  
  [J. Chem. Inf. Model. 2020, 60, 4, 1983–1995](https://pubs.acs.org/doi/10.1021/acs.jcim.9b01120) |  [code](https://github.com/fimrie/DeLinker) 

* **SyntaLinker: automatic fragment linking with deep conditional transformer neural networks** [2020]  
  Yang, Yuyao, Shuangjia Zheng, Shimin Su, Chao Zhao, Jun Xu, and Hongming Chen.  
  [Chemical science 11.31 (2020)](https://pubs.rsc.org/en/content/articlelanding/2020/sc/d0sc03126g) |  [code](https://github.com/YuYaoYang2333/SyntaLinker) 








## Chemical Reaction-based deep molecular generative models



* **Integrating Reaction Schemes, Reagent Databases, and Virtual Libraries into Fragment-Based Design by Reinforcement Learning**  [2023]   
 Sauer, Susanne, Hans Matter, Gerhard Hessler, and Christoph Grebner.   
  [J. Chem. Inf. Model. (2023)](https://doi.org/10.1021/acs.jcim.3c00735) |  [code](https://github.com/Sanofi-Public/IDD-papers-fragrl) 

* **Combining Evolutionary Algorithms with Reaction Rules Towards Focused Molecular Design**  [2023]   
 Correia, João, Vítor Pereira, and Miguel Rocha.   
  [Proceedings of the Genetic and Evolutionary Computation Conference (2023)](https://doi.org/10.1145/3583131.3590413) |  [code](https://github.com/BioSystemsUM/ReactEA) 

* **Uni-RXN: A Unified Framework Bridging the Gap between Chemical Reaction Pretraining and Conditional Molecule Generation** [2023]   
Bo Qiang, Yiran Zhou, Yuheng Ding, Ningfeng Liu, Song Song, Liangren Zhang, Bo Huang, Zhenming Liu   
[arXiv:2303.06965 (2023)](https://arxiv.org/abs/2303.06965) |  [code](https://github.com/qiangbo1222/Uni-RXN-official) 

* **Molecular Generative Model via Retrosynthetically Prepared Chemical Building Block Assembly** [2023]   
Seo, Seonghwan, Jaechang Lim, and Woo Youn Kim.   
[Advanced Science (2023)](https://doi.org/10.1002/advs.202206674) |  [code](https://github.com/SeonghwanSeo/BBAR) 

* **Synthesis-Aware Generation of Structural Analogues** [2022]   
  Dolfus, Uschi, Hans Briem, and Matthias Rarey.   
  [J. Chem. Inf. Model. 2022, 62, 15, 3565–3576](https://pubs.acs.org/doi/10.1021/acs.jcim.2c00246) |  [code](https://github.com/whitead/synspace) 

* **ChemistGA: A Chemical Synthesizable Accessible Molecular Generation Algorithm for Real-World Drug Discovery** [2022]   
  Wang, Jike, Xiaorui Wang, Huiyong Sun, Mingyang Wang, Yundian Zeng, Dejun Jiang, Zhenxing Wu et al.   
[Journal of Medicinal Chemistry 65.18 (2022)](https://pubs.acs.org/doi/10.1021/acs.jmedchem.2c01179) |  [code](https://github.com/jkwang93/ChemistGA) 

* **Generating reaction trees with cascaded variational autoencoders** [2022]   
  Nguyen, Dai Hai, and Koji Tsuda.   
  [The Journal of Chemical Physics 156.4 (2022)](https://doi.org/10.1063/5.0076749) |  [code](https://github.com/haidnguyen0909/rxngenerator) 

* **Synthesis-Aware Generation of Structural Analogues** [2022]   
  Dolfus, Uschi, Hans Briem, and Matthias Rarey.   
  [J. Chem. Inf. Model. 2022, 62, 15, 3565–3576](https://pubs.acs.org/doi/10.1021/acs.jcim.2c00246)

* **SynthI: A New Open-Source Tool for Synthon-Based Library Design** [2022]   
  Zabolotna, Yuliana, Dmitriy M. Volochnyuk, Sergey V. Ryabukhin, Kostiantyn Gavrylenko, Dragos Horvath, Olga Klimchuk, Oleksandr Oksiuta, Gilles Marcou, and Alexandre Varnek.   
  [J. Chem. Inf. Model. 2022, 62, 9, 2151–2163](https://pubs.acs.org/doi/10.1021/acs.jcim.1c00754) |  [code](https://github.com/Laboratoire-de-Chemoinformatique/Synt-On) 

* **Integrating Synthetic Accessibility with AI-based Generative Drug Design** [2021]   
  Parrot, Maud, Hamza Tajmouati, Vinicius Barros Ribeiro da Silva, Brian Atwood, Robin Fourcade, Yann Gaston-Mathé, Nicolas Do Huu, and Quentin Perron.   
  [chemrxiv-2021-jkhzw-v2](https://doi.org/10.26434/chemrxiv-2021-jkhzw-v2) |  [code](https://github.com/iktos/generation-under-synthetic-constraint) 







## Omics-based deep molecular generative models



* **Cross-modal Generation of Hit-like Molecules via Foundation Model Encoding of Gene Expression Signatures** [2023]   
  Jiabei Cheng, Xiaoyong Pan, Kaiyuan Yang, Shenghao Cao, Bin Liu, Ye Yuan.   
  [bioRxiv 2023.11.11.566725.  (2023)](https://doi.org/10.1101/2023.11.11.566725) |  [code](https://github.com/Bunnybeibei/GexMolGen) 

* **De novo drug design based on patient gene expression profiles via deep learning**  [2023]   
 Yamanaka, Chikashige, Shunya Uki, Kazuma Kaitoh, Michio Iwata, and Yoshihiro Yamanishi.   
  [Molecular Informatics (2023)](https://doi.org/10.1002/minf.202300064) |  [code](https://www.dropbox.com/s/vg3nxcio799h4ex/software.zip?dl=0) 

* **De Novo Design of Molecules with Multiaction Potential from Differential Gene Expression using Variational Autoencoder** [2023]   
 Pravalphruekul, Nutaya, Maytus Piriyajitakonkij, Phond Phunchongharn, and Supanida Piyayotai.   
  [J. Chem. Inf. Model.  (2023)](https://pubs.acs.org/doi/full/10.1021/acs.jcim.3c00355) |  [code](https://github.com/ChemEXL/BiCEV) 

* **Gex2SGen: Designing Drug-like Molecules from Desired Gene Expression Signatures** [2023]   
 Das, Dibyajyoti, Broto Chakrabarty, Rajgopal Srinivasan, and Arijit Roy.   
  [J. Chem. Inf. Model. 2023, 63, 7, 1882–1893](https://doi.org/10.1021/acs.jcim.2c01301)

* **PaccMannRL: De novo generation of hit-like anticancer molecules from transcriptomic data via reinforcement learning** [2021]  
  Born, Jannis, Matteo Manica, Ali Oskooei, Joris Cadow, Greta Markert, and María Rodríguez Martínez.   
  [Iscience 24.4 (2021)](https://doi.org/10.1016/j.isci.2021.102269) |  [code](https://github.com/PaccMann/paccmann_omics) 

* **Molecular Generation for Desired Transcriptome Changes With Adversarial Autoencoders** [2020]  
  Shayakhmetov, Rim, Maksim Kuznetsov, Alexander Zhebrak, Artur Kadurin, Sergey Nikolenko, Alexander Aliper, and Daniil Polykovskiy.   
  [Frontiers in Pharmacology (2020)](https://doi.org/10.3389/fphar.2020.00269) |  [code](https://github.com/insilicomedicine/BiAAE) 

* **De novo generation of hit-like molecules from gene expression signatures using artificial intelligence** [2020]  
  Méndez-Lucio, Oscar, Benoit Baillif, Djork-Arné Clevert, David Rouquié, and Joerg Wichard.   
  [Nat Commun 11, 10 (2020)](https://doi.org/10.1038/s41467-019-13807-w) 





## Multi-Objective deep molecular generative models

* **GENERA: A Combined Genetic/Deep-Learning Algorithm for Multiobjective Target-Oriented De Novo Design** [2023]   
  Lamanna, Giuseppe, Pietro Delre, Gilles Marcou, Michele Saviano, Alexandre Varnek, Dragos Horvath, and Giuseppe Felice Mangiatordi.   
  [J. Chem. Inf. Model. (2023)](https://doi.org/10.1021/acs.jcim.3c00963) |  [code](https://github.com/GiuseppeLamanna/GENERA) 

* **Multi-Objective and Many-Objective Optimisation: Present and Future in de novo Drug Design**  [2023]   
 Angelo, Jaqueline S., Isabella Alvim Guedes, Helio JC Barbosa, and Laurent E. Dardenne.   
  [chemrxiv-2023-q0zdf-v2 (2023)](https://doi.org/10.26434/chemrxiv-2023-q0zdf-v2) 

* **FSM-DDTR: End-to-end feedback strategy for multi-objective De Novo drug design using transformers**  [2023]   
 Monteiro, Nelson RC, Tiago O. Pereira, Ana Catarina D. Machado, José L. Oliveira, Maryam Abbasi, and Joel P. Arrais.   
  [Computers in Biology and Medicine (2023)](https://doi.org/10.1016/j.compbiomed.2023.107285) |  [code](https://github.com/larngroup/FSM-DDTR) 

* **MolSearch: Search-based Multi-objective Molecular Generation and Property Optimization** [2022]   
  Sun, Mengying, Jing Xing, Han Meng, Huijun Wang, Bin Chen, and Jiayu Zhou.   
  [KDD '2022](https://dl.acm.org/doi/abs/10.1145/3534678.3542676) |  [code](https://github.com/illidanlab/MolSearch) 

* **MGCVAE: Multi-Objective Inverse Design via Molecular Graph Conditional Variational Autoencoder** [2022]   
  Lee, Myeonghun, and Kyoungmin Min.   
  [J. Chem. Inf. Model. 2022, 62, 12, 2943–2950](https://pubs.acs.org/doi/10.1021/acs.jcim.2c00487) |  [code](https://github.com/mhlee216/MGCVAE) 

* **Multi-Objective Molecule Generation using Interpretable Substructures** [2020]  
  Jin, Wengong, Regina Barzilay, and Tommi Jaakkola.   
  [ICML (2020)](http://proceedings.mlr.press/v119/jin20b.html) |  [code](https://github.com/wengong-jin/multiobj-rationale) 

* **DeepGraphMolGen, a multi-objective, computational strategy for generating molecules with desirable properties: a graph convolution and reinforcement learning approach** [2020]  
  Khemchandani, Yash, Stephen O’Hagan, Soumitra Samanta, Neil Swainston, Timothy J. Roberts, Danushka Bollegala, and Douglas B. Kell.   
  [ J Cheminform 12, 53 (2020)](https://doi.org/10.1186/s13321-020-00454-3) |  [code](https://github.com/dbkgroup/prop_gen) 

* **Multi-objective de novo drug design with conditional graph generative model** [2018]  
  Li, Yibo, Liangren Zhang, and Zhenming Liu.   
  [J Cheminform 10, 33 (2018)](https://doi.org/10.1186/s13321-018-0287-6) |  [code](https://github.com/kevinid/molecule_generator) 





## Quantum deep molecular generative models

* **Quantum computing for near-term applications in generative chemistry and drug discovery** [2023]   
 Pyrkov, Alexey, Alex Aliper, Dmitry Bezrukov, Yen-Chu Lin, Daniil Polykovskiy, Petrina Kamya, Feng Ren, and Alex Zhavoronkov.   
  [Drug Discovery Today (2023)](https://doi.org/10.1016/j.drudis.2023.103675) 

* **Exploring the Advantages of Quantum Generative Adversarial Networks in Generative Chemistry** [2023]   
 Kao, Po-Yu, Ya-Chu Yang, Wei-Yin Chiang, Jen-Yueh Hsiao, Yudong Cao, Alex Aliper, Feng Ren et al.   
  [J. Chem. Inf. Model. 2023, 63, 11, 3307–3318](https://pubs.acs.org/doi/10.1021/acs.jcim.3c00562) |  [code](https://github.com/pykao/QuantumMolGAN-PyTorch) 

* **Quantum Generative Models for Small Molecule Drug Discovery** [2021]  
  Li, Junde, Rasit O. Topaloglu, and Swaroop Ghosh.   
  [IEEE Transactions on Quantum Engineering (2021)](https://ieeexplore.ieee.org/document/9520764) |  [code](https://github.com/jundeli/quantum-gan) 





## Spectra-based



### Mass Spectra-based



* **Deep Learning-Enabled MS/MS Spectrum Prediction Facilitates Automated Identification Of Novel Psychoactive Substances** [2023]   
 Wang, Fei, Daniel Pasin, Michael A. Skinnider, Jaanus Liigand, Jan-Niklas Kleis, David Brown, Eponine Oler et al.   
  [Anal. Chem. (2023)](https://doi.org/10.1021/acs.analchem.3c02413) |  [data](https://nps-ms.ca/) 

* **MIST-CF: Chemical formula inference from tandem mass spectra** [2023]   
 Litsa, E.E., Chenthamarakshan, V., Das, P. et al.   
  [arXiv:2307.08240 (2023)](https://arxiv.org/abs/2307.08240) |  [code](https://github.com/samgoldman97/mist-cf) 

* **An end-to-end deep learning framework for translating mass spectra to de-novo molecules** [2023]   
 Litsa, E.E., Chenthamarakshan, V., Das, P. et al.   
  [Commun Chem 6, 132 (2023)](https://doi.org/10.1038/s42004-023-00932-3) |  [code](https://github.com/KavrakiLab/Spec2Mol)

* **MSNovelist: de novo structure generation from mass spectra**  [2022]   
 Stravs, M.A., Dührkop, K., Böcker, S. et al   
  [Nat Methods 19, 865–870 (2022)](https://doi.org/10.1038/s41592-022-01486-3) |  [code](https://github.com/meowcat/MSNovelist) 





### NMR Spectra-based


* **NMR-TS: de novo molecule identification from NMR spectra**  [2020]   
 Zhang, Jinzhe, Kei Terayama, Masato Sumita, Kazuki Yoshizoe, Kengo Ito, Jun Kikuchi, and Koji Tsuda   
  [Science and technology of advanced materials 21.1 (2020)](https://doi.org/10.1080/14686996.2020.1793382) |  [code](https://github.com/tsudalab/NMR-TS) 



### Cryo-EM Maps-based

* **Protein-Ligand Binding Site Prediction and de Novo Ligand Generation from Cryo-EM Maps**  [2023]   
 Lu, Chunyang, Kaustav Mitra, Kiran Mitra, Hanze Meng, Shane Thomas Rich-New, Fengbin Wang, and Dong Si.   
  [bioRxiv, 2023-11 (2023)](https://doi.org/10.1101/2023.11.16.567458) |  [Website](https://deeptracer.uw.edu/) 
  

